---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")
flemer_data <- read_tsv("../data/process/flemer/flemer.metadata")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in relative risk alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_RR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_tissue_composite.csv"))

#Load in relative risk select genera data tables
select_RR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_stool_composite.csv"))
select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_tissue_composite.csv"))

adn_select_RR_stool <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_stool_composite.csv"))
adn_select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_tissue_composite.csv"))



```



# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Cancer

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that there is a crucial role for the microbiota in colorectal cancer (CRC) pathogenesis. Important drivers within this context have ranged from individual microbes to the whole community. Our study expands on a recent meta-analysis investigating microbial biomarkers for CRC by testing the hypothesis that the bacterial community has important associations to both early (adenoma) and late (carcinoma) stage disease. To test this hypothesis we examined both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and colon tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) across 14 previously published 16S rRNA gene sequencing studies on CRC and the microbiota.

**Results.** Fecal samples had a significant decrease for both Shannon diversity and evenness, after correcting for study effect and variable region sequenced, with more severe disease (P-value `r ifelse(min(filter(stool_lme, !(alpha_metric %in% "sobs"))[, "BH"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05). This reduction in evenness translated into small increases in relative risk for adenoma (P-value = `r format(min(filter(alpha_RR_adn_stool, !(measure %in% "sobs"))[, "pvalue"]), digits = 2)`) and carcinoma stages of CRC (P-value = `r format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2)`) while the reduction in Shannon diversity only translated into an increased relative risk for developing carcinomas (P-value = `r format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2)`). Increases in mouth-associated microbes were commonly in the top 5 most significantly increased relative risk of adenoma and carcinoma for both stool and tissue samples. A prediction model for adenoma and carcinoma was built using either the whole community or selected genera with highest and lowest relative risk from fecal and tissue samples. Both approaches resulted in similar classification success according to Area Under the Curve (AUC) regardless of whether genera or OTUs were used to build the model. The most important groups within the full community models consistently belonged to genera such as *Ruminococcus*, *Bacteroides*, and *Roseburia* across studies. Although a number of associations between the microbiota and CRC were identified, the majority of studies that we used in this meta-analysis were only individually adequately powered for large effect sizes. 

**Conclusions.** These data provide support for the importance of the bacterial community to both adenoma and carcinoma genesis. The evidence collected within this study on the role of the microbiota in CRC identifies a number of correlations that may not have been detected because of the low power associated with the majority of studies that have been performed to date. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world-wide health problem in which the microbiota has been purported to play an active role in disease pathogenesis [@siegel_cancer_2016; @flynn_metabolic_2016]. Numerous studies have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in tumorgenesis using mouse models of CRC. There have also been numerous case/control studies investigating the microbiota in the formation of both adenoma and carcinoma. A recent meta-analysis investigated whether specific biomarkers could be consistently identified using multiple data sets [@shah_leveraging_2017]. This meta-analysis focused on identifying microbial signatures of CRC (biomarkers) but did so on a small total number of individuals and only investigated stool. 


Although there has been an intense focus on microbiota-based biomarker discovery for CRC, the number of candidate genera seem to be endless. Some studies point towards mouth-associated genera such as *Fusobacterium*, *Peptostreptococcus*, *Parvimonas*, and *Porphyromonas* as key enriched genera [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Yet, even in these studies, mouth-associated genera are far from the only microbes identified to be associated with CRC. These other genera include, but are not limited to, *Providencia*, *Mogibacterium*, *Enterococcus*, *Escherichia/Shigella*, *Klebsiella*, and *Streptococcus* [@burns_virulence_2015; @chen_human_2012; @wang_structural_2012]. In fact, there is good *in vivo* evidence that *Escherichia/Shigella* and *Streptococcus* can be important in the pathogenesis of CRC [@ecoli_Arthur_2014; @arthur_intestinal_2012; @strep_Aymeric_2017]. Other studies have also identified *Akkermansia muciniphila* and *Bacteroides fragilis* as potential markers of CRC with good mechanistic studies for the latter [@weir_stool_2013; @wang_structural_2012; @bfrag_Boleij_2014]. A recent meta-analysis confirmed the correlations of certain mouth-associated genera and *Akkermansia muciniphila* with carcinoma [@shah_leveraging_2017]. However, the sample size (n = 509) is equal to or less than some more recent individual studies investigating the microbiota and CRC, making it hard to know how extrapolatable these findings are. That particular meta-analysis also added more potential microbial associations to both carcinoma (*Pantoea agglomerans* *Ruminococcus*, *Lactobacillus*) and adenoma (*Prevotella*, *Methanosphaera*, *Succinovibrio*, *Haemophilus parainfluenzae*, *Ruminococcus*, *Lactobacillus*) stages of CRC that need to be investigated further, since a number of these genera have been found to be enriched in controls and not disease [@baxter_microbiota-based_2016; @zeller_potential_2014; @burns_virulence_2015]. Additionally, genera like *Roseburia* have been found in some studies to be increased in CRC but in others to either be decreased or have no difference [@geng_diversified_2013; @wang_structural_2012; @flemer_tumour-associated_2017; @dejea_microbiota_2014].  

Most of these studies have focused on carcinoma but the adenoma observations are not any clearer at identifying candidate genera of disease. Groups focusing on broad scale community metrics have found that metrics such as richness are decreased in the adenoma stage of CRC versus controls. Other studies have identified *Lactococcus*, *Pseudomonas*, *Acidovorax*, *Cloacibacterium*, *Helicobacter*, *Lactobacillus*, *Bilophila*, *Desulfovibrio*, and *Mogibacterium* to be increased in adenoma [@sanapareddy_increased_2012; @lu_mucosal_2016; @hale_shifts_2017]. Based on the studies mentioned, there seems to be very little overlap between the genera identified to be associated with adenoma and carcinoma, with *Lactobacillus* being one of the few commonalities.   

Targeting the identification of CRC microbial biomarkers within stool seems logical since it offers an easy and cost-effective way to stratify risk and the current gold standard for diagnosis, a colonoscopy, can be time-consuming and is not without risk of complications. Although stool represents an easy and less invasive way to assess risk, it is not clear how well this sample reflects adenoma- and carcinoma- associated microbial communities. Some studies have tried to assess this in health and disease but are limited by their sample size [@flemer_tumour-associated_2017; @Flynn_preprint_2017]. Sampling the microbiota directly associated with colon tissue may provide clearer answers but is not without limitations. The community present for sampling following the colonoscopy bowel prep may reflect the better adhered microbiota versus the resident community. Additionally, these samples contain more host DNA, potentially limiting the types of analysis that can be done. It is well known that low biomass samples can be very difficult to work with and results can be study dependent due to the randomness of contamination [@Salter_contamination_2014].       

In comparison to the previous meta-analysis, this study significantly increases the total stool samples investigated, re-examines important genera across adenoma and carcinoma across study, and examines differences and similarities between stool and tissue microbiota in the context of CRC. Importantly, this analysis and approach could provide valuable insights into the common genera that are both protective and detrimental in CRC and whether broad bacterial community measurements can account for these changes that were not provided by earlier meta-analysis studies [@shah_leveraging_2017].


Using both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and colon tissues (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) totaling over `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T) + sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples across 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] [Table 1 & 2], we expand both the breadth and scope of the previous meta-analysis to investigate whether the bacterial community or specific members are more important risk factors for both adenoma and carcinoma stages of CRC. To accomplish this we first assessed whether bacterial diversity changes throughout disease (control to adenoma to carcinoma) and if it results in an increased relative risk (RR) for adenoma or carcinoma stages of CRC. We then assessed what genera, if any, increase or decrease the RR of adenoma or carcinoma stages of CRC. Next, using Random Forest models, we analyzed whether the full community or only the combined top 5 increased and top 5 decreased RR genera resulted in better model classification, based on the area under the curve (AUC). Finally, we also examined at what effect and sample size the studies used were powered for and the sample size needed to get to the traditionaly accepted 80% power. Our results from these analyses suggests that the bacterial community changes as disease severity worsens, that specific members are important for disease classification, and that many of the studies are underpowered for assessing small effect sizes. 

\newpage

## Results

***Lower Bacterial Diversity is Associated with Increased RR of CRC:***
To assess differences in broad scale community metrics as disease severity worsens Operational Taxonomic Unit (OTU) richness, evenness, and Shannon diversity measurements were power transformed and Z-score normalized. These metrics are commonly used to assess the total number of OTUs, the equality of their abundance, and the overall diversity, respectively. Using linear mixed-effect models to control for study and variable region we assessed whether OTU richness, evenness, or Shannon diversity changed in a step-wise manner with disease severity. In stool, there was a significant decrease in both evenness and Shannon diversity as disease severity moved from control to adenoma to carcinoma (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))` and `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`, respectively) [Figure 1A]. We next tested whether the detectable differences in community significantly increased in RR of having an adenoma or carcinoma. For fecal samples, a decrease versus the overall median in evenness resulted in a significantly increased RR for carcinoma  (RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`),  P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) and adenoma (RR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) while a decrease versus the overal median in Shannon diversity only increased the RR for carcinoma (RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2]. Using the Bray-Curtis distance metric and PERMANOVA, it was also possible to identify significant bacterial community changes, in specific studies, for both carcinoma-associated and adenoma-associated microbiota versus control [Table S1 & S2].

Using similar transformations for tissue samples, linear mixed-effect models were used on the transformed combined data to control for study, re-sampling of the same individual, and 16S variable region to test whether OTU richness, evenness, or Shannon diversity changed in a step-wise manner as disease severity increased. For colon tissue, there were no significant changes in OTU richness, evenness, or Shannon diversity as disease severity progressed from control to adenoma to carcinoma (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05) [Figure 1B & C]. We next analyzed the RR, for matched (unaffected tissue and an adenoma or carcinoma from the same individual) and unmatched (control and adenoma or carcinoma tissue not from the same individual) colon tissue samples. For individuals at either an adenoma or carcinoma stage of disease there was no significant change in RR based on lower than median values for OTU richness, evenness, and Shannon diversity [Table S3-S5]. Similar to stool samples, significant differences in bacterial community, assessed by PERMANOVA, were identified in unmatched tissue samples, for those at either adenoma or carcinoma stage of CRC [Table S6 & S7]. For studies with matched samples no differences in bacterial community were observed when assessed with PERMANOVA [Table S6 & S7]. These tissue results suggest that the microbiota within an individual are similar to each other regardless of disease status.

***Mouth-Associated Genera are Associated with an Increased RR of CRC:***
Next, we asked if being higher than the median relative abundance, for any specific genera, resulted in an altered RR for adenoma or carcinoma, in stool and colon tissue, due to our previous observations of small increases in RR using OTU richness and Shannon diversity. To investigate this we analyzed all common genera across each study, in colon tissue or stool, and assessed whether a relative abundance higher than the median results in an increase or decrease in RR. Mouth-associated genera were commonly found in the top 5 genera associated with an increased RR of having an adenoma (*Pyramidobacter* [Figure 3A] and *Rothia* [Figure 3C]) and carcinoma (*Fusobacterium*, *Parvimonas*, *Porphyromonas*, and *Peptostreptococcus* [Figure 3B] and *Fusobacterium* [Figure 3D]) for both stool and colon tissue samples. Conversely, genera commonly associated with a normal gastrointestinal tract were correlated with a decreased RR for both adenoma and carcinoma for both stool and colon tissue samples [Figure 3]. Even though mouth-associated genera were identified across disease stage, there was little direct overlap of the top 5 increased or decreased RR genera between both stages and sample site. 

When observing RRs with a P-value less than 0.05 there was almost no overlap between genera from stool or colon tissue and when they were similar the RR was in opposite directions (e.g. *Lactococcus*) [Table S8 & S9]. Many of the genera that had RRs with a P-value under 0.05 for colon tissue are also highly prevalent in contamination, specifically, *Novosphingobium*, *Selemonas*, and *Achromobacter* [Figure 3 & Table S8-S9]. For carcinoma stage of CRC, certain mouth-associated genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*) had a high RR for both colon tissue and stool samples [Table S10 & S11]. Finally, these data suggest that the most significantly increased RR genera for tissue was *Camplyobacter* while in stool it was *Peptostreptococcus* [Table S10 & S11]. 


***Select Genera Models can Recapitulate Whole Community Models:***
Since specific genera increased RR for carcinoma over diversity metrics we assessed whether the bacterial community was better at classifying disease versus only a select group of genera. We selected these genera based on their RR and P-value significance and used two approaches to test this question.
The first approach used genus level data and tested for differences in AUC between all genera and selected genera. A single study was used for training the model prior to testing on all other studies and this was repeated for every study in the meta-analysis. The second approach used OTU level data and tested for a generalized decrease in the 10-fold cross validation (CV) model AUC which is a common approach used to guard against over-fitting. This was applied across study and the AUC of the all OTUs model was compared against the model that used only OTUs that taxonomically classified to selected genera. 


For the first approach using the genera-based models, the training set median AUC for model classification was similar for both the select genera and full genera models, for both tissue and stool studies [Figure S2-S3]. When analyzing the tests sets that were comprised of genera data from other studies, both models had a similar ability to detect individuals with adenomas or carcinomas, with the select genera models performing better in some instances [Figure S4-S6]. Conversely, the second approach that used OTU-based models showed a slight decrease in median AUC between the full and select models, with one exception to this generalization being the carcinoma models for matched colon tissue [Figure 4 & 5].  


In stool, the most common genera in the top 10 most imporant variables, in the full community models using the first approach, were *Ruminococcus*, *Bacteroides*, and *Roseburia* [Figure 6A & B]. Regardless of sample type, mouth-associated genera were present in models for the carcinoma stage of CRC [Figure 6A & B]. Yet, none were present in the majority of studies and *Fusobacterium* was the only genus present in the adenoma stage of CRC [Figure 6A & B]. For the second approach that utilized full OTU-based models, *Ruminococcaceae* was present in the top 10 consistently for both adenoma and carcinoma models while *Roseburia* was only present in many adenoma models and *Bacteroides* was present in the overwhelming majority of the carcinoma models [Figure 6C & 6D].


Unlike the stool-based Random Forest models, the tissue-based models, for the full genera from the first approach, showed no consistent representation of *Ruminococcaceae*, *Ruminococcus*, *Bacteroides*, and *Roseburia* in the top 10 most important model variables across study [Figure S7]. The vast majority of the top 10 model variables for the genera- and OTU-based models using colon tissue tended to be study specific. Further, there was very little overlap in the top 10 important variables between adenoma and carcinoma stage models, regardless of whether colon tissue or stool was used  [Figure 6 & S7]. This discordance between stool and colon tissue samples also applies to the mouth-associated genera with one noticeable skew being that *Fusobacterium* and *Fusobacteriaceae* occur more often in the top 10 of matched versus unmatched colon tissue Random Forest models [Figure S7B-C & S7E-F]. This suggests that either the colon tissue microbiota is study and person dependent or that kit and/or other types of contamination associated with low biomass samples may be skewing the results.

***CRC Studies are Underpowered for Detecting Small Effect Sizes:***
Next, we assessed how much confidence should be placed in the reported outcomes from each individual study by calculating the ability to detect a difference (power) and sample size needed for small, medium, and large effect size differences between cases and controls. When assessing the power of each study at different effect sizes the majority of studies acheived 80% power to detect a 30% or greater difference between groups [Figure 7A & B]. No study that we analyzed had the standard 80% power to detect an effect size difference equal to or below 10% [Figure 7A & B].  In order to achieve a power of 80%, for small effect sizes, studies used in our meta-analysis would need to recruit over 1000 individuals for both the case and control arms [Figure 7C]



\newpage

## Discussion

Our study identifies clear differences in diversity, both at the community level and for individual genera, present in patients with and without CRC [Figure 1-3]. Although there was a step-wise decrease in diversity as disease progressed from control to adenoma to carcinoma, this did not translate into large effect sizes for the RR of CRC. Even though mouth-associated genera increased the RR of having a carcinoma, they did not consistently increase the RR of having an adenoma. Additionally, our observations suggest that by combining mouth-associated and CRC protective microbes we can classify either adenoma or carcinoma stage of disease as well as models that use the full community. 


The data presented herein support the importance of select genera for carcinoma, but not necessarily adenoma, formation.
The results that we have presented show that both the genera and OTU select and full models, for the carcinoma stage of CRC, had similar AUCs [Figure 4 & 5]. This suggests that an interplay between a select number of potentially protective and exacerbating microbes within the GI community is crucial for carcinoma formation. Importantly, it suggests that there may be key members of the GI community that might be studied further to potentially reduce the risk of carcinoma. Conversely, using the present data, it is clear that new approaches may be needed to identify members of the community associated with adenoma stage of disease. Regardless of sample type and whether a full or select model was used, our Random Forest models consistently performed poorly. Yet, the step-wise decrease in diversity suggests that the adenoma-associated community is not normal but has changed subtly [Figure 1]. This change in diversity, at this early stage of disease, could be focal to the adenoma itself. One possible explanation is that how the host interacts with these subtle changes at early stages of the disease is what leads to a thoroughly dysfunctional community that is supportive of CRC genesis. 


Within stool, common GI microbes were most consistently present in the top 10 genera or OTUs across studies [Figure 6]. Changes in *Bacteroides*, *Ruminococcaceae*, *Ruminococcus*, and *Roseburia* were consistently found to be in the top 10 most important variables across the different studies for both adenoma and carcinoma [Figure 6]. These data suggest that whether the non-resident bacterium is *Fusobacteria* or *Peptostreptococcus* may not be as important as how these bacteria interact with the changing resident community. Based on these observations, it is possible to hypothesize that small changes in community structure lead to new niches in which any one of the mouth-associated genera can gain a foothold, exacerbating the initial changes in community and facilitating the transition from adenoma to carcinoma stage of disease.     

The colon tissue-based studies did not provide a clearer understanding of how the microbiota may be associated with CRC. Generally, the full OTU-based models of unmatched and matched colon tissue samples were concordant with stool samples showing that GI resident microbes were the most prevalent in the top 10 most important variables across study [Figure S7E & F]. Unlike in stool, *Fusobacterium* was the only mouth-associated bacteria consistently present in the top 10 most important variables of the full carcinoma stage models [Figure S7B-C & E-F]. The majority of the colon tissue-based results seem to be study specific with many of the top 10 taxa being present only in a single study. Additionally, the presence of genera associated with contamination, within the top 10 most important variables for the genera and OTU models is worrying. The low bacterial biomass of tissue samples coupled with potential contamination could explain why these results seem to be more sporadic than the stool results.

One important caveat to this study is that even though genera associated with certain species such as *Bacteroides fragilis* and *Streptococcus gallolyticus* subsp. *gallolyticus* were not identified, it does not necessarily mean that these specific species are not important in human CRC [@bfrag_Boleij_2014; @strep_Aymeric_2017]. Since we are limited in our aggregation of the data to the genus level, it is not possible to clearly delineate which species are contributing to overall disease progression. Our observations are not inconsistent with the previous literature on either *Bacteroides fragilis* or *Streptococcus gallolyticus* subsp. *gallolyticus*. As an example, the stool-based full community models consistently identified the genus *Bacteroides*, as well as OTUs that classified as *Bacteroides*, to be important model components across studies. This suggests that even though *Bacteroides* may not increase the RR of CRC and may not vary in relative abundance, like *Fusobacterium*, it is still important in CRC. Additionally, *Streptococcus gallolyticus* subsp. *gallolyticus* is a mouth-associated micobe, and the results from this study suggest that regardless of sample type, mouth-associated genera are commonly associated with an increased RR for both adenoma and carcinoma stage of disease.    

The associations between the microbiota and adenoma stage of disease are inconclusive, in part, because many studies may not be powered effectively to observe small effect sizes. None of the studies analyzed were properly powered to detect a 10% or lower change between cases and controls. The results within our meta-analysis suggest that a small effect size may well be the scope in which differences consistently occur between controls and adenoma stage of disease. Future studies investigating adenoma stage and the microbiota need to take power into consideration to reproducibly study whether the microbiota contributes to polyp formation. In contrast to adenoma stage of disease, our observations suggest that most studies analyzed have sufficient power to detect many changes in the carcinoma-associated microbiota because of large effect size differences between cases and controls [Figure 7].

\newpage

## Conclusion

By aggregating together a large collection of studies analyzing both fecal and colon tissue samples, we are able to provide evidence supporting the importance of the bacterial community in CRC. Further, the data presented here suggests that mouth-associated microbes can gain a foothold within the colon and are are commonly associated with the greatest RR of having a carcinoma. No conclusive signal with these mouth-associated micobes could be detected for adenoma stage of disease. Our observations also highlight the importance of power and sample number considerations when investigating the microbiota and adenoma stage of disease due to the subtle changes in the community. Overall, associations of microbiota with the carcinoma stage of CRC are much stronger than those with the adenoma stage.

\newpage

## Methods

***Obtaining Data Sets:***
The studies used for this meta-analysis were identified through the review articles written by Keku, *et al.* and Vogtmann, *et al.* [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016] and additional studies not mentioned in the reviews were obtained based on the authors' knowledge of the literature. Studies that used tissue or feces as their sample source for 454 or Illumina 16S rRNA gene sequencing analysis and had data sets with sequences available for analysis were included. Some studies were excluded because they did not have publicly available sequences or did not have metadata in which the authors were able to share. After these filtering steps, the following studies remained: Ahn, *et al.* [@ahn_human_2013], Baxter, *et al.* [@baxter_microbiota-based_2016], Brim, *et al.* [@brim_microbiome_2013], Burns, *et al.* [@burns_virulence_2015], Chen, *et al.*  [@chen_human_2012], Dejea, *et al.* [@dejea_microbiota_2014], Flemer, *et al.* [@flemer_tumour-associated_2017], Geng, *et al.* [@geng_diversified_2013], Hale, *et al.* [@hale_shifts_2017], Kostic, *et al.* [@kostic_genomic_2012], Lu, *et al.* [@lu_mucosal_2016], Sanapareddy, *et al.* [@sanapareddy_increased_2012], Wang, *et al.* [@wang_structural_2012], Weir, *et al.* [@weir_stool_2013], and Zeller, *et al.*  [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included because the 90 individuals analyzed within the study are contained within the larger Baxter study [@baxter_microbiota-based_2016]. After sequence processing, all the case samples for the Kostic study had 100 or less sequences remaining and was excluded, leaving a total of 14 studies that analysis could be completed on.

***Data Set Breakdown:***
In total, there were seven studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), five studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].  

***Sequence Processing:***
For the majority of studies, raw sequences were downloaded from the Sequence Read Archive (SRA) (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata were obtained by searching the respective accession number of the study at the following website: http://www.ncbi.nlm.nih.gov/Traces/study/. Of the studies that did not have sequences and metadata on the SRA, data was obtained from DBGap (n = 1, [@ahn_human_2013]) and directly from the authors (n = 4, [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]). Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009] and quality filtering utilized the default methods for both 454 and Illumina based sequencing. If it was not possible to use the defaults, the stated quality cut-offs, from the study itself, were used instead. Sequences that were made up of an artificial combination of two or more different sequences and commonly known as chimeras were identified and removed using VSEARCH [@rognes_vsearch_2016] before *de novo* OTU clustering at 97% similarity was completed using the OptiClust algorithm [@westcott_opticlust_2017]. 

***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R (v`r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. For OTU richness, evenness, and Shannon diversity analysis, values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package [@rcompanion_citation_2017] and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package [@car_citation_2011]. Testing for $\alpha$-diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package [@lme4_citation_2015] to correct for study, repeat sampling of individuals (tissue only), and 16S hypervariable region used. Relative risk was analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages [@epir_citation_2017; @metafor_citation_2010] by assessing how many individuals with and without disease were above and below the overall median value within each specific study. Relative risk significance testing utilized the chi-squared test. $\beta$-diversity differences utilized a Bray-Curtis distance matrix and PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package [@vegan_citation_2017]. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages [@caret_citation_2017; @randomforest_citation_2002]. Power analysis and estimations were made using the pwr (v`r paste(packageVersion("pwr"))[1]`) and statmod (v`r paste(packageVersion("statmod"))[1]`) packages [@pwr_citation_2017; @statmod_citation_2016]. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages [@ggplot2_citation_2009; @gridextra_citation_2017]. 

***Study Analysis Overview:***
OTU richness, evenness, and Shannon diversity was first assessed for differences between controls, adenoma stage, and carcinoma stage using both linear mixed-effect models and RR. The Bray-Curtis index was used to assess, for each individual study, differences between control-adenoma and control-carcinoma. Next, all common genera were assessed for differences in RR for having an adenoma or carcinoma and ranked based on P-value. We then built Random Forest models based on full or  selected community, based on the top 5 increased and top 5 decreased RR based on P-value, and these models were trained on one study then tested on the remaining studies. This process was repeated for every study in the meta-analysis. A similar approach was then applied at the OTU level with the exception that a 10-fold CV over 100 different models, based on random 80/20 splitting of the data, was used to generate a range of expected AUCs. For these OTU-based models, the selected model included all OTUs that had a taxonomic classification to a taxa in the top 5 increased and top 5 decreased RR based on P-value. Finally, the power of each study was assessed for an effect size ranging from 1% to 30% and an estimated sample size, for these effect sizes, was generated based on 80% power. For comparisons in which only control versus adenoma stage were made, the carcinoma samples were excluded from each respective study. Similarly, for comparisons in which control versus carcinoma stage were made the adenoma samples were excluded from each respective study. The data were split between feces and tissue samples. Within the tissue groups the data were further divided between samples from the same individual (matched) and those from different individuals (unmatched). Where applicable for each study, predictions for adenoma and carcinoma stage of disease were then tested for feces, matched tissue, and unmatched tissue.

***Obtaining Genera Relative Abundance and Selected Models:***
For the genera analysis of the RR, OTUs were added together based on the genus or lowest available taxonomic classification level and the total average counts, for 100 different subsamplings, were collected. The OTU based Random Forest Models using selected OTUs utilized a similar approach except that the OTUs were not aggregated together by taxonomic identity but kept as separate OTUs. OTU Random Forest models using the full community included all OTUs while those for the selected model included only those OTUs that had a taxonomic classification to a variable in the top 5 increased of top 5 decreased RR based on P-value.   

***Matched versus Unmatched Tissue Samples:***
In general, tissue samples with control and CRC samples from different indivdiuals were classified as unmatched while samples that belonged to the same individual were classified as matched. Studies with matched data included Burns, Dejea, Geng, and Lu while those with unmatched data were from Burns, Flemer, Chen, and Sanapareddy. For some studies samples became unmatched when a corresponding matched sample did not make it through sequence processing. All samples, from both tissue sample types, were analyzed together for the linear mixed-effect models with samples from the same individual corrected for. For all other analysis, not mentioned herein, matched and unmatched samples were analyzed separately using the statistical approaches mentioned in the Statistical Analysis section. 

***Assessing Important Random Forest Model Variables:***
Using Mean Decrease in Accuracy (MDA) the top 10 most important variables to the Random Forest model were obtained in two different ways depending on whether the model used genera or OTU data. For the genus based models, the number of times that a genus showed up in the top 10 of the training set across each study was counted while, for the OTU based models, the medians for each OTU across 100 different 80/20 splits of the data was generated and the top 10 OTUs then counted for each study. Common taxa, for the OTU based models, were identified by using the lowest classification within the RDP database for each of the specific OTUs obtained from the previous counts and the number of times this classification occurred in the top 10, in each study, was recorded. The two studies that had adenoma tissue were equally divided between matched and unmatched groups and were grouped together for the counting of the top 10 genera and OTUs.

***Reproducible Methods:***
The code and analysis can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless otherwise mentioned, the accession number of raw sequences from the studies used in this analysis can be found directly in the respective batch file in the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscripts used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective study batch file in the GitHub repository or within the original publication. For instances when sequences are not publicly available, they may be accessed by contacting the corresponding authors from whence the data came.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a Canadian Institute of Health Research fellowship and a University of Michigan Postdoctoral Translational Scholar Program grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally, we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Total Individuals in each Study Included in the Stool Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-V3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: Community Differences between Control, Adenoma, and Carcinoma Across Sampling Site.** A) Stool sample community differences by disease group. B) Unmatched tissue samples differences by disease group. C) Matched tissue sample differences by group disease group. The dashed line represents a Z-score of 0 or no difference from the median.

**Figure 2: Relative Risk for Adenoma or Carcinoma based on Bacterial Community Metrics in Stool.** A) Community-based relative risk for adenoma. B) Community-based relative risk for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 3: Top 5 Genera that Decrease and Increase Relative Risk for Lesion.** A) Adenoma relative risk in stool. B) Carcinoma relative risk in stool. C) Adenoma relative risk in tissue. D) Carcinoma relative risk in tissue. For all panels the relative risk was also compared to whether one, two, three, or four of the CRC-associated genera were present.  

**Figure 4: Stool OTU Random Forest Model Across Studies.** A) Adenoma random forest model between the full community and select genera OTUs only. B) Carcinoma random forest model between the full community and select genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between. The solid red line represents the median AUC of all the studies for either the full community or select genera OTUS only model.

**Figure 5: Tissue OTU Random Forest Model Across Studies.** A) Adenoma random forest model between the full community and select genera OTUs only. B) Carcinoma random forest model between the full community and select genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between. The solid red line represents the median AUC of all the studies for either the full community or select genera OTUS only model.

**Figure 6: Most Common Genera Across Full Community Stool Study Models.** A) Common genera in the top 10 for adenoma Random Forest genus models. B) Common genera in the top 10 for carcinoma Random Forest genus models. C) Common genera in the top 10 for adenoma Random Forest OTU models. D) Common genera in the top 10 for carcinoma Random Forest OTU models.

**Figure 7: Power and Effect Size Analysis of Studies Included.** A) Power based on effect size for studies with adenoma individuals. B) Power based on effect size for studies with carcinoma individuals. C) The estimated sample number needed for each arm of each study to detect an effect size of 1-30%. The dotted red lines in A) and B) represent a power of 0.8.  


\newpage


**Figure S1: Relative Risk for Adenoma or Carcinoma based on Bacterial Community Metrics in Tissue.** A) Community-based relative risk for adenoma. B) Community-based relative risk for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Stool Random Forest Genus Model AUC for each Study.** A) AUC of adenoma models using all genera or select genera only. B) AUC of carcinoma models using all genera or select genera only. The black line represents the median within each group.  

**Figure S3: Tissue Random Forest Genus Model AUC for each Study.** A) AUC of adenoma models using all genera or only select genera divided between matched and unmatched tissue. B) AUC of carcinoma models using all genera or select genera only. The black line represents the median within each group divided between matched and unmatched tissue. 

**Figure S4: Stool Random Forest Prediction Success Using Genera Across Studies.** A) AUC for prediction in adenoma using all genera or select genera only. B) AUC for prediction in carcinoma using all genera or select genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. The red lines represent the median AUC using that specific study as the training set.

**Figure S5: Tissue Random Forest Prediction Success of Carcinoma Using Genera Across Studies.** A) AUC for prediction in unmatched tissue for all genera or select genera only. B) AUC for prediction in matched tissue using all genera or select genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. The red lines represent the median AUC using that specific study as the training set.

**Figure S6: Tissue Random Forest Prediction Success of Adenoma Using Genera Across Studies.** The red lines represent the median AUC using that specific study as the training set.

**Figure S7: Most Common Genera Across Full Community Tissue Study Models.** A) Common genera in the top 10 for adenoma Random Forest genus models. B) Common genera in the top 10 for unmatched carcinoma Random Forest genus models. B) Common genera in the top 10 for matched carcinoma Random Forest genus models. D) Common genera in the top 10 for adenoma Random Forest OTU models. E) Common genera in the top 10 for unmatched carcinoma Random Forest OTU models. F) Common genera in the top 10 for matched carcinoma Random Forest OTU models.

\newpage
