---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales"))

```

```{r rdata_tables}

```



# The Importance of the Microbiota and Individual Community Members in Colorectal Cancer

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** 

**Results.** 

**Conclusions.** 


### Keywords

microbiota; colorectal cancer; polyps; treatment; risk factor.

\newpage


## Background



\newpage

## Results

***Section 1***



***Section 2***



***Section 3***



***Section 4***




\newpage

## Discussion




## Conclusion





\newpage

## Methods

***Obtaining Data Sets:***
Studies used for this meta-analysis were identified through the review articles written by Keku, et al. and Vogtmann, et al. [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016]. All studies were included that used tissue or feces as their sample source for 16S rRNA gene sequencing analysis. Studies using either 454 or Illumina sequencing technology were included. Only data sets that had the raw sequences available for analysis were included. Some studies did not have publically available raw sequences or did not have meta data in which the authors were able to share. After this filtering step the following studies remained: Ahn [@ahn_human_2013], Baxter [@baxter_microbiota-based_2016], Brim [@brim_microbiome_2013], Burns [@burns_virulence_2015], Chen [@chen_human_2012], Dejea [@dejea_microbiota_2014], Flemer [@flemer_tumour-associated_2017], Geng [@geng_diversified_2013], Hale [@hale_shifts_2017], Kostic [@kostic_genomic_2012], Lu [@lu_mucosal_2016], Sanapareddy [@sanapareddy_increased_2012], Wang [@wang_structural_2012], Weir [@weir_stool_2013], and Zeller [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included becasue the 90 individuals analyzed within the study are contained within the larger Baxter study. The Kostic study was not used since after sequence processing all the case samples did not have more than 100 sequences remaining. This left a total of 13 studies in which complete analysis could be completed.

***Data Set Breakdown:***
In total there were 7 studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), 5 studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and 2 studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals initially run through the sequence processing for the fecal samples was 1899 and for the tissue samples was 462.    

***Sequence Processing:***
For the majority of studies raw sequences were downloaded from the SRA (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata was obtained from the following website: http://www.ncbi.nlm.nih.gov/Traces/study/ by searching the respective accession number of the study. Of the studies that did not have sequences and meta data on the SRA one study had the data stored on DBGap [@ahn_human_2013] and four studies the data was obtained directly from the authors [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]. Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009]. Where possible quality filtering utilized the default methods used in mothur for either 454 or Illumina based sequencing. If it was not possible to use these defaults the author stated quality cut-offs were used instead. Chimeras were identifed and removed using the VSEARCH [@rognes_vsearch_2016] program and *de novo* OTU clustering at 97% similarity using the OptiClust algorithm [@westcott_opticlust_2017] was utilized. 

***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R software package (v`r paste(version$major, ".", version$minor, sep = "")`). For the alpha diversity analysis values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package. Testing for alpha diversity differences utilized both T-tests and ANOVA. Linear mixed-effect models were created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package to correct for both study and variable region effect in the diversity measures when analyzing colorectal cancer groups. Relative Risk was analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages. Relative risk significance testing utilized the chi-squred test. Beta-diversity differences utilized a Bray-Curtis distance matrix and PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages. Random Forest testing of the obtained AUC versus a random model AUC utilized T-tests. Power analysis and estimations were made using the pwr (v`r paste(packageVersion("pwr"))[1]`) and statmod (v`r paste(packageVersion("statmod"))[1]`) packages. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages. 

***Study Analysis Overview:***


***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_CRCMetaAnalysis_XXXX_2016. Unless mentioned otherwise the accession number for the raw sequences for the studies used in this analysis can be found directly in the respective batch file, on the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Need to fill in.

### Consent for publication

Not applicable.

### Availability of data and material

Need to fill in.

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

Need to fill in.

### Authors' contributions

Need to fill in.

### Acknowledgements

Need to fill in.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: **

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

```


\newpage


**Table 2: **

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}


```


\newpage



**Figure 1: **

**Figure 2: ** 

**Figure 3: ** 

**Figure 4: **  




\newpage


**Figure S1: ** 

**Figure S2: ** 

**Figure S3: **

\newpage
