\documentclass[12pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{longtable,booktabs}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{}
  \pretitle{\vspace{\droptitle}}
  \posttitle{}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\usepackage{amsmath}

\usepackage{mathtools}

\begin{document}

\section{Making Sense of the Noise: Leveraging Existing 16S rRNA Gene
Surveys to Identify Key Community Members in Colorectal
Tumors}\label{making-sense-of-the-noise-leveraging-existing-16s-rrna-gene-surveys-to-identify-key-community-members-in-colorectal-tumors}

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

\begin{itemize}
\tightlist
\item
  \href{mailto:marcsze@med.umich.edu}{\nolinkurl{marcsze@med.umich.edu}}
\end{itemize}

\newpage

\linenumbers

\subsection{Abstract}\label{abstract}

An increasing body of literature suggests that both individual and
collections of bacteria are associated with the progression of
colorectal cancer. As the number of studies investigating these
associations increases and the number of subjects in each study
increases, a meta-analysis to identify the associations that are the
most predictive of disease progression is warranted. We analyzed
previously published 16S rRNA gene sequencing data collected from feces
and colon tissue.

We quantified the odds ratios (ORs) for individual bacterial taxa that
were associated with an individual having tumors relative to a normal
colon. Among the fecal samples, there were no taxa that had a
significant ORs associated with adenoma and there were 8 taxa with
significant ORs associated with carcinoma. Similarly, among the tissue
samples, there were no taxa that had a significant OR associated with
adenoma and there were 3 taxa with significant ORs associated with
carcinoma. Among the significant ORs, the association between individual
taxa and tumor diagnosis was equal or below 7.11. Because individual
taxa had limited association with tumor diagnosis, we trained Random
Forest classification models using only the taxa that had significant
ORs, using the entire collection of taxa found in each study, and using
operational taxonomic units defined based on a 97\% similarity
threshold. All training approaches yielded similar classification
success as measured using the Area Under the Curve. The ability to
correctly classify individuals with adenomas was poor and the ability to
classify individuals with carcinomas was considerably better using
sequences from fecal or tissue.

\newpage

\subsection{Importance}\label{importance}

Colorectal cancer is a significant and growing health problem in which
animal models and epidemiological data suggest that the colonic
microbiota have a role in tumorigenesis. These observations indicate
that the colonic microbiota is a reservoir of biomarkers that may
improve our ability to detect colonic tumors using non-invasive
approaches. This meta-analysis identifies and validates a set of 8
bacterial taxa that can be used within a Random Forest modeling
framework to differentiate individuals as having normal colons or
carcinomas. When models trained using one dataset were tested on other
datasets, the models performed well. These results lend support to the
use of fecal biomarkers for the detection of tumors. Furthermore, these
biomarkers are plausible candidates for further mechanistic studies into
the role of the gut microbiota in tumorigenesis.

\subsubsection{Keywords}\label{keywords}

microbiota; colorectal cancer; polyps; adenoma; tumor; meta-analysis.

\newpage

\subsection{Background}\label{background}

Colorectal cancer (CRC) is a growing world-wide health problem in which
the microbiota has been hypothesized to have a role in disease
progression (1, 2). Numerous studies using murine models of CRC have
shown the importance of both individual microbes (3--7) and the overall
community (8--10) in tumorigenesis. Numerous case-control studies have
characterized the microbiota of individuals with colonic adenomas and
carcinomas in an attempt to identify biomarkers of disease progression
(6, 11--17). Because current CRC screening recommendations are poorly
adhered to due to person's socioeconomic status, test invasiveness, and
frequency of tests, development and validation of microbiota-associated
biomarkers for CRC progression could further attempts to develop
non-invasive diagnostics (18).

Recently, there has been an intense focus on identifying
microbiota-based biomarkers yielding a seemingly endless number of
candidate taxa. Some studies point towards mouth-associated genera such
as \emph{Fusobacterium}, \emph{Peptostreptococcus}, \emph{Parvimonas},
and \emph{Porphyromonas} that are enriched in people with carcinomas (6,
11--17). Other studies have identified members of \emph{Akkermansia},
\emph{Bacteroides}, \emph{Enterococcus}, \emph{Escherichia},
\emph{Klebsiella}, \emph{Mogibacterium}, \emph{Streptococcus}, and
\emph{Providencia} (13--15). Additionally, \emph{Roseburia} has been
found in some studies to be more abundant in people with tumors but in
other studies it has been found to be less abundant than what is found
in subjects with normal colons (14, 17, 19, 20). There is support from
mechanistic studies using tissue culture and murine models that
\emph{Fusobacterium nucleatum}, pks-positive strains of
\emph{Escherichia coli}, \emph{Streptococcus gallolyticus}, and an
entertoxin-producing strain of \emph{Bacteroides fragilis} are important
in tumorigenesis (5, 14, 21--24). These results point to a causative
role for the microbiota in tumorigenesis as well as their potential as
diagnostic biomarkers.

Most studies have focused on identifying biomarkers in patients with
carcinomas but there is a clinical need to identify biomarkers
associated with adenomas to facilitate early detection of the tumors.
Studies focusing on broad scale community metrics have found that
measures such as the total number of taxa (i.e.~richness) are lower in
those with adenomas versus controls (25). Other studies have identified
\emph{Acidovorax}, \emph{Bilophila}, \emph{Cloacibacterium},
\emph{Desulfovibrio}, \emph{Helicobacter}, \emph{Lactobacillus},
\emph{Lactococcus}, \emph{Mogibacterium}, and \emph{Pseudomonas} to be
enriched in those with adenomas (25--27). The ability to classify
individuals as having normal colons or adenomas based solely on the taxa
within fecal samples has been limited. However, when 16S rRNA gene
sequence data was combined with the results of a fecal immunochemical
test (FIT), the ability to diagnose individuals with adenomas was
improved relative to using the FIT results alone (12).

A recent meta-analysis found that 16S rRNA gene sequences from members
of \emph{Akkermansia}, \emph{Fusobacterium}, and \emph{Parvimonas} were
fecal biomarkers for the presence of carcinomas (28). Contrary to
previous studies they found sequences similar to members of
\emph{Lactobacillus} and \emph{Ruminococcus} to be enriched in patients
with adenoma or carcinoma relative to those with normal colons (12, 15,
16). In addition, they found that 16S rRNA gene sequences from members
of \emph{Haemophilus}, \emph{Methanosphaera}, \emph{Prevotella}, and
\emph{Succinovibrio} were enriched in patients with adenomas and
\emph{Pantoea} were enriched in patients with carcinomas. Although this
meta-analysis was helpful for distilling a large number of possible
biomarkers, the aggregate number of samples included in the analysis
(n=509) was smaller than several larger case-control studies that have
since been published (12, 27)

Here we provide an updated meta-analysis using 16S rRNA gene sequence
data from both feces (n=1737) and colon tissue (492 samples from 350
individuals) from 14 studies (11--17, 19, 20, 23, 25--27, 29) {[}Table 1
\& 2{]}. We expand both the breadth and scope of the previous
meta-analysis to investigate whether biomarkers describing the bacterial
community or specific members of the community can more accurately
classify patients as having adenoma or carcinoma. Our results suggest
that the bacterial community changes as disease severity worsens and
that a subset of the microbial community can be used to diagnose the
presence of carcinoma.

\newpage

\subsection{Results}\label{results}

\textbf{\emph{Lower bacterial diversity is associated with higher odds
ratio (OR) of tumors.}} We first assessed whether variation in broad
community metrics like total number of operational taxonomic units
(OTUs) (i.e.~richness), the evenness of their abundance, and the overall
diversity of the communities were associated with disease stage after
controlling for study and variable region differences. In fecal samples,
both evenness and diversity were significantly lower in successive
disease severity categories (evenness P-value=0.025 and diversity
P-value=0.043) {[}Figure 1{]}; there was no significant difference for
richness (P-value=0.21). We next tested whether the lower value of these
community metrics translated into significant ORs for having an adenoma
or carcinoma. For fecal samples, the ORs for richness were not
significantly greater than 1.0 for adenoma or carcinoma (P-value=0.40)
{[}Figure 2A{]}. The ORs for evenness were significantly higher than 1.0
for adenoma (OR=1.3 (95\% Confidence Interval: 1.02 - 1.65),
P-value=0.035) and carcinoma (OR=1.66 (1.2 - 2.3), P-value=0.0021)
{[}Figure 2B{]}. The ORs for diversity were only significantly greater
than 1.0 for carcinoma (OR=1.61 (1.14 - 2.28), P-value=0.0069), but not
for adenoma (P-value=0.11) {[}Figure 2C{]}. Although these ORs are
significantly greater than 1.0, it is doubtful that they are clinically
meaningful.

Similar to our analysis of sequences obtained from fecal samples, we
repeated the analysis using sequences obtained from colon tissue. There
were no significant differences in richness, evenness, or diversity as
disease severity progressed from control to adenoma to carcinoma
(P-values \textgreater{} 0.05). We next analyzed the ORs, for matched
(i.e.~where unaffected tissue and tumors were obtained from the same
individual) and unmatched (i.e.~where unaffected tissue and tumor tissue
were not obtained from the same individual) tissue samples. The ORs for
adenoma and carcinoma were not significantly different from 1.0 for any
measure (P-values \textgreater{} 0.05) {[}Figure S1 \& Table S1{]}. This
is likely due to the combination of a small effect size and the
relatively small number of studies and size of studies used in the
analysis.

\textbf{\emph{Disease progression is associated with changes in
community structure.}} Based on the differences in evenness and
diversity, we next asked whether there were community-wide differences
in the structure of the communities associated with different disease
stages. We identified significant bacterial community differences in the
feces of patients with adenomas relative to those with normal colons in
1 of 4 studies and in patients with carcinomas relative to those with
normal colons in 6 of 7 studies (PERMANOVA; P-value \textless{} 0.05)
{[}Table S2{]}. Similar to the analyses using fecal samples, there were
significant differences in the bacterial community structure of subjects
with normal colons and those with adenomas (1 of 2 studies) and
carcinomas (1 of 3 studies) {[}Table S2{]}. For studies that used
matched samples, we did not observe any differences in bacterial
community structures {[}Table S2{]}. Combined, these results indicate
that there were consistent and significant community-wide changes in the
fecal community structure of subjects with carcinomas. However, the
signal observed in subjects with adenomas or when using tissue samples
was not as consistent. This is likely due to a smaller effect size or
the relatively small sample sizes among the studies that characterized
the tissue microbiota.

\textbf{\emph{Individual taxa are associated with significant ORs for
carcinomas.}} Next we identified those taxa that had ORs that were
significantly associated with having a normal colon or the presence of
adenomas or carcinomas. No taxa had a significant OR for the presence of
adenomas when we used data collected from fecal or tissue samples (Table
S3 \& S4). In contrast, 8 taxa had significant ORs for the presence of
carcinomas using data from fecal samples. Of these, 4 are commonly
associated with the oral cavity: \emph{Fusobacterium} (OR=2.74 (1.95 -
3.85)), \emph{Parvimonas} (OR=3.07 (2.11 - 4.46)), \emph{Porphyromonas}
(OR=3.2 (2.26 - 4.54)), and \emph{Peptostreptococcus} (OR=7.11 (3.84 -
13.17)) {[}Table S3{]}. The other 4 were \emph{Clostridium XI} (OR=0.65
(0.49 - 0.86)), \emph{Enterobacteriaceae} (OR=1.79 (1.33 - 2.41)),
\emph{Escherichia} (OR = 2.15 (1.57 - 2.95)), and \emph{Ruminococcus}
(OR=0.63 (0.48 - 0.83)). Among the data collected from tissue samples,
only unmatched carcinoma samples had taxa with a significant OR. Those
included \emph{Dorea} (OR=0.35 (0.22 - 0.55)), \emph{Blautia} (OR=0.47
(0.3 - 0.73)), and \emph{Weissella} (OR=5.15 (2.02 - 13.14)).
Mouth-associated genera were not significantly associated with a higher
OR for carcinoma in tissue samples {[}Table S4{]}. For example,
\emph{Fusobacterium} had an OR of 3.98 (1.19 - 13.24; however, due to
the small number of studies and considerable variation in the data, the
Benjimani-Hochberg corrected P-value was 0.93 {[}Table S4{]}. It is
interesting to note that \emph{Ruminococcus} and members of
\emph{Clostridium XI} in fecal samples and \emph{Dorea} and
\emph{Blautia} in tissue had ORs that were significantly less than 1.0,
which suggests that these populations are protective against the
development of carcinomas. Overall, there was no overlap in the taxa
with significant OR between fecal and tissue samples.

\textbf{\emph{Individual taxa with a significant OR do a poor job of
differentiating subjects with normal colons and those with carcinoma.}}
We next asked whether those taxa that had a significant OR associated
with having a normal colon or carcinomas could be used individually, to
classify subjects as having a normal colon or carcinomas. OR values were
caluclated based on whether the relative abundance for a taxon in a
subject was above or below the median relative abundance for that taxon
across all subjects in a study. To measure the ability of these taxa to
classify individuals we instead generated receiver operator
characteristic (ROC) curves for each taxon in each study and calculated
the area under the curve (AUC). This allowed us to use a more fluid
relative abundance threshold for classifying individuals by their
disease status. Using data from fecal samples, the 8 taxa did no better
at classifying the subjects than one would expect by chance
(i.e.~AUC=0.50) {[}Figure 3A{]}. The taxa that performed the best
included \emph{Clostridium XI}, \emph{Ruminococcus}, and
\emph{Escherichia}. However, these had median AUC values less than 0.588
ndicating their limited value as biomarkers when used individually.
Likewise, in unmatched tissue samples the 3 taxa with significant OR
taxa had AUC values that were marginally better than one would expect by
chance {[}Figure 3B{]}. The relative abundance of \emph{Dorea} was the
best predictor of carcinomas and its median AUC was only 0.62. These
results suggest that although these taxa are associated with a
significant OR for the presences of carcinomas, they do a poor job of
classifying a subject's disease status when used individually.

\textbf{\emph{Combined taxa model classifies subjects better than using
individual taxa.}} Instead of attempting to classify subjects based on
individual taxa, next we combined information from the individual taxa
and evaluated the ability to classify a subject's disease status using
Random Forest models. For data from fecal samples, the combined model
had an AUC of 0.75, which was significantly higher than any of the AUC
values for the individual taxa (P-value \textless{} 0.033). When this
approach was used to train models using data from each study, the most
important taxa were \emph{Ruminococcus} and \emph{Clostridium XI}
{[}Figure 4A{]}. Similarly, using data from the unmatched tissue
samples, the combined model had an AUC of 0.77, which was significantly
higher than the AUC values for classifying based on the relative
abundances of \emph{Blautia} and \emph{Weissella} individually (P-value
\textless{} 0.037). Both \emph{Dorea} and \emph{Blautia} were the most
important taxa in the tissue-based models {[}Figure 4B{]}. Pooling the
information from the taxa with significant ORs resulted in models that
outperformed classifications made using the same taxa individually.

\textbf{\emph{Performance of models based on taxa relative abundance in
full community is better than that of models based on taxa with
significant ORs.}} Next, we asked whether a Random Forest classification
model built using all of the taxa found in the communities would
outperform the models generated using those taxa with a significant OR.
Similar to our inability to identify taxa associated with a significant
OR for the presence of adenomas, the median AUCs to classify subjects as
having normal colons or having adenomas using data from fecal or tissue
samples were only marginally better than 0.5 for any study (median
AUC=0.549 (range: 0.367 - 0.971)) {[}Figure 5A \& S2A{]}. In contrast,
the models for classifying subjects as having normal colons or having
carcinomas using data from fecal or tissue samples yielded AUC values
meaningfully higher than 0.5 {[}Figure 5B \& S2B-C{]}. When we compared
the models based on all of the taxa in a community to models based on
the taxa with significant ORs, the results were mixed. Using the data
from fecal samples, we found that the AUC for 6 of 7 studies were an
average of 14.8\% higher and AUC for the Flemer study was 0.54\% lower
when using the relative abundance data from all taxa relative to using
the relative abundance of only the taxa with significant ORs. The
overall improvement in performance was statistically significant
(mean=12.61\%, one-tailed paired T-test; P-value=0.005). Among the
models trained using data from fecal samples, \emph{Bacteroides} and
\emph{Lachnospiraceae} were the most common taxa in the top 10\% mean
decrease in accuracy across studies {[}Figure S3{]}. Using data from
unmatched tissue samples to train classification models, we found that
the AUC of studies was an average 19.11\% higher when we used all of the
taxa rather than the 3 taxa with significant ORs (one-tailed paired
T-test; P-value=0.03). For the models trained using data from unmatched
tissue samples, \emph{Lachnospiraceae}, \emph{Bacteroidaceae}, and
\emph{Ruminococcaceae} were the most common taxa in the top 10\% mean
decrease in accuracy across studies {[}Figure S4{]}. Although the models
trained using those taxa with a significant OR perform well for
classifying individuals with and without carcinomas, models trained
using data from the full community perform better.

\textbf{\emph{Performance of models based on OTU relative abundances are
not significantly better than those based on taxa with significant
ORs.}} The previous models were based on relative abundance data where
sequences were classified to coarse taxonomic assignments
(i.e.~typically genus or family level). To determine whether model
performance improved with finer scale classification, we assigned
sequences to operational taxonomic units (OTUs) where the similarity
among sequences within an OTU was more than 97\%. We again found that
classification models built using all of the sequence data for a
community did a poor job of differentiating between subjects with normal
colons and those with adenomas (median AUC: 0.53 (0.37- 0.56)). However,
they did a good job of differentiating between subjects with normal
colons and those with carcinomas (median AUC: 0.71 (0.50- 0.90)). The
OTU-based models performed similarly to those constructed using the taxa
with significant ORs (one-tailed paired T-test; P-value=0.966) and those
using all taxa (one-tailed paired T-test; P-value=0.146) {[}Figure 4{]}.
Among the OTUs that had the highest mean decrease in accuracy for the
OTU-based models, we found that OTUs that affiliated with all of the 8
taxa that had a significant OR were within the top 10\% for at least one
study. This result was surprising as it indicated that a finer scale
classification of the sequences and thus a larger number of features to
select from, did not yield improved classification of the subjects.

\textbf{\emph{Generalizability of taxon-based models trained on one
dataset to the other datasets.}} Considering the good performance of the
Random Forest models trained using the relative abundance of taxa with
significant ORs and models trained using the relative abundance of all
taxa, we next asked how well the models would perform when given data
from a different cohort. For instance, if a model was trained using data
from the Ahn study, we wanted to know how well it would perform using
the data from the Baxter study. The models trained using the taxa with
significant ORs all had a higher median AUC than the models trained
using all of the taxa when tested on the other datasets {[}Figure 6 \&
S5{]}. As might be expected, the difference between the performance of
the modeling approaches appeared to vary with the size of the training
cohort (R\textsuperscript{2}=0.66) {[}Figure 6{]}. These data suggest
that given a sufficient number of subjects with normal colons and
carcinomas, Random Forest models trained using a small number of taxa
can accurately classify individuals from a different cohort.

\newpage

\subsection{Discussion}\label{discussion}

We performed a meta-analysis to identify and validate microbiota-base
biomarkers that could be used to classify individuals as having normal
colons or colonic tumors using fecal or tissue samples. To our surprise,
Random Forest classification models constructed to differentiate
individuals with normal colons from those with carcinomas using a subset
of the community performed well relative to models constructed using the
full communities. When we applied the models trained on each dataset to
the other datasets in our study, we found that the models trained using
the subset of the communities performed better than those using the full
communities. These models were trained using data in which sequences
were assigned to bacterial taxa using a classifier that typically
assigned sequences to the family or genus level. When we attempted to
improve the specificity of the classification by using an OTU-based
approach the resulting models performed as well as those constructed
using coarse taxonomic assignments. These results are significant
because they strengthen the growing literature indicating a role for the
colonic microbiota in tumorigenesis, as a potential tool as a
non-invasive diagnostic, and for assessing risk of disease and
recurrence (9, 12, 30).

Fine scale classification of sequences into OTUs did not improve our
classification models. This was also tested in earlier efforts to use
shotgun metagenomic data to classify individuals as having normal colons
or tumors; however, it was shown that analyses performed using shotgun
metagenomic data did not perform better than using 16S rRNA gene
sequencing data (31). We hypothesize that fine scale classification may
not result in better classification because distribution of microbiota
between individuals is patchy. In contrast, models using coarser
taxonomic assignments will pool the fine scale diversity, resulting in
less patchiness and better classification. Furthermore, the ability of
models trained using a subset of the community to outperform those using
the full community when testing the models on the other datasets may
also be a product of the patchiness of the human-associated microbiota.
The models based on the 8 taxa that had significant ORs used taxa that
were found in every study and tended to have higher relative abundances.
Similar to the OTU-based models, those models based on the full
community taxonomy assignments were still sensitive to the patchy
distribution of taxa. Regardless, it is encouraging that a collection of
8 taxa could reliably classify individuals as having carcinomas
considering the differences in cohorts, DNA extraction procedures,
regions of the 16S rRNA gene, and sequencing methods.

When used to classify individuals with carcinomas, the taxa with
significant ORs could not reliably classify individuals on their own
{[}Figure 3{]}. This result further supports the hypothesis that
carcinoma-associated microbiota have a patchy distribution. Two
individuals may have had the same classification, based on the relative
abundance of different populations within this group of 8 taxa. Although
these results only reflect associations with disease, it is tempting to
hypothesize that the patchiness is indicative of distinct mechanisms of
exacerbating tumorigenesis or that multiple taxa have the same mechanism
of exacerbating tumorigenesis. For example, strains of \emph{Escherichia
coli} and \emph{Fusobacterium nucleatum} have been shown to worsen
inflammation in mouse models of tumorigenesis (5, 6, 21). In contrast to
the patchiness of the taxa that were positively associated with
carcinomas, potentially beneficial taxa had a more consistent
association {[}Figure 6{]}. This result was particularly interesting
because members of these taxa (i.e. \emph{Ruminococcus} and
\emph{Clostridium XI} in fecal samples and \emph{Dorea} and
\emph{Blautia} in tissue) are thought to be beneficial due to their
involvement in production of anti-inflammatory short chain fatty acids
(32--34).

All of the adenoma classification models performed poorly, which is
consistent with previous studies (27, 30). However, the classification
results are at odds with results of the multitarget microbiota test
(MMT) from Baxter, et al. (12) who observed an AUC of 0.755 when the
test was applied to individuals with adenomas. There are two major
differences between the models generated in this meta-analysis and that
analysis. The MMT attempted to classify individuals as having a normal
colon or having colonic lesions (i.e.~adenomas or carcinomas) and not
adenomas alone. Further, the MMT incorporated fecal immunoglobulin test
(FIT) data while our models only used 16S rRNA gene sequencing data.
Because FIT data were not available for the other studies in our
meta-analysis, it was not possible to validate the MMT approach. The
ability to differentiate between individuals with and without adenomas
is an important problem since early detection of tumors is critical to
patient survivorship. However, it is possible that we might have been
able to detect differences in the bacterial community if individuals
with non-advanced and advanced adenomas were separated. This is a
clinically relevant distinction since advanced adenomas are at highest
risk of progressing to carcinomas. The initial changes of the microbiota
during tumorigenesis could be focal to where the initial adenoma
develops and would not be easily assessed using fecal samples from an
individual with non-advanced adenomas. Unfortunately, distinguishing
between individuals with advanced and non-advanced adenomas was not
possible in our meta-analysis since the studies did not provide the
clinical data needed to make that distinction.

Fecal samples represent a non-invasive approach to assess the structure
of the gut microbiota and are potentially useful for diagnosing
individuals as having colonic tumors. However, they do not reflect the
structure of the mucosal microbiota (35). Regardless, the taxa that were
the most important in the feces-based models overlapped with those from
the models trained using the data from unmatched and matched colon
tissue samples {[}Figure S3{]}. Mucosal biopsies are preferred for
focused mechanistic studies and have offered researchers the opportunity
to sample healthy and diseased tissue from the same individuals
(i.e.~matched) using each individual as their own control or in a
cross-sectional design (i.e.~unmatched). Because obtaining these samples
is invasive, carries risks to the individual, and is expensive, studies
investigating the structure of the mucosal microbiota generally have a
limited number of participants. Thus, it was not surprising that
tissue-based studies did not provide clearer associations between the
mucosal microbiota and the presence of tumors. Interestingly,
\emph{Fusobacterium}, which has received increased attention for its
potential role in tumorigenesis (6) was not consistently identified
across the studies in our meta-analysis. This could be due to the
relatively small number of individuals in the limited number of studies.
The classification models trained using the tissue-based data performed
well when tested with the training data (Figure S4), but performed
poorly when tested on the other tissue-associated datasets (Figure S5).
Disturbingly, taxa that are commonly associated with reagent
contamination (e.g. \emph{Novosphingobium}, \emph{Acidobacteria Gp2},
\emph{Sphingomonas}, etc.) were detected within the tissue datasets.
Such contamination is common in studies where there is relatively low
bacterial biomass (36). The lack of replication among the tissue-based
biomarkers may be a product of the relatively small number of studies
and individuals per study and possible reagent contamination.

Among the fecal sample data, we failed to identify several notable
populations that are commonly associated with carcinomas including an
enterotoxigenic strain of \emph{Bacteroides fragilis} (ETBF) and
\emph{Streptococcus gallolyticus} subsp. \emph{gallolyticus} (22, 24).
ETBF have been found in tumors in the proximal colon where they tend to
form biofilms (20, 37). Considering DNA from bacteria that are more
prevalent in the proximal colon may be degraded by the time it leaves
the body, it is not surprising that we failed to identify a significant
OR for \emph{Bacteroides} with carcinomas. In addition, since our
approach could only classify sequences to the genus level and there are
likely multiple \emph{Bacteroides} populations in the colon, it is
possible that sequences from ETBF and non-oncogenic \emph{Bacteroides}
were pooled. This would then reduce the OR between \emph{Bacterioides}
and whether an individual had carcinomas. It is also necessary to
distinguish between populations that are biomarkers for a disease and
those that are known to cause disease. Although the latter have been
shown to have a causative role, they may appear at low relative
abundance, be found in specific locations, or may have a highly patchy
distribution among affected individuals.

Meta-analyses are a useful tool in microbiome research because they can
demonstrate whether a result can be replicated and facilitate new
discoveries by pooling multiple independent investigations. There have
been several meta-analyses similar to this study that have sought
biomarkers for obesity (38--40), inflammatory bowel disease (39), and
colorectal cancer (28). Considering microbiome research is particularly
prone to hype and overgeneralization of results (41), these analyses are
critical. Meta-analyses are difficult to perform because the underlying
16S rRNA gene sequence data are not publicly available, metadata are
missing, incomplete, or vague, sequence data are of poor quality or
derived by non-standard approaches, and the original studies may be
significantly underpowered. Reluctance to publish negative results
(i.e.~the ``file drawer effect'') is also likely to skew our
understanding of the relationship between microbiota and disease. Better
attention to these specific issues will increase the reproducibility and
replicability of microbiota studies and make it easier to perform these
crucial meta-analyses. Moving forward, meta-analyses will be important
tools to help aggregate and find commonalities across studies when
investigating the microbiota in the context of a specific disease (28,
38--40).

Our meta-analysis suggests a strong association between the gut
microbiota and colon tumorigenesis. By aggregating the results from
studies that sequenced the 16S rRNA gene from fecal and tissue samples,
we are able to provide evidence supporting the use of microbial
biomarkers to diagnose the presence of colonic tumors. Further
development of microbial biomarkers should focus on including other
biomarkers (e.g.~FIT), better categorizing of people with adenomas, and
expanding datasets to include larger numbers of individuals. Based on
prior research into the physiology of the biomarkers we identified, it
is likely that they have a causative role in tumorigenesis. Their patchy
distribution across individuals suggests that there are either multiple
mechanisms causing disease or a single mechanism (e.g.~inflammation)
that can be mediated by multiple, diverse bacteria.

\newpage

\subsection{Methods}\label{methods}

\textbf{\emph{Datasets.}} The studies used for this meta-analysis were
identified through the review articles written by Keku, et al. (42) and
Vogtmann, et al. (43). Additional studies, not mentioned in those
reviews were obtained based on the authors' knowledge of the literature.
Studies were included that used tissue or feces as their sample source
for 454 or Illumina 16S rRNA gene sequencing. A significant number of
studies (N=12) were excluded from the meta-analysis because they did not
have publicly available sequences, did not use 454 or Illumina
sequencing platforms, or did not have metadata that the authors were
able to share. We were able to obtain sequence data and metadata from
the following studies: Ahn, et al. (11), Baxter, et al. (12), Brim, et
al. (29), Burns, et al. (15), Chen, et al. (13), Dejea, et al. (20),
Flemer, et al. (17), Geng, et al. (19), Hale, et al. (27), Kostic, et
al. (44), Lu, et al. (26), Sanapareddy, et al. (25), Wang, et al. (14),
Weir, et al. (23), and Zeller, et al. (16). The Zackular (45) study was
excluded because the individuals studied were included within the larger
Baxter study (12). The Kostic study was excluded because after we
processed the sequences, all of the case samples had 100 or fewer
sequences. The final analysis included 14 studies (Tables 1 and 2).
There were seven studies with only fecal samples (Ahn, Baxter, Brim,
Hale, Wang, Weir, and Zeller), five studies with only tissue samples
(Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal
and tissue samples (Chen and Flemer). After curating the sequences, 1737
fecal samples and 492 tissue samples remained in the analysis {[}Tables
1 and 2{]}.

\textbf{\emph{Sequence Processing.}} Raw sequence data and metadata were
primarily obtained from the Sequence Read Archive (SRA) and dbGaP. Other
sequence and metadata were obtained directly from the authors (n=4, (17,
23, 25, 27)). Each dataset was processed separately using mothur
(v1.39.3) using the default quality filtering methods for both 454 and
Illumina sequence data (46). If it was not possible to use the defaults
because the trimmed sequences were too short, then the stated quality
cut-offs from the original study were used. Chimeric sequences were
identified and removed using VSEARCH (47). The curated sequences were
assigned to OTUs at 97\% similarity using the OptiClust algorithm (48)
and classified to the deepest taxonomic level that had 80\% support
using the naïve Bayesian classifier trained on the RDP taxonomy outline
(version 14, (49)).

\textbf{\emph{Community analysis.}} We calculated alpha diversity
metrics (i.e.~OTU richness, evenness, and Shannon diversity) for each
sample. Within each dataset, we ensured that the data followed a normal
distribution using power transformations. Using the transformed data, we
tested the hypothesis that individuals with normal colons, adenomas, and
carcinomas had significantly different alpha diversity metrics using
linear mixed-effect models. We also calculated the OR for each study and
metric by considering any value above the median alpha diversity value
to be positive. We measured the dissimilarity between individuals by
calculating the pairwise Bray-Curtis index and used PERMANOVA (50) to
test whether individuals with normal colons were significantly different
from those with adenomas or carcinomas. Finally, after binning sequences
into the deepest taxa that the naïve Bayesian classifier could calssify
the sequences, we quantified the ORs for individuals having an adenoma
or carcinoma and corrected for multiple comparisons using the
Benjamini-Hochberg method (51). Again, for each taxon, if the relative
abundance was greater than the median relative abundance for that taxon
in the study, the individual was considered to be positive.

\textbf{\emph{Random Forest classification analysis.}} To classify
individuals as having normal colons or tumors, we built Random Forest
classification models for each dataset and comparison using taxa with
significant ORs (after multiple comparison correction), all taxa, or
OTUs. Because no taxa were identified as having a significant OR
associated with adenomas using stool samples or tissue samples,
classification models based on OR data were not constructed to classify
individuals as having normal colons or adenomas. Within the training
dataset, 10-fold cross validation (5-fold cross validation for small
datasets) was used to build a model that was then evaluated on the
testing set. For the models constructed using the taxa with significant
ORs, the default mtry setting was used to train the model and this model
was tested on the other datasets in the meta-analysis. The reported AUC
values are the AUCs for the application of the model on the test sets.
For the OTU-based models, the dataset was split into training (80\% of
samples) and testing (20\%) sets and 10-fold cross validation (5-fold
cross validation for small datasets) on the training set was used to
generate the model for the testing set. The original 80/20 split and
fitting was repeated 100 times and the average AUC from these 100
repeats was reported. The Mean Decrease in Accuracy (MDA), a measure of
the importance of each taxon to the overall model was used to rank the
taxa used in each model. For all models, the default setting used was
\(\sqrt{p}\), where p is all the variables used in the respective model.
Normally, \(\sqrt{p}\) has been found to be what is chosen as the ideal
mtry after model tuneing (52).

\textbf{\emph{Statistical Analysis.}} All statistical analysis after
sequence processing utilized the R (v3.4.3) software package (53). For
OTU richness, evenness, and Shannon diversity analysis, values were
power transformed using the rcompanion (v1.11.1) package (54) and
Z-score normalized using the car (v2.1.6) package (55). Testing for OTU
richness, evenness, and Shannon diversity differences utilized linear
mixed-effect models to correct for study, repeat sampling of individuals
(tissue only), and 16S rRNA gene sequence region used using the lme4
(v1.1.15) package (56). ORs were analyzed using both the epiR (v0.9.93)
and metafor (v2.0.0) packages (57, 58) by assessing how many individuals
with and without disease were above and below the overall median value
within each specific study. OR significance testing utilized the
chi-squared test. Community structure differences were calculated using
the Bray-Curtis dissimilarity index and PERMANOVA was used to test for
tumor-associated differences in structure with the vegan (v2.4.5)
package (59). Random Forest models were built using both the caret
(v6.0.78) and randomForest (v4.6.12) packages (60, 61). All figures were
created using both ggplot2 (v2.2.1) and gridExtra (v2.3) packages (62,
63).

\textbf{\emph{Reproducible Methods.}} The analysis code can be found at
\url{https://github.com/SchlossLab/Sze_CRCMetaAnalysis_mBio_2017}.
Unless otherwise mentioned, the accession number of raw sequences from
the studies used in this analysis can be found directly in the
respective batch file in the GitHub repository or in the original
manuscript.

\newpage

\subsubsection{Acknowledgements}\label{acknowledgements}

The authors would like to thank all the study participants who were a
part of each of the individual studies analyzed. We would also like to
thank each of the study authors for making their sequencing reads and
metadata available for use. Finally, we would like to thank the members
of the Schloss lab for their valuable feedback and proofreading during
the formulation of this manuscript.

\newpage

\subsection{References}\label{references}

\hypertarget{refs}{}
\hypertarget{ref-siegel_cancer_2016}{}
1. \textbf{Siegel, R. L.}, \textbf{K. D. Miller}, and \textbf{A. Jemal}.
2016. Cancer statistics, 2016. CA: a cancer journal for clinicians
\textbf{66}:7--30.

\hypertarget{ref-flynn_metabolic_2016}{}
2. \textbf{Flynn, K. J.}, \textbf{N. T. Baxter}, and \textbf{P. D.
Schloss}. 2016. Metabolic and Community Synergy of Oral Bacteria in
Colorectal Cancer. mSphere \textbf{1}.

\hypertarget{ref-goodwin_polyamine_2011}{}
3. \textbf{Goodwin, A. C.}, \textbf{C. E. Destefano Shields}, \textbf{S.
Wu}, \textbf{D. L. Huso}, \textbf{X. Wu}, \textbf{T. R. Murray-Stewart},
\textbf{A. Hacker-Prietz}, \textbf{S. Rabizadeh}, \textbf{P. M. Woster},
\textbf{C. L. Sears}, and \textbf{R. A. Casero}. 2011. Polyamine
catabolism contributes to enterotoxigenic Bacteroides fragilis-induced
colon tumorigenesis. Proceedings of the National Academy of Sciences of
the United States of America \textbf{108}:15354--15359.

\hypertarget{ref-abed_fap2_2016}{}
4. \textbf{Abed, J.}, \textbf{J. E. M. Emgård}, \textbf{G. Zamir},
\textbf{M. Faroja}, \textbf{G. Almogy}, \textbf{A. Grenov}, \textbf{A.
Sol}, \textbf{R. Naor}, \textbf{E. Pikarsky}, \textbf{K. A. Atlan},
\textbf{A. Mellul}, \textbf{S. Chaushu}, \textbf{A. L. Manson},
\textbf{A. M. Earl}, \textbf{N. Ou}, \textbf{C. A. Brennan}, \textbf{W.
S. Garrett}, and \textbf{G. Bachrach}. 2016. Fap2 Mediates Fusobacterium
nucleatum Colorectal Adenocarcinoma Enrichment by Binding to
Tumor-Expressed Gal-GalNAc. Cell Host \& Microbe \textbf{20}:215--225.

\hypertarget{ref-arthur_intestinal_2012}{}
5. \textbf{Arthur, J. C.}, \textbf{E. Perez-Chanona}, \textbf{M.
Mühlbauer}, \textbf{S. Tomkovich}, \textbf{J. M. Uronis}, \textbf{T.-J.
Fan}, \textbf{B. J. Campbell}, \textbf{T. Abujamel}, \textbf{B. Dogan},
\textbf{A. B. Rogers}, \textbf{J. M. Rhodes}, \textbf{A. Stintzi},
\textbf{K. W. Simpson}, \textbf{J. J. Hansen}, \textbf{T. O. Keku},
\textbf{A. A. Fodor}, and \textbf{C. Jobin}. 2012. Intestinal
inflammation targets cancer-inducing activity of the microbiota. Science
(New York, N.Y.) \textbf{338}:120--123.

\hypertarget{ref-kostic_fusobacterium_2013}{}
6. \textbf{Kostic, A. D.}, \textbf{E. Chun}, \textbf{L. Robertson},
\textbf{J. N. Glickman}, \textbf{C. A. Gallini}, \textbf{M. Michaud},
\textbf{T. E. Clancy}, \textbf{D. C. Chung}, \textbf{P. Lochhead},
\textbf{G. L. Hold}, \textbf{E. M. El-Omar}, \textbf{D. Brenner},
\textbf{C. S. Fuchs}, \textbf{M. Meyerson}, and \textbf{W. S. Garrett}.
2013. Fusobacterium nucleatum potentiates intestinal tumorigenesis and
modulates the tumor-immune microenvironment. Cell Host \& Microbe
\textbf{14}:207--215.

\hypertarget{ref-wu_human_2009}{}
7. \textbf{Wu, S.}, \textbf{K.-J. Rhee}, \textbf{E. Albesiano},
\textbf{S. Rabizadeh}, \textbf{X. Wu}, \textbf{H.-R. Yen}, \textbf{D. L.
Huso}, \textbf{F. L. Brancati}, \textbf{E. Wick}, \textbf{F.
McAllister}, \textbf{F. Housseau}, \textbf{D. M. Pardoll}, and
\textbf{C. L. Sears}. 2009. A human colonic commensal promotes colon
tumorigenesis via activation of T helper type 17 T cell responses.
Nature Medicine \textbf{15}:1016--1022.

\hypertarget{ref-zackular_manipulation_2016}{}
8. \textbf{Zackular, J. P.}, \textbf{N. T. Baxter}, \textbf{G. Y. Chen},
and \textbf{P. D. Schloss}. 2016. Manipulation of the Gut Microbiota
Reveals Role in Colon Tumorigenesis. mSphere \textbf{1}.

\hypertarget{ref-zackular_gut_2013}{}
9. \textbf{Zackular, J. P.}, \textbf{N. T. Baxter}, \textbf{K. D.
Iverson}, \textbf{W. D. Sadler}, \textbf{J. F. Petrosino}, \textbf{G. Y.
Chen}, and \textbf{P. D. Schloss}. 2013. The gut microbiome modulates
colon tumorigenesis. mBio \textbf{4}:e00692--00613.

\hypertarget{ref-baxter_structure_2014}{}
10. \textbf{Baxter, N. T.}, \textbf{J. P. Zackular}, \textbf{G. Y.
Chen}, and \textbf{P. D. Schloss}. 2014. Structure of the gut microbiome
following colonization with human feces determines colonic tumor burden.
Microbiome \textbf{2}:20.

\hypertarget{ref-ahn_human_2013}{}
11. \textbf{Ahn, J.}, \textbf{R. Sinha}, \textbf{Z. Pei}, \textbf{C.
Dominianni}, \textbf{J. Wu}, \textbf{J. Shi}, \textbf{J. J. Goedert},
\textbf{R. B. Hayes}, and \textbf{L. Yang}. 2013. Human gut microbiome
and risk for colorectal cancer. Journal of the National Cancer Institute
\textbf{105}:1907--1911.

\hypertarget{ref-baxter_microbiota-based_2016}{}
12. \textbf{Baxter, N. T.}, \textbf{M. T. Ruffin}, \textbf{M. A. M.
Rogers}, and \textbf{P. D. Schloss}. 2016. Microbiota-based model
improves the sensitivity of fecal immunochemical test for detecting
colonic lesions. Genome Medicine \textbf{8}:37.

\hypertarget{ref-chen_human_2012}{}
13. \textbf{Chen, W.}, \textbf{F. Liu}, \textbf{Z. Ling}, \textbf{X.
Tong}, and \textbf{C. Xiang}. 2012. Human intestinal lumen and
mucosa-associated microbiota in patients with colorectal cancer. PloS
One \textbf{7}:e39743.

\hypertarget{ref-wang_structural_2012}{}
14. \textbf{Wang, T.}, \textbf{G. Cai}, \textbf{Y. Qiu}, \textbf{N.
Fei}, \textbf{M. Zhang}, \textbf{X. Pang}, \textbf{W. Jia}, \textbf{S.
Cai}, and \textbf{L. Zhao}. 2012. Structural segregation of gut
microbiota between colorectal cancer patients and healthy volunteers.
The ISME journal \textbf{6}:320--329.

\hypertarget{ref-burns_virulence_2015}{}
15. \textbf{Burns, M. B.}, \textbf{J. Lynch}, \textbf{T. K. Starr},
\textbf{D. Knights}, and \textbf{R. Blekhman}. 2015. Virulence genes are
a signature of the microbiome in the colorectal tumor microenvironment.
Genome Medicine \textbf{7}:55.

\hypertarget{ref-zeller_potential_2014}{}
16. \textbf{Zeller, G.}, \textbf{J. Tap}, \textbf{A. Y. Voigt},
\textbf{S. Sunagawa}, \textbf{J. R. Kultima}, \textbf{P. I. Costea},
\textbf{A. Amiot}, \textbf{J. Böhm}, \textbf{F. Brunetti}, \textbf{N.
Habermann}, \textbf{R. Hercog}, \textbf{M. Koch}, \textbf{A. Luciani},
\textbf{D. R. Mende}, \textbf{M. A. Schneider}, \textbf{P.
Schrotz-King}, \textbf{C. Tournigand}, \textbf{J. Tran Van Nhieu},
\textbf{T. Yamada}, \textbf{J. Zimmermann}, \textbf{V. Benes},
\textbf{M. Kloor}, \textbf{C. M. Ulrich}, \textbf{M. von Knebel
Doeberitz}, \textbf{I. Sobhani}, and \textbf{P. Bork}. 2014. Potential
of fecal microbiota for early-stage detection of colorectal cancer.
Molecular Systems Biology \textbf{10}:766.

\hypertarget{ref-flemer_tumour-associated_2017}{}
17. \textbf{Flemer, B.}, \textbf{D. B. Lynch}, \textbf{J. M. R. Brown},
\textbf{I. B. Jeffery}, \textbf{F. J. Ryan}, \textbf{M. J. Claesson},
\textbf{M. O'Riordain}, \textbf{F. Shanahan}, and \textbf{P. W.
O'Toole}. 2017. Tumour-associated and non-tumour-associated microbiota
in colorectal cancer. Gut \textbf{66}:633--643.

\hypertarget{ref-GimenoGarca2012}{}
18. \textbf{García, A. Z. G.} 2012. Factors influencing colorectal
cancer screening participation. Gastroenterology Research and Practice.
Hindawi Limited \textbf{2012}:1--8.

\hypertarget{ref-geng_diversified_2013}{}
19. \textbf{Geng, J.}, \textbf{H. Fan}, \textbf{X. Tang}, \textbf{H.
Zhai}, and \textbf{Z. Zhang}. 2013. Diversified pattern of the human
colorectal cancer microbiome. Gut Pathogens \textbf{5}:2.

\hypertarget{ref-dejea_microbiota_2014}{}
20. \textbf{Dejea, C. M.}, \textbf{E. C. Wick}, \textbf{E. M.
Hechenbleikner}, \textbf{J. R. White}, \textbf{J. L. Mark Welch},
\textbf{B. J. Rossetti}, \textbf{S. N. Peterson}, \textbf{E. C.
Snesrud}, \textbf{G. G. Borisy}, \textbf{M. Lazarev}, \textbf{E. Stein},
\textbf{J. Vadivelu}, \textbf{A. C. Roslani}, \textbf{A. A. Malik},
\textbf{J. W. Wanyiri}, \textbf{K. L. Goh}, \textbf{I. Thevambiga},
\textbf{K. Fu}, \textbf{F. Wan}, \textbf{N. Llosa}, \textbf{F.
Housseau}, \textbf{K. Romans}, \textbf{X. Wu}, \textbf{F. M.
McAllister}, \textbf{S. Wu}, \textbf{B. Vogelstein}, \textbf{K. W.
Kinzler}, \textbf{D. M. Pardoll}, and \textbf{C. L. Sears}. 2014.
Microbiota organization is a distinct feature of proximal colorectal
cancers. Proceedings of the National Academy of Sciences of the United
States of America \textbf{111}:18321--18326.

\hypertarget{ref-ecoli_Arthur_2014}{}
21. \textbf{Arthur, J. C.}, \textbf{R. Z. Gharaibeh}, \textbf{M.
Mühlbauer}, \textbf{E. Perez-Chanona}, \textbf{J. M. Uronis}, \textbf{J.
McCafferty}, \textbf{A. A. Fodor}, and \textbf{C. Jobin}. 2014.
Microbial genomic analysis reveals the essential role of inflammation in
bacteria-induced colorectal cancer. Nature Communications. Springer
Nature \textbf{5}:4724.

\hypertarget{ref-strep_Aymeric_2017}{}
22. \textbf{Aymeric, L.}, \textbf{F. Donnadieu}, \textbf{C. Mulet},
\textbf{L. du Merle}, \textbf{G. Nigro}, \textbf{A. Saffarian},
\textbf{M. Bérard}, \textbf{C. Poyart}, \textbf{S. Robine}, \textbf{B.
Regnault}, \textbf{P. Trieu-Cuot}, \textbf{P. J. Sansonetti}, and
\textbf{S. Dramsi}. 2017. Colorectal cancer specific conditions
promoteStreptococcus gallolyticusgut colonization. Proceedings of the
National Academy of Sciences. Proceedings of the National Academy of
Sciences \textbf{115}:E283--E291.

\hypertarget{ref-weir_stool_2013}{}
23. \textbf{Weir, T. L.}, \textbf{D. K. Manter}, \textbf{A. M. Sheflin},
\textbf{B. A. Barnett}, \textbf{A. L. Heuberger}, and \textbf{E. P.
Ryan}. 2013. Stool microbiome and metabolome differences between
colorectal cancer patients and healthy adults. PloS One
\textbf{8}:e70803.

\hypertarget{ref-bfrag_Boleij_2014}{}
24. \textbf{Boleij, A.}, \textbf{E. M. Hechenbleikner}, \textbf{A. C.
Goodwin}, \textbf{R. Badani}, \textbf{E. M. Stein}, \textbf{M. G.
Lazarev}, \textbf{B. Ellis}, \textbf{K. C. Carroll}, \textbf{E.
Albesiano}, \textbf{E. C. Wick}, \textbf{E. A. Platz}, \textbf{D. M.
Pardoll}, and \textbf{C. L. Sears}. 2014. The bacteroides fragilis toxin
gene is prevalent in the colon mucosa of colorectal cancer patients.
Clinical Infectious Diseases. Oxford University Press (OUP)
\textbf{60}:208--215.

\hypertarget{ref-sanapareddy_increased_2012}{}
25. \textbf{Sanapareddy, N.}, \textbf{R. M. Legge}, \textbf{B. Jovov},
\textbf{A. McCoy}, \textbf{L. Burcal}, \textbf{F. Araujo-Perez},
\textbf{T. A. Randall}, \textbf{J. Galanko}, \textbf{A. Benson},
\textbf{R. S. Sandler}, \textbf{J. F. Rawls}, \textbf{Z. Abdo},
\textbf{A. A. Fodor}, and \textbf{T. O. Keku}. 2012. Increased rectal
microbial richness is associated with the presence of colorectal
adenomas in humans. The ISME journal \textbf{6}:1858--1868.

\hypertarget{ref-lu_mucosal_2016}{}
26. \textbf{Lu, Y.}, \textbf{J. Chen}, \textbf{J. Zheng}, \textbf{G.
Hu}, \textbf{J. Wang}, \textbf{C. Huang}, \textbf{L. Lou}, \textbf{X.
Wang}, and \textbf{Y. Zeng}. 2016. Mucosal adherent bacterial dysbiosis
in patients with colorectal adenomas. Scientific Reports
\textbf{6}:26337.

\hypertarget{ref-hale_shifts_2017}{}
27. \textbf{Hale, V. L.}, \textbf{J. Chen}, \textbf{S. Johnson},
\textbf{S. C. Harrington}, \textbf{T. C. Yab}, \textbf{T. C. Smyrk},
\textbf{H. Nelson}, \textbf{L. A. Boardman}, \textbf{B. R. Druliner},
\textbf{T. R. Levin}, \textbf{D. K. Rex}, \textbf{D. J. Ahnen},
\textbf{P. Lance}, \textbf{D. A. Ahlquist}, and \textbf{N. Chia}. 2017.
Shifts in the Fecal Microbiota Associated with Adenomatous Polyps.
Cancer Epidemiology, Biomarkers \& Prevention: A Publication of the
American Association for Cancer Research, Cosponsored by the American
Society of Preventive Oncology \textbf{26}:85--94.

\hypertarget{ref-shah_leveraging_2017}{}
28. \textbf{Shah, M. S.}, \textbf{T. Z. DeSantis}, \textbf{T.
Weinmaier}, \textbf{P. J. McMurdie}, \textbf{J. L. Cope}, \textbf{A.
Altrichter}, \textbf{J.-M. Yamal}, and \textbf{E. B. Hollister}. 2017.
Leveraging sequence-based faecal microbial community survey data to
identify a composite biomarker for colorectal cancer. Gut.

\hypertarget{ref-brim_microbiome_2013}{}
29. \textbf{Brim, H.}, \textbf{S. Yooseph}, \textbf{E. G. Zoetendal},
\textbf{E. Lee}, \textbf{M. Torralbo}, \textbf{A. O. Laiyemo},
\textbf{B. Shokrani}, \textbf{K. Nelson}, and \textbf{H. Ashktorab}.
2013. Microbiome analysis of stool samples from African Americans with
colon polyps. PloS One \textbf{8}:e81352.

\hypertarget{ref-Sze2017}{}
30. \textbf{Sze, M. A.}, \textbf{N. T. Baxter}, \textbf{M. T. Ruffin},
\textbf{M. A. M. Rogers}, and \textbf{P. D. Schloss}. 2017.
Normalization of the microbiota in patients after treatment for colonic
lesions. Microbiome. Springer Nature \textbf{5}.

\hypertarget{ref-Hannigan2017}{}
31. \textbf{Hannigan, G. D.}, \textbf{M. B. Duhaime}, \textbf{M. T.
Ruffin}, \textbf{C. C. Koumpouras}, and \textbf{P. D. Schloss}. 2017.
Diagnostic potential \& the interactive dynamics of the colorectal
cancer virome. Cold Spring Harbor Laboratory.

\hypertarget{ref-Venkataraman2016}{}
32. \textbf{Venkataraman, A.}, \textbf{J. R. Sieber}, \textbf{A. W.
Schmidt}, \textbf{C. Waldron}, \textbf{K. R. Theis}, and \textbf{T. M.
Schmidt}. 2016. Variable responses of human microbiomes to dietary
supplementation with resistant starch. Microbiome. Springer Nature
\textbf{4}.

\hypertarget{ref-Herrmann2018}{}
33. \textbf{Herrmann, E.}, \textbf{W. Young}, \textbf{V.
Reichert-Grimm}, \textbf{S. Weis}, \textbf{C. Riedel}, \textbf{D.
Rosendale}, \textbf{H. Stoklosinski}, \textbf{M. Hunt}, and \textbf{M.
Egert}. 2018. In vivo assessment of resistant starch degradation by the
caecal microbiota of mice using RNA-based stable isotope probingA
proof-of-principle study. Nutrients. MDPI AG \textbf{10}:179.

\hypertarget{ref-Reichardt2017}{}
34. \textbf{Reichardt, N.}, \textbf{M. Vollmer}, \textbf{G. Holtrop},
\textbf{F. M. Farquharson}, \textbf{D. Wefers}, \textbf{M. Bunzel},
\textbf{S. H. Duncan}, \textbf{J. E. Drew}, \textbf{L. M. Williams},
\textbf{G. Milligan}, \textbf{T. Preston}, \textbf{D. Morrison},
\textbf{H. J. Flint}, and \textbf{P. Louis}. 2017. Specific
substrate-driven changes in human faecal microbiota composition contrast
with functional redundancy in short-chain fatty acid production. The
ISME Journal. Springer Nature \textbf{12}:610--622.

\hypertarget{ref-Flynn_preprint_2017}{}
35. \textbf{Flynn, K. J.}, \textbf{M. T. Ruffin}, \textbf{D. K.
Turgeon}, and \textbf{P. D. Schloss}. 2017. Spatial variation of the
native colon microbiota in healthy adults. Cold Spring Harbor
Laboratory.

\hypertarget{ref-Salter_contamination_2014}{}
36. \textbf{Salter, S. J.}, \textbf{M. J. Cox}, \textbf{E. M. Turek},
\textbf{S. T. Calus}, \textbf{W. O. Cookson}, \textbf{M. F. Moffatt},
\textbf{P. Turner}, \textbf{J. Parkhill}, \textbf{N. J. Loman}, and
\textbf{A. W. Walker}. 2014. Reagent and laboratory contamination can
critically impact sequence-based microbiome analyses. BMC Biology.
Springer Nature \textbf{12}.

\hypertarget{ref-Purcell2017}{}
37. \textbf{Purcell, R. V.}, \textbf{J. Pearson}, \textbf{A. Aitchison},
\textbf{L. Dixon}, \textbf{F. A. Frizelle}, and \textbf{J. I. Keenan}.
2017. Colonization with enterotoxigenic bacteroides fragilis is
associated with early-stage colorectal neoplasia. PLOS ONE. Public
Library of Science (PLoS) \textbf{12}:e0171602.

\hypertarget{ref-Sze2016}{}
38. \textbf{Sze, M. A.}, and \textbf{P. D. Schloss}. 2016. Looking for a
signal in the noise: Revisiting obesity and the microbiome. mBio.
American Society for Microbiology \textbf{7}:e01018--16.

\hypertarget{ref-Walters2014}{}
39. \textbf{Walters, W. A.}, \textbf{Z. Xu}, and \textbf{R. Knight}.
2014. Meta-analyses of human gut microbes associated with obesity and
IBD. FEBS Letters. Wiley-Blackwell \textbf{588}:4223--4233.

\hypertarget{ref-Finucane2014}{}
40. \textbf{Finucane, M. M.}, \textbf{T. J. Sharpton}, \textbf{T. J.
Laurent}, and \textbf{K. S. Pollard}. 2014. A taxonomic signature of
obesity in the microbiome? Getting to the guts of the matter. PLoS ONE.
Public Library of Science (PLoS) \textbf{9}:e84689.

\hypertarget{ref-Hanage2014}{}
41. \textbf{Hanage, W. P.} 2014. Microbiology: Microbiome science needs
a healthy dose of scepticism. Nature. Springer Nature
\textbf{512}:247--248.

\hypertarget{ref-keku_gastrointestinal_2015}{}
42. \textbf{Keku, T. O.}, \textbf{S. Dulal}, \textbf{A. Deveaux},
\textbf{B. Jovov}, and \textbf{X. Han}. 2015. The gastrointestinal
microbiota and colorectal cancer. American Journal of Physiology -
Gastrointestinal and Liver Physiology \textbf{308}:G351--G363.

\hypertarget{ref-vogtmann_epidemiologic_2016}{}
43. \textbf{Vogtmann, E.}, and \textbf{J. J. Goedert}. 2016.
Epidemiologic studies of the human microbiome and cancer. British
Journal of Cancer \textbf{114}:237--242.

\hypertarget{ref-kostic_genomic_2012}{}
44. \textbf{Kostic, A. D.}, \textbf{D. Gevers}, \textbf{C. S.
Pedamallu}, \textbf{M. Michaud}, \textbf{F. Duke}, \textbf{A. M. Earl},
\textbf{A. I. Ojesina}, \textbf{J. Jung}, \textbf{A. J. Bass},
\textbf{J. Tabernero}, \textbf{J. Baselga}, \textbf{C. Liu}, \textbf{R.
A. Shivdasani}, \textbf{S. Ogino}, \textbf{B. W. Birren}, \textbf{C.
Huttenhower}, \textbf{W. S. Garrett}, and \textbf{M. Meyerson}. 2012.
Genomic analysis identifies association of Fusobacterium with colorectal
carcinoma. Genome Research \textbf{22}:292--298.

\hypertarget{ref-zackular_human_2014}{}
45. \textbf{Zackular, J. P.}, \textbf{M. A. M. Rogers}, \textbf{M. T.
Ruffin}, and \textbf{P. D. Schloss}. 2014. The human gut microbiome as a
screening tool for colorectal cancer. Cancer Prevention Research
(Philadelphia, Pa.) \textbf{7}:1112--1121.

\hypertarget{ref-schloss_introducing_2009}{}
46. \textbf{Schloss, P. D.}, \textbf{S. L. Westcott}, \textbf{T.
Ryabin}, \textbf{J. R. Hall}, \textbf{M. Hartmann}, \textbf{E. B.
Hollister}, \textbf{R. A. Lesniewski}, \textbf{B. B. Oakley}, \textbf{D.
H. Parks}, \textbf{C. J. Robinson}, \textbf{J. W. Sahl}, \textbf{B.
Stres}, \textbf{G. G. Thallinger}, \textbf{D. J. Van Horn}, and
\textbf{C. F. Weber}. 2009. Introducing mothur: Open-Source,
Platform-Independent, Community-Supported Software for Describing and
Comparing Microbial Communities. Appl.Environ.Microbiol.
\textbf{75}:7537--7541.

\hypertarget{ref-rognes_vsearch_2016}{}
47. \textbf{Rognes, T.}, \textbf{T. Flouri}, \textbf{B. Nichols},
\textbf{C. Quince}, and \textbf{F. Mahé}. 2016. VSEARCH: A versatile
open source tool for metagenomics. PeerJ \textbf{4}:e2584.

\hypertarget{ref-westcott_opticlust_2017}{}
48. \textbf{Westcott, S. L.}, and \textbf{P. D. Schloss}. 2017.
OptiClust, an Improved Method for Assigning Amplicon-Based Sequence Data
to Operational Taxonomic Units. mSphere \textbf{2}.

\hypertarget{ref-rdp_Wang2007}{}
49. \textbf{Wang, Q.}, \textbf{G. M. Garrity}, \textbf{J. M. Tiedje},
and \textbf{J. R. Cole}. 2007. Naive bayesian classifier for rapid
assignment of rRNA sequences into the new bacterial taxonomy. Applied
and Environmental Microbiology. American Society for Microbiology
\textbf{73}:5261--5267.

\hypertarget{ref-permanova_Anderson2013}{}
50. \textbf{Anderson, M. J.}, and \textbf{D. C. I. Walsh}. 2013.
PERMANOVA, ANOSIM, and the mantel test in the face of heterogeneous
dispersions: What null hypothesis are you testing? Ecological
Monographs. Wiley-Blackwell \textbf{83}:557--574.

\hypertarget{ref-benjamini_controlling_1995}{}
51. \textbf{Benjamini, Y.}, and \textbf{Y. Hochberg}. 1995. Controlling
the false discovery rate: A practical and powerful approach to multiple
testing. Journal of the Royal Statistical Society. Series B
(Methodological) \textbf{57}:289--300.

\hypertarget{ref-Breiman2001}{}
52. \textbf{Breiman, L.} 2001. Machine Learning. Springer Nature
\textbf{45}:5--32.

\hypertarget{ref-r_citation_2017}{}
53. \textbf{R Core Team}. 2017. R: A language and environment for
statistical computing. R Foundation for Statistical Computing, Vienna,
Austria.

\hypertarget{ref-rcompanion_citation_2017}{}
54. \textbf{Mangiafico, S.} 2017. Rcompanion: Functions to support
extension education program evaluation.

\hypertarget{ref-car_citation_2011}{}
55. \textbf{Fox, J.}, and \textbf{S. Weisberg}. 2011. An R companion to
applied regressionSecond. Sage, Thousand Oaks CA.

\hypertarget{ref-lme4_citation_2015}{}
56. \textbf{Bates, D.}, \textbf{M. Mächler}, \textbf{B. Bolker}, and
\textbf{S. Walker}. 2015. Fitting linear mixed-effects models using
lme4. Journal of Statistical Software \textbf{67}:1--48.

\hypertarget{ref-epir_citation_2017}{}
57. \textbf{Telmo Nunes, M. S. with contributions from}, \textbf{C.
Heuer}, \textbf{J. Marshall}, \textbf{J. Sanchez}, \textbf{R. Thornton},
\textbf{J. Reiczigel}, \textbf{J. Robison-Cox}, \textbf{P. Sebastiani},
\textbf{P. Solymos}, \textbf{K. Yoshida}, \textbf{G. Jones}, \textbf{S.
Pirikahu}, \textbf{S. Firestone}, and \textbf{R. Kyle.} 2017. EpiR:
Tools for the analysis of epidemiological data.

\hypertarget{ref-metafor_citation_2010}{}
58. \textbf{Viechtbauer, W.} 2010. Conducting meta-analyses in R with
the metafor package. Journal of Statistical Software \textbf{36}:1--48.

\hypertarget{ref-vegan_citation_2017}{}
59. \textbf{Oksanen, J.}, \textbf{F. G. Blanchet}, \textbf{M. Friendly},
\textbf{R. Kindt}, \textbf{P. Legendre}, \textbf{D. McGlinn}, \textbf{P.
R. Minchin}, \textbf{R. B. O'Hara}, \textbf{G. L. Simpson}, \textbf{P.
Solymos}, \textbf{M. H. H. Stevens}, \textbf{E. Szoecs}, and \textbf{H.
Wagner}. 2017. Vegan: Community ecology package.

\hypertarget{ref-caret_citation_2017}{}
60. \textbf{Jed Wing, M. K. C. from}, \textbf{S. Weston}, \textbf{A.
Williams}, \textbf{C. Keefer}, \textbf{A. Engelhardt}, \textbf{T.
Cooper}, \textbf{Z. Mayer}, \textbf{B. Kenkel}, \textbf{the R Core
Team}, \textbf{M. Benesty}, \textbf{R. Lescarbeau}, \textbf{A. Ziem},
\textbf{L. Scrucca}, \textbf{Y. Tang}, \textbf{C. Candan}, and
\textbf{T. Hunt.} 2017. Caret: Classification and regression training.

\hypertarget{ref-randomforest_citation_2002}{}
61. \textbf{Liaw, A.}, and \textbf{M. Wiener}. 2002. Classification and
regression by randomForest. R News \textbf{2}:18--22.

\hypertarget{ref-ggplot2_citation_2009}{}
62. \textbf{Wickham, H.} 2009. Ggplot2: Elegant graphics for data
analysis. Springer-Verlag New York.

\hypertarget{ref-gridextra_citation_2017}{}
63. \textbf{Auguie, B.} 2017. GridExtra: Miscellaneous functions for
``grid'' graphics.

\newpage

\textbf{Table 1: Total Individuals in each Study Included in the Fecal
Analysis}

\footnotesize

\begin{longtable}[]{@{}cccccc@{}}
\toprule
Study & Data Stored & Region & Control (n) & Adenoma (n) & Carcinoma
(n)\tabularnewline
\midrule
\endhead
Ahn & DBGap & V3-4 & 148 & 0 & 62\tabularnewline
Baxter & SRA & V4 & 172 & 198 & 120\tabularnewline
Brim & SRA & V1-3 & 6 & 6 & 0\tabularnewline
Flemer & Author & V3-4 & 37 & 0 & 43\tabularnewline
Hale & Author & V3-5 & 473 & 214 & 17\tabularnewline
Wang & SRA & V3 & 56 & 0 & 46\tabularnewline
Weir & Author & V4 & 4 & 0 & 7\tabularnewline
Zeller & SRA & V4 & 50 & 37 & 41\tabularnewline
\bottomrule
\end{longtable}

\normalsize
\newpage

\textbf{Table 2: Studies with Tissue Samples Included in the Analysis}

\footnotesize

\begin{longtable}[]{@{}cccccc@{}}
\toprule
Study & Data Stored & Region & Control (n) & Adenoma (n) & Carcinoma
(n)\tabularnewline
\midrule
\endhead
Burns & SRA & V5-6 & 18 & 0 & 16\tabularnewline
Chen & SRA & V1-3 & 9 & 0 & 9\tabularnewline
Dejea & SRA & V3-5 & 31 & 0 & 32\tabularnewline
Flemer & Author & V3-4 & 103 & 37 & 94\tabularnewline
Geng & SRA & V1-2 & 16 & 0 & 16\tabularnewline
Lu & SRA & V3-4 & 20 & 20 & 0\tabularnewline
Sanapareddy & Author & V1-2 & 38 & 0 & 33\tabularnewline
\bottomrule
\end{longtable}

\normalsize
\newpage

\textbf{Figure 1: Significant Bacterial Community Metrics for Adenoma or
Carcinoma in Feces} A) Adenoma evenness. B) Carcinoma evenness. C)
Carcinoma Shannon diversity. Blue represents controls and red represents
either adenoma (panel A) or carcinoma (panel B and C). The black lines
represent the median value for each repsective group.

\textbf{Figure 2: Odds Ratio for Adenoma or Carcinoma based on Bacterial
Community Metrics in Feces.} A) Community-based odds ratio for adenoma.
B) Community-based odds ratio for carcinoma. Colors represent the
different variable regions used within the respective study.

\textbf{Figure 3: The AUC of Indivdiual Significant OR Taxa to classify
Carcinoma.} A) Fecal samples. B) Unmatched tissue samples. The larger
circle represents the median AUC of all studies and the smaller circles
represent the individual AUC for a particular study. The dotted line
denotes an AUC of 0.5.

\textbf{Figure 4: Most Important Members in Significant OR Taxa
Carcinoma Models.} A) Common taxa in the top 10 percent for carcinoma
Random Forest fecal-based models. B) Common taxa in the top 10 percent
for carcinoma Random Forest unmatched tissue-based models. Blue
represents less important and red represents more important to the
Random Forest Model. White means that particular taxa was not in the top
10\%.

\textbf{Figure 5: Fecal Sample Random Forest Model Train AUCs.} A)
Adenoma random forest model AUCs between all genera, all OTU, and select
model based on significant OR taxa. B) Carcinoma random forest model
AUCs between all genera, all OTU, and select model based on significant
OR taxa. The black line represents the median AUC for the respective
group. If no values are present in the singificant OR taxa group then
there were no significant taxa identified and no model was tested.

\textbf{Figure 6: Fecal Sample Random Forest Genus-Based Model Test
AUCs.} A) Test AUCs of adenoma models using all genera across study. B)
Test AUCs of carcinoma models using all genera or significant OR taxa
only. The black line represents the AUC at 0.5. The red lines represent
the median AUC of all test AUCs for a specific study.

\newpage

\textbf{Figure S1: Odds Ratio for Adenoma or Carcinoma based on
Bacterial Community Metrics in Tissue.} A) Community-based odds ratio
for adenoma. B) Community-based odds ratio for carcinoma. Colors
represent the different variable regions used within the respective
study.

\textbf{Figure S2: Tissue Random Forest Model Train AUCs.} A) Adenoma
random forest model AUCs between all genera, all OTU, and select model
based on significant OR taxa in unmatched and matched tissue. B)
Carcinoma random forest model AUCs between all genera, all OTU, and
select model based on significant OR taxa in unmatched and matched
tissue. The black line represents the median AUC for the respective
group. If no values are present in the singificant OR taxa group then
there were no significant taxa identified and no model was tested.

\textbf{Figure S3: Most Common Taxa Across Carcinoma Full Community
Fecal Sample Study Models.} A) Common taxa in the top 10 percent for
carcinoma Random Forest all taxa-based models. B) Common taxa in the top
10 percent for carcinoma Random Forest all OTU-based models. Blue
represents less important and red represents more important to the
Random Forest Model. White means that particular taxa was not in the top
10\%.

\textbf{Figure S4: Most Common Genera Across Full Community Tissue Study
Models.} A) Common genera in the top 10 percent for matched carcinoma
Random Forest all genera-based models. B) Common genera in the top 10
percent for unmatched carcinoma Random Forest all genera-based models.
C) Common genera in the top 10 percent for matched carcinoma Random
Forest all OTU-based models. D) Common genera in the top 10 percent for
unmatched carcinoma Random Forest all OTU-based models. Blue represents
less important and red represents more important to the Random Forest
Model. White means that particular taxa was not in the top 10\%.

\textbf{Figure S5: Tissue Random Forest Genus-Based Model Test AUCs.} A)
Test AUCs of adenoma models using all genera across study. B) Test AUCs
of carcinoma models using all genera for matched tissue studies. C) Test
AUCs of carcinoma models using all genera or significant OR taxa only
for unmatched tissue studies The black line represents the AUC at 0.5.
The red lines represent the median AUC of all test AUCs for a specific
study.

\newpage


\end{document}
