---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")
flemer_data <- read_tsv("../data/process/flemer/flemer.metadata")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in odds ratio alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_OR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_tissue_composite.csv"))

# Load in PERMANOVA data
adn_beta_stool <- read_csv("../data/process/tables/beta_perm_adn_stool_summary.csv")
crc_beta_stool <- read_csv("../data/process/tables/beta_perm_stool_summary.csv")

adn_unmatched <- read_csv("../data/process/tables/beta_perm_adn_unmatched_tissue_summary.csv")
adn_matched <- read_csv("../data/process/tables/beta_perm_adn_matched_tissue_summary.csv")

crc_unmatched <- read_csv("../data/process/tables/beta_perm_unmatched_tissue_summary.csv")
crc_matched <- read_csv("../data/process/tables/beta_perm_matched_tissue_summary.csv")

#Load in odds ratio select genera data tables
crc_OR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(bh < 0.05))

crc_unmatched_OR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_unmatched_tissue_composite.csv") %>% mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(bh < 0.05))

crc_matched_OR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_matched_tissue_composite.csv") %>% mutate(bh = p.adjust(pvalue, method = "BH")))




```



# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Tumors

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that both individual and collections of bacteria are associated with the progression of colorectal cancer. As the number of studies investigating these associations increases and the number of subjects in each study increases, a meta-analysis to identify the associations that are the most predictive of disease progression is warranted. For our meta-analysis, we analyzed previously published 16S rRNA gene sequencing data collected from feces (`r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` individuals from `r length(stool_n$study)` studies) and colon tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals from `r length(tissue_n$study)` studies).

**Results.** We quantified the odds ratios for individual bacterial genera that were associated with an individual having tumors relative to a normal colon. Among the stool samples, there were no genera that had a significant odds ratio associated with adenoma and there were `r length(crc_OR_stool$measure)` genera with significant odds ratios associated with carcinoma. Similarly, among the tissue samples, there were no genera that had a significant odds ratio associated with adenoma and there were `r length(crc_unmatched_OR_tissue$measure)` genera with significant odds ratios associated with carcinoma. Among the significant odds ratios, the association between individual taxa and tumor diagnosis was equal or below `r ifelse(max(crc_OR_stool$rr) > max(crc_unmatched_OR_tissue$rr), round(max(crc_OR_stool$rr), digits = 2), round(max(crc_unmatched_OR_tissue$rr), digits = 2))`. Because individual taxa had limited association with tumor diagnosis, we trained Random Forest classification models using the genera with the five highest and lowest odds ratios, using the entire collection of genera found in each study, and using operational taxonomic units defined based on a 97% similarity threshold. All training approaches yielded similar classification success as measured using the  Area Under the Curve. The ability to correctly classify individuals with adenomas was poor and the ability to classify individuals with carcinomas was considerably better using sequences from stool or tissue.

**Conclusions.** This meta-analysis confirms previous results indicating that individuals with adenomas cannot be readily classified based on their bacterial community, but that those with carcinomas can. Regardless of the dataset, we found a subset of the fecal community that was associated with carcinomas was as predictive as the full community. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; tumor; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world-wide health problem in which the microbiota has been hypothesized to have a role in disease progression [@siegel_cancer_2016; @flynn_metabolic_2016]. Numerous studies using murine models of CRC have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in tumorigenesis. Numerous case-control studies have characterized the microbiota of individuals with colonic adenomas and carcinomas in an attempt to identify biomarkers of disease progression [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Because current CRC screening recommendations are poorly adhered to due to socioeconomic status, test invasiveness, and frequency of tests, development and validation of microbiome-associated biomarkers for CRC progression could further attempts to develop non-invasive diagnostics [@GimenoGarca2012].


Recently, there has been an intense focus on identifying microbiota-based biomarker yielding a seemingly endless number of candidate taxa. Some studies point towards mouth-associated genera such as Fusobacterium, Peptostreptococcus, Parvimonas, and Porphyromonas that are enriched in people with carcinomas [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Other studies have identified members of *Akkermansia*, *Bacteroides*, *Enterococcus*, *Escherichia*, *Klebsiella*, *Mogibacterium*, *Streptococcus*, and *Providencia* are also associated with carciomas  [@burns_virulence_2015; @chen_human_2012; @wang_structural_2012]. Additionally, *Roseburia* has been found in some studies to be more abundant in people with tumors but in other studies it has been found to be either less abundant or no different than what is found in subjects with normal colons  [@geng_diversified_2013; @wang_structural_2012; @flemer_tumour-associated_2017; @dejea_microbiota_2014]. There are strong results from tissue culture and murine models that Fusobacterium nucleatum, pks-positive strains of Escherichia coli, Streptococcus gallolyticus, and an entertoxin-producing strain of Bacteroides fragilis are important in the pathogenesis of CRC [@ecoli_Arthur_2014; @arthur_intestinal_2012; @strep_Aymeric_2017; @weir_stool_2013; @wang_structural_2012; @bfrag_Boleij_2014]. These results point to a causative role for the microbiota in CRC pathogenesis as well as their potential as diagnostic biomarkers.


Most studies have focused on identifying biomarkers in patients with carcinomas but there is a greater clinical need to identify biomarkers associated with adenomas. Studies focusing on broad scale community metrics have found that measures such as the total number of Operational Taxonomic Units (OTUs) are decreased in those with adenomas versus controls [@sanapareddy_increased_2012]. Other studies have identified *Acidovorax*, *Bilophila*, *Cloacibacterium*, *Desulfovibrio*, *Helicobacter*, *Lactobacillus*, *Lactococcus*, *Mogibacterium*, and *Pseudomonas* to be enriched in those with adenomas [@sanapareddy_increased_2012; @lu_mucosal_2016; @hale_shifts_2017]. There are few genera that are enriched in patients with adenoma or carcinoma tumors.


Confirming some of these previous findings, a recent meta-analysis found that 16S rRNA gene sequences from members of the *Akkermansia*, *Fusobacterium*, and *Parvimonas* were fecal biomarkers for the presence of carcinomas [@shah_leveraging_2017]. Contrary to previous studies they found sequences similar to members of *Lactobacillus* and *Ruminococcus* to be enriched in patients with adenoma or carcinoma relative to those with normal colons [@baxter_microbiota-based_2016; @burns_virulence_2015; @zeller_potential_2014]. In addition, they found 16S rRNA gene sequences from members of *Haemophilus*, *Methanosphaera*, *Prevotella*, *Succinovibrio* were enriched in patients with adenoma and *Pantoea* were enriched in patients with carcinomas. Although this meta-analysis was helpful for distilling a large number of possible biomarkers, the aggregate number of samples included in the analysis (n = 509) was smaller than several larger case-control studies that have since been published [@baxter_microbiota-based_2016; @hale_shifts_2017]


Here we provide an updated meta-analysis using 16S rRNA gene sequence data from both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and colon tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) from 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] [Table 1 & 2]. We expand both the breadth and scope of the previous meta-analysis to investigate whether biomarkers describing the bacterial community or specific members of the community can more accurately classify patients as having adenoma or carcinoma. Our results suggest that the bacterial community changes as disease severity worsens and that that a subset of the microbial community can be used to diagnose the presence of carcinoma.


\newpage

## Results

***Lower Bacterial Diversity is Associated with Increased Odds Ratio (OR) of Tumors:***
We first assessed whether variation in broad community metrics like total number of operational taxonomic units (OTUs) (i.e. richness), the evenness of their abundance, and the overall diversity was associated with disease stage after controlling for study and variable region differences. In stool, there was a significant decrease in both evenness and diversity as disease severity progressed from normal to adenoma to carcinoma (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))` and `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`, respectively) [Figure 1]; there was not a significant difference for richness (P-value = `r paste(format(filter(stool_lme, alpha_metric == "sobs")[, "BH"], digits = 2, nsmall = 2))`). We next tested whether the decrease in these community metrics translated into significant ORs for having an adenoma or carcinoma. For fecal samples, the ORs for richness were not significantly greater than 1.0 for adenoma or carcinoma (P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "sobs")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2A]. The ORs for evenness were significantly higher than 1.0 for adenoma (OR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) and carcinoma (OR =  `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2B]. The ORs for diversity were only significantly greater than 1.0 for carcinoma (OR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`), but not for adenoma (P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2C]. Although these OR are significantly greater than 1.0, it is doubtful that these are clinically meaningful values.

Similar to our analysis of sequences obtained from stool samples, we repeated the analysis using sequences obtained from colon tissue. There were no significant changes in richness, evenness, or diversity as disease severity progressed from control to adenoma to carcinoma (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05). We next analyzed the OR, for matched (i.e. where unaffected tissue and tumors were obtained from the same individual) and unmatched (i.e. where unaffected tissue and tumor tissue were not obtained from the same individual) tissue samples. The ORs for adenoma and carcinoma by any measure were not significantly different from 1.0 (P-value `r ifelse(min(alpha_RR_crc_unmatch_tissue$pvalue) < 0.05 & min(alpha_RR_crc_match_tissue$pvalue) < 0.05 & min(alpha_RR_adn_tissue$pvalue) < 0.05, invisible("check tables"), invisible(">"))` 0.05) [Figure S1 & Table S1]. This is likely due to the combination of a small effect size, as suggested from the results using stool, and the relatively small number of studies and size of studies used in the analysis.

***Disease Progression is Associated with Community-Wide Changes in Composition and Abundance:*** Based on the differences in evenness and diversity, we next asked whether there were community-wide differences in the structure of the communities associated with different disease stages. We identified significant bacterial community differences in the stool of patients with adenomas relative to those with normal colons in `r length(as.data.frame(filter(adn_beta_stool, pvalue < 0.05))[, "study"])` of `r length(as.data.frame(adn_beta_stool)[, "study"])` studies and in patients with carcinomas relative to those with normal colons in 6 of 7 studies (PERMANOVA; P-value < 0.05) [Table S2]. Similar to the analyses using stool samples, there were significant differences in bacterial community structure between subjects with normal colons and those with adenoma (1 of 2 studies) and carcinoma (1 of 3 studies) [Table S2]. For studies that used matched samples no differences in bacterial community structures were observed [Table S2]. Combined, these results indicate that there consistent and significant community-wide changes in the fecal community structure of subjects with carcinomas. However, the signal observed in subjects with adenomas or when using tissue samples was not as consistent. This is likely due to a smaller effect size or the relatively small sample sizes among the studies that characterized the tissue microbiota.


***Individual Taxa are Associated with Significant ORs for Carcinomas:*** 
Next we identified those taxa were associated with ORs that were significantly associated with having a normal colon or the presence of adenomas or carcinomas. No taxa had a significant OR for the presence of adenomas when we used data collected from stool or tissue samples (Table S3 & S4). In contrast, `r length(crc_OR_stool$measure)` taxa had significant ORs for the presence of carcinomas using data from stool samples. Of these, 4 are commonly associated with the oral cavity: Fusobacterium (OR = `r paste(round(filter(crc_OR_stool, measure == "Fusobacterium")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Fusobacterium")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Fusobacterium")[, "ci_ub"], digits = 2))`)), Parvimonas (OR = `r paste(round(filter(crc_OR_stool, measure == "Parvimonas")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Parvimonas")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Parvimonas")[, "ci_ub"], digits = 2))`)), Porphyromonas (OR = `r paste(round(filter(crc_OR_stool, measure == "Porphyromonas")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Porphyromonas")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Porphyromonas")[, "ci_ub"], digits = 2))`)), and Peptostreptococcus (OR = `r paste(round(filter(crc_OR_stool, measure == "Peptostreptococcus")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Peptostreptococcus")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Peptostreptococcus")[, "ci_ub"], digits = 2))`)) [Table S3]. The other 4 were Clostridium XI (OR = `r paste(round(filter(crc_OR_stool, measure == "Clostridium_XI")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Clostridium_XI")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Clostridium_XI")[, "ci_ub"], digits = 2))`)), Enterobacteriaceae  (OR = `r paste(round(filter(crc_OR_stool, measure == "Enterobacteriaceae_unclassified")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Enterobacteriaceae_unclassified")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Enterobacteriaceae_unclassified")[, "ci_ub"], digits = 2))`)), Escherichia (OR = `r paste(round(filter(crc_OR_stool, measure == "Escherichia.Shigella")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Escherichia.Shigella")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Escherichia.Shigella")[, "ci_ub"], digits = 2))`)), and Ruminococcus (OR = `r paste(round(filter(crc_OR_stool, measure == "Ruminococcus")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Ruminococcus")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Ruminococcus")[, "ci_ub"], digits = 2))`)). Among the data collected from tissue samples, only unmatched carcinoma samples had taxa with a significant OR. Those included Dorea (OR = `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "rr"], digits = 2))` (`r paste(round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "ci_ub"], digits = 2))`)), Blautia (OR = `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Blautia")[, "rr"], digits = 2))` (`r paste(round(filter(crc_unmatched_OR_tissue, measure == "Blautia")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Blautia")[, "ci_ub"], digits = 2))`)), and Weissella (OR = `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "rr"], digits = 2))` (`r paste(round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "ci_ub"], digits = 2))`)). Mouth-associated genera were not significantly associated with an increased OR for carcinoma in tissue samples [Table S4]. For example, Fusobacterium had an OR of `r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "rr"], digits = 2))` (`r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "ci_ub"], digits = 2))`; however, due to the small number of studies and considerable variation in the data, the Benjimani-Hochberg-corrected P-value was `r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "bh"], digits = 2))` [Table S4]. It is interesting to note that Ruminococcus and members of Clostridium group XI in stool and Dorea and Blautia in tissue had ORs that were significantly less than 1.0, which suggests that these populations are protective against the development of carcinomas. Overall, there was no overlap in the taxa with significant OR between stool and tissue samples.


***Individual Significant OR Taxa Classify Carcinoma Poorly:***
Since specific taxa were associated with a significant OR for carcinoma we tested whether they would also be good classifiers of carcinoma. For stool, the `r length(crc_OR_stool$measure)` significant taxa did no better at classifying those with normal colons versus those with carcinomas than chance [Figure 3A]. Likewise in unmathced tissue samples the `r length(crc_unmatched_OR_tissue$measure)` significant taxa were no better than an AUC of 0.5 [Figure 3B]. These results suggest that although these taxa are significantly associated with a decreased or increased OR of carcinoma, individually they are poor classifiers of disease.  


***Select Community Models can Recapitulate Whole Community Models:***
Since specific taxa increased or decreased the OR for carcinoma but performed poorly as an individual classifier of carcinoma and assessing ORs for adenoma we assessed whether the overall bacterial community was better at classifying disease. Three models were tested and included a full taxa model, full OTU model, and significant OR taxa model. If no taxa were significant after multiple comparison correction then no model for that specific grouping (i.e. adenoma stool) was analyzed. We first tested three model AUCs. Next, the all genera models and any significant OR taxa models were tested across all studies that were not used to train the model. 

For stool, all models used had similar AUCs for both adenoma and carcinoma [Figure 4]. However, the adenoma AUCs were barely better than chance [Figure 4A]. Similarily, when analyzing the tests sets that were comprised of genera data from other studies the adenoma models performed poorly [Figure 5A]. The carcinoma models performed much better, with both the all genera and singificant OR taxa only models having a similar ability to detect individuals with carcinomas [Figure 5B]. The most common genera in the top 10 most important variables, in the full genera models used to classify carcinomas, were *Bacteroides* and *Lachnospiraceae* [Figure 6A]. Regardless of sample type, mouth-associated genera were present in models for carcinomas but not consistently across studies [Figure 6]. For the full community OTU-based models, both *Bacteroides*, *Blautia*, *Lachnospiraceae*, and *Ruminococcaceae* were present in the top 10 consistently across studies [Figure 6B]. Overall, these results suggest that multiple microbes could act as the inflammatory stimulus needed to exacerbate mutations leading each individual microbe as a poor individual classifier but much better in aggregate with others.


The tissue-based models had results that were dependent on genera or OTU level data and matched or unmatched samples [Figure S2]. Similarily to stool, the significant OR taxa performed as well as both the full taxa and OTU models [Figure S3B]. When analyzing the tests sets that were comprised of genera data from other studies, all models performed poorly with one exception being carcinoma classification in unmatched tissue [Figure S4]. Unlike stool, the test sets for the significant OR taxa did not perform as well as the all genera-based models [Figure S4C]. For the full genera models built using either matched or unmatched samples, showed no consistent taxa representation in the top 10 most important model variables across study [Figure S4A & S4B]. Conversely, the OTU-based models built using either matched or unmatched samples showed more commonalities [Figure S4C & S4D]. In models built using matched samples *Lachnospiraceae*, *Fusobacteriaceae*, *Comamonadaceae*, and *Bacteroidaceae* appeared in the top 10 percent in the majority of studies [Figure S4C]. Although models built with unmatched samples also had *Lachnospiraceae* and *Bacteroidaceae*, a major difference was the presence of *Ruminococcaceae* in the top 10 of every study [Figure S4D]. This results suggests that either the colon tissue microbiota is study or person dependent or that that kit and/or other types of contamination associated with low biomass samples may be skewing the results.



\newpage

## Discussion

*Targeting the identification of tumor microbial biomarkers within stool seems logical since it offers an easy and cost-effective way to stratify risk of disease. The current gold standard for diagnosis, a colonoscopy, can be time-consuming and is not without risk of complications. Although stool represents an easy and less invasive way to assess risk, it is not clear how well this sample reflects adenoma- and carcinoma- associated microbial communities. Some studies have tried to assess this in health and disease but are limited by their sample size [@flemer_tumour-associated_2017; @Flynn_preprint_2017]. Sampling the microbiota directly associated with colon tissue may provide clearer answers but is not without their own limitations. After the colonoscopy bowel prep the bacterial community sampled may reflect the better adhered microbiota versus the resident community. Additionally, these samples contain more host DNA, potentially limiting the types of analysis that can be done. It is well known that low biomass samples can be very difficult to work with and results can be study dependent due to the randomness of contamination [@Salter_contamination_2014].* 


Our study identifies clear but small differences in diversity at the community level and larger differences for individual genera, present in patients with tumors versus controls [Figure 1-3]. Although there was a step-wise decrease in diversity as disease progressed from control to adenoma to carcinoma, this did not translate into large effect size increases in OR for either adenoma or carcinoma tumors. Even though mouth-associated genera increased individaul's OR of having a carcinoma for certain sample types, they did not consistently increase the OR of having an adenoma. By using these taxa that had significant ORs after multiple comparison correction we found that we could classify indviduals with either adenoma or carcinoma as well as models that use either all genera or all OTUs. Finally, many studies were individually under powered to be able to reject the null hypothesis and this could one reason only the comparison between control and carcinoma individuals for stool samples had relible detectable differences.


The data presented herein support the importance of specific taxa for carcinoma, but not necessarily adenoma, tumor formation. The results that we have presented show that the significant OR taxa model and both the full genera and OTU models, for indviduals with carcinoma, had similar AUCs [Figure 2 & 3]. This suggests that an interplay between a select number of potentially protective and exacerbating microbes within the GI community could be crucial for carcinoma formation. Importantly, it suggests that there may be key members of the GI community that should be studied further to potentially help reduce the risk of carcinoma tumor formation. Conversely, using the present data, it is clear that new approaches may be needed to identify members of the community associated with adenoma tumors. Regardless of sample type and whether a full genera- or OTU-based model was used, our Random Forest models consistently performed poorly. Yet, the step-wise decrease in diversity suggests that the adenoma-associated community is not normal but has changed subtly. This change in diversity, at this early stage of disease, could be focal to the adenoma itself. How the host interacts with these subtle changes at early stages of the disease could be what leads to a thoroughly dysfunctional community that is supportive of tumorgenesis. 


For the full genera- and OTU-based models within stool, common GI microbes were most consistently present in the top 10 genera or OTUs across studies [Figure 4]. Changes in *Bacteroides*, *Ruminococcaceae*, *Ruminococcus*, and *Roseburia* were consistently found to be in the top 10 most important variables across the different studies for both individuals with adenoma and carcinoma [Figure 4]. These data suggest that whether the non-resident bacterium is *Fusobacteria* or *Peptostreptococcus* may not be as important as how these bacteria interact with the changing resident community. Based on these observations, it is possible to hypothesize that small changes in community structure lead to new niches in which any one of the mouth-associated or general inflammatory genera can gain a foothold, exacerbating the initial changes in community and facilitating the transition from adenoma to carcinoma stage of disease.     

The colon tissue-based studies did not provide a clearer understanding of how the microbiota may be associated with tumors. Generally, the full OTU-based models of unmatched and matched colon tissue samples were concordant with stool samples showing that GI resident microbes were the most prevalent in the top 10 most important variables across study [Figure S4E & F]. Unlike in stool, *Fusobacterium* was the only mouth-associated bacteria consistently present in the top 10 most important variables of the full carcinoma stage models [Figure S4B-C & E-F]. The majority of the colon tissue-based results seem to be study specific with many of the top 10 taxa being present only in a single study. Additionally, the presence of genera associated with contamination, within the top 10 most important variables for the genera and OTU models is worrying. The low bacterial biomass of tissue samples coupled with potential contamination could explain why these results seem to be more sporadic than the stool results.

One important caveat to this study is that even though genera associated with certain species such as *Bacteroides fragilis* and *Streptococcus gallolyticus* subsp. *gallolyticus* were not identified, it does not necessarily mean that these specific species are not important in human CRC [@bfrag_Boleij_2014; @strep_Aymeric_2017]. Since we are limited in our aggregation of the data to the genus level, it is not possible to clearly delineate which species are contributing to overall disease progression. Our observations are not inconsistent with the previous literature on either *Bacteroides fragilis* or *Streptococcus gallolyticus* subsp. *gallolyticus*. As an example, the stool-based full community models consistently identified the genus *Bacteroides*, as well as OTUs that classified as *Bacteroides*, to be important model components across studies. This suggests that even though *Bacteroides* may not increase the OR of individuals having an adenoma or carcinoma and may not vary in relative abundance, like *Fusobacterium*, it is still important in CRC. Additionally, *Streptococcus gallolyticus* subsp. *gallolyticus* is a mouth-associated microbe, and the results from this study suggest that regardless of sample type, mouth-associated genera are commonly associated with an increased OR for individuals to have a carcinoma tumor.    

The associations between the microbiota and individuals with adenoma tumors are inconclusive, in part, because many studies may not be powered effectively to observe small effect sizes. None of the studies analyzed were properly powered to detect a 10% or lower change between cases and controls. The results within our meta-analysis suggest that a small effect size may well be the scope in which differences consistently occur between controls and those with adenomas. Future studies investigating adenoma tumors and the microbiota need to take power into consideration to reproducibly study whether the microbiota contributes to adenoma formation. In contrast to adenoma stage of disease, our observations suggest that most studies analyzed have sufficient power to detect many changes in the carcinoma-associated microbiota because of large effect size differences between cases and controls [Figure 5].

\newpage

## Conclusion

By aggregating together a large collection of studies analyzing both fecal and colon tissue samples, we are able to provide evidence supporting the importance of the bacterial community in colorectal tumors. The data presented here suggests that mouth-associated microbes can gain a foothold within the colon and are commonly associated with the greatest OR of individuals having a carcinoma. Conversely, no conclusive signal with these mouth-associated microbes could be detected for individuals with an adenoma. Our observations also highlight the importance of power and sample number considerations when investigating the microbiota and adenoma tumors due to possible subtle changes in the community. Overall, the associations between the microbiota and individuals with carcinomas were much stronger than with those with adenomas. 

\newpage

## Methods

***Obtaining Data Sets:***
The studies used for this meta-analysis were identified through the review articles written by Keku, *et al.* and Vogtmann, *et al.* [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016] and additional studies not mentioned in the reviews were obtained based on the authors' knowledge of the literature. Studies that used tissue or feces as their sample source for 454 or Illumina 16S rRNA gene sequencing analysis and had data sets with sequences available for analysis were included. Some studies were excluded because they did not have publicly available sequences or did not have metadata in which the authors were able to share. After these filtering steps, the following studies remained: Ahn, *et al.* [@ahn_human_2013], Baxter, *et al.* [@baxter_microbiota-based_2016], Brim, *et al.* [@brim_microbiome_2013], Burns, *et al.* [@burns_virulence_2015], Chen, *et al.*  [@chen_human_2012], Dejea, *et al.* [@dejea_microbiota_2014], Flemer, *et al.* [@flemer_tumour-associated_2017], Geng, *et al.* [@geng_diversified_2013], Hale, *et al.* [@hale_shifts_2017], Kostic, *et al.* [@kostic_genomic_2012], Lu, *et al.* [@lu_mucosal_2016], Sanapareddy, *et al.* [@sanapareddy_increased_2012], Wang, *et al.* [@wang_structural_2012], Weir, *et al.* [@weir_stool_2013], and Zeller, *et al.*  [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included because the 90 individuals analyzed within the study are contained within the larger Baxter study [@baxter_microbiota-based_2016]. After sequence processing, all the case samples for the Kostic study had 100 or less sequences remaining and was excluded, leaving a total of 14 studies that analysis could be completed on.

***Data Set Breakdown:***
In total, there were seven studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), five studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].  

***Sequence Processing:***
For the majority of studies, raw sequences were downloaded from the Sequence Read Archive (SRA) (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata were obtained by searching the respective accession number of the study at the following website: http://www.ncbi.nlm.nih.gov/Traces/study/. Of the studies that did not have sequences and metadata on the SRA, data was obtained from DBGap (n = 1, [@ahn_human_2013]) and directly from the authors (n = 4, [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]). Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009] and quality filtering utilized the default methods for both 454 and Illumina based sequencing. If it was not possible to use the defaults, the stated quality cut-offs, from the study itself, were used instead. Sequences that were made up of an artificial combination of two or more different sequences and commonly known as chimeras were identified and removed using VSEARCH [@rognes_vsearch_2016] before *de novo* OTU clustering at 97% similarity was completed using the OptiClust algorithm [@westcott_opticlust_2017]. 


***Study Analysis Overview:***
OTU richness, evenness, and Shannon diversity were first assessed for differences between controls, adenoma tumors, and carcinoma turmors using both linear mixed-effect models and ORs. For each individual study the Bray-Curtis index was used to assess differences between control-adenoma and control-carcinoma individuals. Next, all common genera were assessed for differences in ORs for individuals having an adenoma or carcinoma and corrected for multiple comparisons using the Benjamini-Hochberg method [@benjamini_controlling_1995]. We then built Random Forest models based on all genera, all OTUs, or significant OR taxa (only using taxa still significant after multiple comparison correction). For both the full genera and significant OR taxa, models were trained on one study then tested on the remaining studies using genera-based relative abundances. The OTU-based models were built using OTU level data and a 10-fold CV over 100 different iterations, based on random 80/20 splitting of the data, was used to generate a range of expected AUCs. This process was repeated for every study in the meta-analysis. Comparisons of the initial trained model AUCs for the full genera and significant OR taxa were made to the mean AUC generated from the 100 different 10-fold CV runs of the respective OTU-based model. For comparisons in which only control versus adenoma individuals were made, the carcinoma individuals were excluded from each respective study. Similarly, for comparisons in which control versus carcinoma individuals were made the adenoma individuals were excluded from each respective study. For all analysis completed fecal and tissue samples were kept separate. Within the tissue groups the data were further divided between samples from the same individual (matched) and those from different individuals (unmatched).

***Obtaining Genera Relative Abundance and Significant OR Taxa Models:***
For the genera analysis of the ORs, OTUs were added together based on the genus or lowest available taxonomic classification level and the total average counts, for 100 different subsamplings was obtained. The significant OR taxa models for the  Random Forest models utilized all taxa that had significant ORs after multiple comparison correction. This meant only models for the carcinoma stool (`r length(crc_OR_stool$measure)` variables) and carcinoma unmatched (`r length(crc_unmatched_OR_tissue$measure)` variables) samples were possible to be created and analyzed. 


***Matched versus Unmatched Tissue Samples:***
In general, tissue samples with control and tumor samples from different individuals were classified as unmatched while samples that belonged to the same individual were classified as matched. Studies with matched data included Burns, Dejea, Geng, and Lu while those with unmatched data were from Burns, Flemer, Chen, and Sanapareddy. For some studies samples became unmatched when a corresponding matched sample did not make it through sequence processing. All samples, from both matched and unmatched tissue samples, were analyzed together for the linear mixed-effect models with samples from the same individual being corrected for. All other analysis, where it is not specified explicitly, matched and unmatched samples were analyzed separately using the statistical approaches mentioned in the Statistical Analysis section. 

***Assessing Important Random Forest Model Variables:***
Using Mean Decrease in Accuracy (MDA) the top 10 most important variables to the Random Forest model were obtained for the full models of the two different approaches used. For the first approach utilizing genus-based models, the number of times that a specific taxa showed up in the top 10 of the training set across each study was counted. For the second approach, that utilized the OTU-based models, the medians for each OTU across 100 different 80/20 splits of the data was generated and the top 10 OTUs then counted for each study. Common taxa were then identified by using the lowest classification for each of the specific OTUs obtained from these counts and the number of times this classification occurred across the top 10 of each study was recorded. Finally, the two studies that had adenoma tumor tissue (Lu and Flemer) were equally divided between matched and unmatched studies and were grouped together for the counting of the top 10 genera and OTUs for both Random Forest approaches.


***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R (v`r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. For OTU richness, evenness, and Shannon diversity analysis, values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package [@rcompanion_citation_2017] and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package [@car_citation_2011]. Testing for OTU richness, evenness, and Shannon diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package [@lme4_citation_2015] to correct for study, repeat sampling of individuals (tissue only), and 16S hyper-variable region used. Odds ratios (OR) were analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages [@epir_citation_2017; @metafor_citation_2010] by assessing how many individuals with and without disease were above and below the overall median value within each specific study. OR significance testing utilized the chi-squared test. Diversity differences measured by the Bray-Curtis index utilized the creation of distance matrix and testing with PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package [@vegan_citation_2017]. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages [@caret_citation_2017; @randomforest_citation_2002]. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages [@ggplot2_citation_2009; @gridextra_citation_2017]. 


***Reproducible Methods:***
The code and analysis can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless otherwise mentioned, the accession number of raw sequences from the studies used in this analysis can be found directly in the respective batch file in the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscripts used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective study batch file in the GitHub repository or within the original publication. For instances when sequences are not publicly available, they may be accessed by contacting the corresponding authors from whence the data came.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a Canadian Institute of Health Research fellowship and a University of Michigan Postdoctoral Translational Scholar Program grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally, we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Total Individuals in each Study Included in the Stool Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-V3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: Significant Bacterial Community Metrics for Adenoma or Carcinoma in Stool.** A) Adenoma evenness. B) Carcinoma evenness. C) Carcinoma Shannon diversity. Blue represents controls and red represents either adenoma (panel A) or carcinoma (panel B and C). The black lines represent the median value for each repsective group. 

**Figure 2: Odds Ratio for Adenoma or Carcinoma based on Bacterial Community Metrics in Stool.** A) Community-based odds ratio for adenoma. B) Community-based odds ratio for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 3: The AUC of Indivdiual Significant OR Taxa to classify Carcinoma.** A) Stool samples. B) Unmatched tissue samples. The larger circle represents the median AUC of all studies and the smaller circles represent the individual AUC for a particular study. The dotted line denotes an AUC of 0.5. 

**Figure 4: Stool Random Forest Model Train AUCs.** A) Adenoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa. B) Carcinoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa. The black line represents the median AUC for the respective group. If no values are present in the singificant OR taxa group then there were no significant taxa identified and no model was tested.

**Figure 5: Stool Random Forest Genus-Based Model Test AUCs.** A) Test AUCs of adenoma models using all genera across study. B) Test AUCs of carcinoma models using all genera or significant OR taxa only. The black line represents the AUC at 0.5. The red lines represent the median AUC of all test AUCs for a specific study.  

**Figure 6: Most Common Taxa Across Carcinoma Full Community Stool Study Models.** A) Common taxa in the top 10 percent for carcinoma Random Forest all taxa-based models. B) Common taxa in the top 10 percent for carcinoma Random Forest all OTU-based models.



\newpage


**Figure S1: Odds Ratio for Adenoma or Carcinoma based on Bacterial Community Metrics in Tissue.** A) Community-based odds ratio for adenoma. B) Community-based odds ratio for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Tissue Random Forest Model Train AUCs.** A) Adenoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa in unmatched and matched tissue. B) Carcinoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa in unmatched and matched tissue. The black line represents the median AUC for the respective group. If no values are present in the singificant OR taxa group then there were no significant taxa identified and no model was tested.

**Figure S3: Tissue Random Forest Genus-Based Model Test AUCs.** A) Test AUCs of adenoma models using all genera across study. B) Test AUCs of carcinoma models using all genera for matched tissue studies. C) Test AUCs of carcinoma models using all genera or significant OR taxa only for unmatched tissue studies The black line represents the AUC at 0.5. The red lines represent the median AUC of all test AUCs for a specific study.

**Figure S4: Most Common Genera Across Full Community Tissue Study Models.** A) Common genera in the top 10 percent for matched carcinoma Random Forest all genera-based models. B) Common genera in the top 10 percent for unmatched carcinoma Random Forest all genera-based models. C) Common genera in the top 10 percent for matched carcinoma Random Forest all OTU-based models. D) Common genera in the top 10 percent for unmatched carcinoma Random Forest all OTU-based models. 

\newpage
