---
title: ''
csl: mbio.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")
flemer_data <- read_tsv("../data/process/flemer/flemer.metadata")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in odds ratio alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_OR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_tissue_composite.csv"))

# Load in PERMANOVA data
adn_beta_stool <- read_csv("../data/process/tables/beta_perm_adn_stool_summary.csv")
crc_beta_stool <- read_csv("../data/process/tables/beta_perm_stool_summary.csv")

adn_unmatched <- read_csv("../data/process/tables/beta_perm_adn_unmatched_tissue_summary.csv")
adn_matched <- read_csv("../data/process/tables/beta_perm_adn_matched_tissue_summary.csv")

crc_unmatched <- read_csv("../data/process/tables/beta_perm_unmatched_tissue_summary.csv")
crc_matched <- read_csv("../data/process/tables/beta_perm_matched_tissue_summary.csv")

#Load in odds ratio select genera data tables
crc_OR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(bh < 0.05))

crc_unmatched_OR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_unmatched_tissue_composite.csv") %>% mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(bh < 0.05))

crc_matched_OR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_matched_tissue_composite.csv") %>% mutate(bh = p.adjust(pvalue, method = "BH")))

#Load in AUC data for individual genera
ind_crc_stool_auc <- read_csv("../data/process/tables/ind_genera_auc_stool.csv") %>%  mutate(auc = ifelse(taxa == "Ruminococcus" | taxa == "Clostridium_XI", invisible(1-auc), invisible(auc)))
ind_crc_combined_stool_auc <- ind_crc_stool_auc %>% group_by(taxa) %>% summarise(auc = median(auc)) %>% mutate(study = "median") %>% ungroup()

ind_unmatched_tissue_crc_auc <- read_csv("../data/process/tables/ind_genera_auc_unmatched_tissue.csv") %>% mutate(auc = ifelse(taxa == "Dorea" | taxa == "Blautia", invisible(1-auc), invisible(auc)))
ind_crc_combined_unmatched_auc <- ind_unmatched_tissue_crc_auc %>% group_by(taxa) %>% summarise(auc = median(auc)) %>% 
  mutate(study = "median") %>% ungroup()

#Load in AUC data between select and full taxa models
adn_tissue_model_aucs <- 
  as.data.frame(read_csv("../data/process/tables/adn_ALL_genus_unmatched_tissue_RF_fullvsselect_pvalue_summary.csv") %>% 
  mutate(type = rep(c("unmatched", "matched"), 2), 
         pvalue = as.numeric(pvalue)) %>% 
  gather(key = model_type, value = AUC, full_model, select_model) %>% 
  mutate(AUC = as.numeric(AUC)) %>% 
  bind_rows(
    read_csv("../data/process/tables/adn_tissue_rf_otu_random_comparison_summary.csv") %>% 
      mutate(type = c("matched", "unmatched"), 
             model_type = "full_otu", 
             train_model = study, 
             act_mean_auc = as.numeric(act_mean_auc)) %>% 
      rename(AUC = act_mean_auc) %>% 
      select(pvalue, study, train_model, type, model_type, AUC)))


adn_stool_model_aucs <- as.data.frame(read_csv("../data/process/tables/adn_ALL_genus_stool_RF_fullvsselect_pvalue_summary.csv") %>% gather(key = model_type, value = AUC, full_model, select_model) %>% 
  mutate(AUC = as.numeric(AUC), 
         pvalue = as.numeric(pvalue)) %>% 
  bind_rows(read_csv("../data/process/tables/adn_stool_rf_otu_random_comparison_summary.csv") %>% 
              mutate(model_type = "full_otu", 
                     train_model = study, 
                     act_mean_auc = as.numeric(act_mean_auc)) %>% 
              rename(AUC = act_mean_auc) %>% 
              select(pvalue, study, train_model, model_type, AUC)))


crc_tissue_model_aucs <- as.data.frame(read_csv("../data/process/tables/ALL_genus_unmatched_tissue_RF_fullvsselect_pvalue_summary.csv") %>% mutate(type = "unmatched") %>% 
  bind_rows(read_csv("../data/process/tables/ALL_genus_matched_tissue_RF_fullvsselect_pvalue_summary.csv") %>% 
  mutate(type = "matched")) %>% 
  gather(key = model_type, value = AUC, full_model, select_model) %>% 
  mutate(AUC = as.numeric(AUC)) %>% 
  bind_rows(
    read_csv("../data/process/tables/matched_tissue_rf_otu_random_comparison_summary.csv") %>% 
      mutate(type = "matched", 
             model_type = "full_otu", 
             train_model = study, 
             act_mean_auc = as.numeric(act_mean_auc)) %>% 
      rename(AUC = act_mean_auc) %>% 
      select(pvalue, study, train_model, type, model_type, AUC) %>% 
      bind_rows(
        read_csv("../data/process/tables/unmatched_tissue_rf_otu_random_comparison_summary.csv") %>% 
          mutate(type = "unmatched", 
                 model_type = "full_otu", 
                 train_model = study, 
                 act_mean_auc = as.numeric(act_mean_auc)) %>% 
          rename(AUC = act_mean_auc) %>% 
          select(pvalue, study, train_model, type, model_type, AUC))))


crc_stool_model_aucs <- as.data.frame(read_csv("../data/process/tables/ALL_genus_stool_RF_fullvsselect_pvalue_summary.csv") %>% gather(key = model_type, value = AUC, full_model, select_model) %>% 
  mutate(AUC = as.numeric(AUC), 
         pvalue = as.numeric(pvalue)) %>% 
  bind_rows(read_csv("../data/process/tables/stool_rf_otu_random_comparison_summary.csv") %>% 
              mutate(model_type = "full_otu", 
                     train_model = study, 
                     act_mean_auc = as.numeric(act_mean_auc)) %>% 
              rename(AUC = act_mean_auc) %>% 
              select(pvalue, study, train_model, model_type, AUC)))

# Pvalue comparisons tables of the different individual taxa and full taxa models
stool_ind_vs_full_taxa <- read_csv("../data/process/tables/stool_ind_vs_full_taxa_pvalue_summary.csv")

unmatched_tissue_ind_vs_full_taxa <- read_csv("../data/process/tables/unmatched_tissue_ind_vs_full_taxa_pvalue_summary.csv")


```



# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Tumors

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that both individual and collections of bacteria are associated with the progression of colorectal cancer. As the number of studies investigating these associations increases and the number of subjects in each study increases, a meta-analysis to identify the associations that are the most predictive of disease progression is warranted. For our meta-analysis, we analyzed previously published 16S rRNA gene sequencing data collected from feces (`r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` individuals from `r length(stool_n$study)` studies) and colon tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals from `r length(tissue_n$study)` studies).

**Results.** We quantified the odds ratios for individual bacterial genera that were associated with an individual having tumors relative to a normal colon. Among the stool samples, there were no genera that had a significant odds ratio associated with adenoma and there were `r length(crc_OR_stool$measure)` genera with significant odds ratios associated with carcinoma. Similarly, among the tissue samples, there were no genera that had a significant odds ratio associated with adenoma and there were `r length(crc_unmatched_OR_tissue$measure)` genera with significant odds ratios associated with carcinoma. Among the significant odds ratios, the association between individual taxa and tumor diagnosis was equal or below `r ifelse(max(crc_OR_stool$rr) > max(crc_unmatched_OR_tissue$rr), round(max(crc_OR_stool$rr), digits = 2), round(max(crc_unmatched_OR_tissue$rr), digits = 2))`. Because individual taxa had limited association with tumor diagnosis, we trained Random Forest classification models using the genera with the five highest and lowest odds ratios, using the entire collection of genera found in each study, and using operational taxonomic units defined based on a 97% similarity threshold. All training approaches yielded similar classification success as measured using the  Area Under the Curve. The ability to correctly classify individuals with adenomas was poor and the ability to classify individuals with carcinomas was considerably better using sequences from stool or tissue.

**Conclusions.** This meta-analysis confirms previous results indicating that individuals with adenomas cannot be readily classified based on their bacterial community, but that those with carcinomas can. Regardless of the dataset, we found a subset of the fecal community that was associated with carcinomas was as predictive as the full community. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; tumor; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world-wide health problem in which the microbiota has been hypothesized to have a role in disease progression [@siegel_cancer_2016; @flynn_metabolic_2016]. Numerous studies using murine models of CRC have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in tumorigenesis. Numerous case-control studies have characterized the microbiota of individuals with colonic adenomas and carcinomas in an attempt to identify biomarkers of disease progression [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Because current CRC screening recommendations are poorly adhered to due to socioeconomic status, test invasiveness, and frequency of tests, development and validation of microbiome-associated biomarkers for CRC progression could further attempts to develop non-invasive diagnostics [@GimenoGarca2012].


Recently, there has been an intense focus on identifying microbiota-based biomarker yielding a seemingly endless number of candidate taxa. Some studies point towards mouth-associated genera such as Fusobacterium, Peptostreptococcus, Parvimonas, and Porphyromonas that are enriched in people with carcinomas [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Other studies have identified members of *Akkermansia*, *Bacteroides*, *Enterococcus*, *Escherichia*, *Klebsiella*, *Mogibacterium*, *Streptococcus*, and *Providencia* are also associated with carciomas  [@burns_virulence_2015; @chen_human_2012; @wang_structural_2012]. Additionally, *Roseburia* has been found in some studies to be more abundant in people with tumors but in other studies it has been found to be either less abundant or no different than what is found in subjects with normal colons  [@geng_diversified_2013; @wang_structural_2012; @flemer_tumour-associated_2017; @dejea_microbiota_2014]. There are strong results from tissue culture and murine models that Fusobacterium nucleatum, pks-positive strains of Escherichia coli, Streptococcus gallolyticus, and an entertoxin-producing strain of Bacteroides fragilis are important in the pathogenesis of CRC [@ecoli_Arthur_2014; @arthur_intestinal_2012; @strep_Aymeric_2017; @weir_stool_2013; @wang_structural_2012; @bfrag_Boleij_2014]. These results point to a causative role for the microbiota in CRC pathogenesis as well as their potential as diagnostic biomarkers.


Most studies have focused on identifying biomarkers in patients with carcinomas but there is a greater clinical need to identify biomarkers associated with adenomas. Studies focusing on broad scale community metrics have found that measures such as the total number of Operational Taxonomic Units (OTUs) are decreased in those with adenomas versus controls [@sanapareddy_increased_2012]. Other studies have identified *Acidovorax*, *Bilophila*, *Cloacibacterium*, *Desulfovibrio*, *Helicobacter*, *Lactobacillus*, *Lactococcus*, *Mogibacterium*, and *Pseudomonas* to be enriched in those with adenomas [@sanapareddy_increased_2012; @lu_mucosal_2016; @hale_shifts_2017]. There are few genera that are enriched in patients with adenoma or carcinoma tumors.


Confirming some of these previous findings, a recent meta-analysis found that 16S rRNA gene sequences from members of the *Akkermansia*, *Fusobacterium*, and *Parvimonas* were fecal biomarkers for the presence of carcinomas [@shah_leveraging_2017]. Contrary to previous studies they found sequences similar to members of *Lactobacillus* and *Ruminococcus* to be enriched in patients with adenoma or carcinoma relative to those with normal colons [@baxter_microbiota-based_2016; @burns_virulence_2015; @zeller_potential_2014]. In addition, they found 16S rRNA gene sequences from members of *Haemophilus*, *Methanosphaera*, *Prevotella*, *Succinovibrio* were enriched in patients with adenoma and *Pantoea* were enriched in patients with carcinomas. Although this meta-analysis was helpful for distilling a large number of possible biomarkers, the aggregate number of samples included in the analysis (n = 509) was smaller than several larger case-control studies that have since been published [@baxter_microbiota-based_2016; @hale_shifts_2017]


Here we provide an updated meta-analysis using 16S rRNA gene sequence data from both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and colon tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) from 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] [Table 1 & 2]. We expand both the breadth and scope of the previous meta-analysis to investigate whether biomarkers describing the bacterial community or specific members of the community can more accurately classify patients as having adenoma or carcinoma. Our results suggest that the bacterial community changes as disease severity worsens and that that a subset of the microbial community can be used to diagnose the presence of carcinoma.


\newpage

## Results

***Lower Bacterial Diversity is Associated with Increased Odds Ratio (OR) of Tumors:***
We first assessed whether variation in broad community metrics like total number of operational taxonomic units (OTUs) (i.e. richness), the evenness of their abundance, and the overall diversity was associated with disease stage after controlling for study and variable region differences. In stool, there was a significant decrease in both evenness and diversity as disease severity progressed from normal to adenoma to carcinoma (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))` and `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`, respectively) [Figure 1]; there was not a significant difference for richness (P-value = `r paste(format(filter(stool_lme, alpha_metric == "sobs")[, "BH"], digits = 2, nsmall = 2))`). We next tested whether the decrease in these community metrics translated into significant ORs for having an adenoma or carcinoma. For fecal samples, the ORs for richness were not significantly greater than 1.0 for adenoma or carcinoma (P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "sobs")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2A]. The ORs for evenness were significantly higher than 1.0 for adenoma (OR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) and carcinoma (OR =  `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2B]. The ORs for diversity were only significantly greater than 1.0 for carcinoma (OR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`), but not for adenoma (P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2C]. Although these OR are significantly greater than 1.0, it is doubtful that these are clinically meaningful values.

Similar to our analysis of sequences obtained from stool samples, we repeated the analysis using sequences obtained from colon tissue. There were no significant changes in richness, evenness, or diversity as disease severity progressed from control to adenoma to carcinoma (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05). We next analyzed the OR, for matched (i.e. where unaffected tissue and tumors were obtained from the same individual) and unmatched (i.e. where unaffected tissue and tumor tissue were not obtained from the same individual) tissue samples. The ORs for adenoma and carcinoma by any measure were not significantly different from 1.0 (P-value `r ifelse(min(alpha_RR_crc_unmatch_tissue$pvalue) < 0.05 & min(alpha_RR_crc_match_tissue$pvalue) < 0.05 & min(alpha_RR_adn_tissue$pvalue) < 0.05, invisible("check tables"), invisible(">"))` 0.05) [Figure S1 & Table S1]. This is likely due to the combination of a small effect size, as suggested from the results using stool, and the relatively small number of studies and size of studies used in the analysis.

***Disease Progression is Associated with Community-Wide Changes in Composition and Abundance:*** Based on the differences in evenness and diversity, we next asked whether there were community-wide differences in the structure of the communities associated with different disease stages. We identified significant bacterial community differences in the stool of patients with adenomas relative to those with normal colons in `r length(as.data.frame(filter(adn_beta_stool, pvalue < 0.05))[, "study"])` of `r length(as.data.frame(adn_beta_stool)[, "study"])` studies and in patients with carcinomas relative to those with normal colons in 6 of 7 studies (PERMANOVA; P-value < 0.05) [Table S2]. Similar to the analyses using stool samples, there were significant differences in bacterial community structure between subjects with normal colons and those with adenoma (1 of 2 studies) and carcinoma (1 of 3 studies) [Table S2]. For studies that used matched samples no differences in bacterial community structures were observed [Table S2]. Combined, these results indicate that there consistent and significant community-wide changes in the fecal community structure of subjects with carcinomas. However, the signal observed in subjects with adenomas or when using tissue samples was not as consistent. This is likely due to a smaller effect size or the relatively small sample sizes among the studies that characterized the tissue microbiota.


***Individual Taxa are Associated with Significant ORs for Carcinomas:*** 
Next we identified those taxa were associated with ORs that were significantly associated with having a normal colon or the presence of adenomas or carcinomas. No taxa had a significant OR for the presence of adenomas when we used data collected from stool or tissue samples (Table S3 & S4). In contrast, `r length(crc_OR_stool$measure)` taxa had significant ORs for the presence of carcinomas using data from stool samples. Of these, 4 are commonly associated with the oral cavity: Fusobacterium (OR = `r paste(round(filter(crc_OR_stool, measure == "Fusobacterium")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Fusobacterium")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Fusobacterium")[, "ci_ub"], digits = 2))`)), Parvimonas (OR = `r paste(round(filter(crc_OR_stool, measure == "Parvimonas")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Parvimonas")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Parvimonas")[, "ci_ub"], digits = 2))`)), Porphyromonas (OR = `r paste(round(filter(crc_OR_stool, measure == "Porphyromonas")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Porphyromonas")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Porphyromonas")[, "ci_ub"], digits = 2))`)), and Peptostreptococcus (OR = `r paste(round(filter(crc_OR_stool, measure == "Peptostreptococcus")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Peptostreptococcus")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Peptostreptococcus")[, "ci_ub"], digits = 2))`)) [Table S3]. The other 4 were Clostridium XI (OR = `r paste(round(filter(crc_OR_stool, measure == "Clostridium_XI")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Clostridium_XI")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Clostridium_XI")[, "ci_ub"], digits = 2))`)), Enterobacteriaceae  (OR = `r paste(round(filter(crc_OR_stool, measure == "Enterobacteriaceae_unclassified")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Enterobacteriaceae_unclassified")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Enterobacteriaceae_unclassified")[, "ci_ub"], digits = 2))`)), Escherichia (OR = `r paste(round(filter(crc_OR_stool, measure == "Escherichia.Shigella")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Escherichia.Shigella")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Escherichia.Shigella")[, "ci_ub"], digits = 2))`)), and Ruminococcus (OR = `r paste(round(filter(crc_OR_stool, measure == "Ruminococcus")[, "rr"], digits = 2))` (`r paste(round(filter(crc_OR_stool, measure == "Ruminococcus")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_OR_stool, measure == "Ruminococcus")[, "ci_ub"], digits = 2))`)). Among the data collected from tissue samples, only unmatched carcinoma samples had taxa with a significant OR. Those included Dorea (OR = `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "rr"], digits = 2))` (`r paste(round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "ci_ub"], digits = 2))`)), Blautia (OR = `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Blautia")[, "rr"], digits = 2))` (`r paste(round(filter(crc_unmatched_OR_tissue, measure == "Blautia")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Blautia")[, "ci_ub"], digits = 2))`)), and Weissella (OR = `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "rr"], digits = 2))` (`r paste(round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "ci_ub"], digits = 2))`)). Mouth-associated genera were not significantly associated with an increased OR for carcinoma in tissue samples [Table S4]. For example, Fusobacterium had an OR of `r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "rr"], digits = 2))` (`r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "ci_lb"], digits = 2))` - `r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "ci_ub"], digits = 2))`; however, due to the small number of studies and considerable variation in the data, the Benjimani-Hochberg-corrected P-value was `r paste(round(filter(crc_matched_OR_tissue, measure == "Fusobacterium")[, "bh"], digits = 2))` [Table S4]. It is interesting to note that Ruminococcus and members of Clostridium group XI in stool and Dorea and Blautia in tissue had ORs that were significantly less than 1.0, which suggests that these populations are protective against the development of carcinomas. Overall, there was no overlap in the taxa with significant OR between stool and tissue samples.


***Individual taxa with a significant OR do a poor job of differentiating subjects with normal colons and those with carcinoma:***
 We next asked whether those taxa that had a significant OR associated with having a normal colon or carcinomas could be used individually, to classify subjects as having a normal colon or carcinomas. Whereas the OR was defined based on whether the relative abundance for a taxon in a subject was above or below the median relative abundance for that taxon across all subjects in a study, we generated receiver operator characteristic (ROC) curves for each taxon in each study and calculated the area under the curve (AUC). This allowed us to use a more fluid relative abundance threshold for defining disease status. Using data from stool samples, the `r length(crc_OR_stool$measure)` taxa did no better at classifying the subjects than one would expect by chance (i.e. AUC=0.50) [Figure 3A]. The taxa that performed the best included Clostridium XI, Ruminococcus, and Escherichia and even these had median AUC values less than `r round(max(filter(ind_crc_combined_stool_auc, taxa %in% c("Escherichia", "Clostridium_XI", "Ruminococcus"))[, "auc"]), digits = 3)`. Likewise, in unmatched tissue samples the `r length(crc_OR_stool$measure)` taxa with significant ORs taxa were marginally better than one would expect by chance [Figure 3B]. The relative abundance of Dorea was the best predictor of carcinomas and its median AUC was only `r round(max(filter(ind_crc_combined_unmatched_auc, taxa == "Dorea")[, "auc"]), digits = 3)`. These results suggest that although these taxa are associated with a decreased or increased OR for the presences of carcinomas, individually, they do a poor job of classifying a subject’s disease status.


***Combined taxa model classifies subjects better than using individual taxa:*** 
Instead of attempting to classify subjects based on individual taxa, next we generated Random Forest models that combined the individual taxa and evaluated the ability to classify as subject’s disease status. For data from stool samples, the combined model had an AUC of `r round(median(filter(crc_stool_model_aucs, study == train_model, model_type == "full_model")[, "AUC"]), digits = 2)`, which was significantly higher than any of the AUC values for the individual taxa (P-value < `r round(max(stool_ind_vs_full_taxa$pvalue), digits = 3)`). For the full taxa models using stool, *Bacteroides* and *Lachnospiraceae* were the most common taxa in the top 10% mean decrease in accuracy (MDA) across studies [Figure S2]. Similarly, using data from the unmatched tissue samples, the combined model had an AUC of `r round(median(filter(crc_tissue_model_aucs, study == train_model, model_type == "full_model", type == "unmatched")[, "AUC"]), digits = 2)`, which was significantly higher than the AUC values for Blautia and Weissella (P-value < `r round(max(filter(unmatched_tissue_ind_vs_full_taxa, taxa != "Dorea")[, "pvalue"]), digits = 3)`).  For the full taxa models using unmatched tissue, *Lachnospiraceae*, *Bacteroidaceae*, and *Ruminococcaceae* were the most common taxa in the top 10% mean decrease in accuracy across studies [Figure S3]. Clearly, pooling the information from the taxa with significant ORs results in a model that outperforms classifications made using individual taxa.


***Performance of models based on taxa relative abundance in full community are better than those based on taxa with significant ORs:*** 
Next, we asked whether a Random Forest classification model built using all of the taxa found in the communities would outperform the models generated using those taxa with a significant OR. Similar to our inability to identify taxa associated with a significant OR for the presence of adenomas, the median AUCs to classify subjects as having normal colons or having adenomas using data from stool or tissue samples were marginally better than 0.5 for any study[Figure 4A & S4A]. In contrast, the models for classifying subjects as having normal colons or having carcinomas using data from stool or tissue samples yielded AUC values meaningfully higher than 0.5 [Figure 4B & S4B-C]. When we compared the models based on all of the taxa in a community to models based on the taxa with significant ORs, the results were mixed. Using the data from stool samples we found that although the AUC for 6 of 7 studies increased (mean decrease = `r round(mean((filter(crc_stool_model_aucs, study == train_model, model_type == "full_model", study != "flemer")[, "AUC"] - filter(crc_stool_model_aucs, study == train_model, model_type == "select_model", study != "flemer")[, "AUC"])*100), digits = 2)`%), the more expansive models performed worse for 1 of the studies (decrease = `r round(abs(mean((filter(crc_stool_model_aucs, study == train_model, model_type == "full_model", study == "flemer")[, "AUC"] - filter(crc_stool_model_aucs, study == train_model, model_type == "select_model", study == "flemer")[, "AUC"])*100)), digits = 2)`%). The overall improvement in performance was statistically significant (one-tailed paired T-test; P-value = `r round(t.test(filter(crc_stool_model_aucs, study == train_model, model_type == "full_model")[, "AUC"], filter(crc_stool_model_aucs, study == train_model, model_type == "select_model")[, "AUC"], alternative = "greater", paired = TRUE)$p.value, digits = 3)`). Of the `r length(crc_OR_stool$measure)` taxa with significant ORs, all `r length(crc_OR_stool$measure)` were among the top 10% most important taxa as measured by mean decrease in accuracy, in at least one study . Similarly, using the data from unmatched tissue samples we found that the AUC for 4 out of 4 studies decreased between full versus select OR models (mean decrease = `r round(mean((filter(crc_tissue_model_aucs, study == train_model, model_type == "full_model", type == "unmatched")[, "AUC"] - filter(crc_tissue_model_aucs, study == train_model, model_type == "select_model", type == "unmatched")[, "AUC"])*100), digits = 2)`%, one-tailed paired T-test; P-value = `r round(t.test(filter(crc_tissue_model_aucs, study == train_model, model_type == "full_model", type == "unmatched")[, "AUC"], filter(crc_tissue_model_aucs, study == train_model, model_type == "select_model", type == "unmatched")[, "AUC"], alternative = "greater", paired = TRUE)$p.value, digits = 3)`). Of the `r length(crc_unmatched_OR_tissue$measure)` taxa with significant ORs, all `r length(crc_unmatched_OR_tissue$measure)` were among the top 10% most important taxa as measured by mean decrease in accuracy, in at least one study. The most important taxa across study within the significant OR taxa only models for stool were *Ruminococcus* and *Clostridium XI* [Figure 5A]. For the singificant OR taxa unmatched tissue models both *Dorea* and *Blautia* were the important based on mean decrease in accuracy [Figure 5B]. These results were surprising because it demonstrated that the ability to classify subjects could be done based on a limited characterization of the communities.



***Performance of models based on OTU relative abundance in full community are not significantly better than those based on taxa with significant ORs:*** 
The previous models were based on relative abundance data where sequences were assigned to coarse taxonomic assignments (i.e. typically genus or family level). To determine whether model performance improved with a more fine scale classification, we assigned sequences to operational taxonomic units (OTUs) where the similarity among sequences within an OTU was more than 97%. We again found that classification models built using all of the sequence data for a community did a poor job of differentiating between subjects with normal colons and those with adenomas (median AUC: `r round(median(filter(adn_stool_model_aucs, model_type == "full_otu")[, "AUC"]), digits = 2)` [`r round(min(filter(adn_stool_model_aucs, model_type == "full_otu")[, "AUC"]), digits = 2)`- `r round(max(filter(adn_stool_model_aucs, model_type == "full_otu")[, "AUC"]), digits = 2)`]), but did a good job of differentiating between subjects with normal colons and those with carcinomas (median AUC: `r round(median(filter(crc_stool_model_aucs, model_type == "full_otu")[, "AUC"]), digits = 2)` [`r round(min(filter(crc_stool_model_aucs, model_type == "full_otu")[, "AUC"]), digits = 2)`- `r round(max(filter(crc_stool_model_aucs, model_type == "full_otu")[, "AUC"]), digits = 2)`]). The OTU-based models performed similarly to those constructed using the taxa with significant ORs (one-tailed paired T-test; P-value = `r round(t.test(filter(crc_tissue_model_aucs, study == train_model, model_type == "select_model", type == "unmatched")[, "AUC"], filter(crc_tissue_model_aucs, study == train_model, model_type == "full_otu", type == "unmatched")[, "AUC"], alternative = "greater", paired = TRUE)$p.value, digits = 3)`) and those using all taxa (one-tailed paired T-test; P-value = `r round(t.test(filter(crc_tissue_model_aucs, study == train_model, model_type == "full_model", type == "unmatched")[, "AUC"], filter(crc_tissue_model_aucs, study == train_model, model_type == "full_otu", type == "unmatched")[, "AUC"], alternative = "greater", paired = TRUE)$p.value, digits = 3)`). Among the OTUs that had the highest mean decrease in accuracy for the OTU-based models, we found that OTUs that affiliated with all of the `r length(crc_OR_stool$measure)` taxa that had a significant OR were within the top 10% for at least one study. Again, this result was surprising as it indicated that a finer scale classification of the sequence data and thus a larger number of features to select from, did not yield improved classification of the subjects.


***Generalizability of taxon-based models trained on one dataset to the other datasets:*** 
Considering the good performance of the Random Forest models using taxa with a significant OR and using all of the taxa, we next asked how well the models would perform when given data from a different subject cohort. For instance, if a model was trained using data from the Ahn study, we wanted to know how well it would perform using the data from the Baxter study. We found the models trained using the taxa with a significant OR all had a higher median AUC than the models trained using all of the taxa when tested on the other datasets [Figure 6 & S5]. As might be expected, the difference between the performance of the modelling approaches appeared to vary with the size of the training cohort [Figure 6]. These data suggest that given a sufficient number of subjects with normal colons and carcinomas, Random Forest models trained using a small number of taxa can accurately classify individuals from a different cohort.




\newpage

## Discussion

Although we expected that the full OTU models would perform the best at classifying individuals with and without carcinomas, our observations suggest that both the full and significant OR taxa models performed equally well. These results suggest that lower level classification to species and strain may not add extra useful information with respect to prediction models. This has been suggested in previous literature where metagenomics did not perform better than 16S rRNA gene sequencing data at classifying individuals with normal colons and those with carcinomas [@Hannigan2017]. One possible reason as to why lower level classification may not result in better models is that the communities are patchy and higher level taxonomic information pools some of this patchiness, allowing for better prediction models. There may also be a fair bit of data redundancy within models that utilize more of the community. An example of this redundancy would be when we trained models on one study and tested it on the other studies and the AUCs of the models created with the select OR taxa performed as well as full taxa models [Figure 6B].   

Our observations also suggest that a small collection of taxa can classify disease as well as full OTU-based models but that these taxa individually perform quite poorly [Figure 3]. This result supports the contention that there might be redundancy of function amongst the taxa included in the significant OR models. As an example, multiple different microbes could be similarily stimulating the activation of inflammatory pathways and by doing so exacerbate disease progression. Multiple reports within the literature have found that different bacteria, such as *Escherichia coli* and *Fusobacterium nucleatum*, can similarily worsen inflammation in mouse models of tumorigenesis [@ecoli_Arthur_2014; @arthur_intestinal_2012; @kostic_fusobacterium_2013]. Although the inflammatory taxa were patchy in their importance and presence across studies those that were not typically associated with inflammation were consistently important for every study [Figure 6]. The loss of these taxa (*Ruminococcus* and *Clostridium XI* in stool and *Dorea* and *Blautia* in unmatched tissue) is particularly interesting because many are commonly thought to be beneficial due to their involvement in production of short chain fatty acids [@Venkataraman2016; @Herrmann2018; @Reichardt2017].   

The adenoma models as a whole performed poorly in classifying individuals with and without adenomas. This outcome is not inconsistent with what has been published previously [@hale_shifts_2017; @Sze2017]. However, the modeling results are at odds with results obtained in Baxter, *et al.* [@baxter_microbiota-based_2016]. There are some major differences between the models generated in this meta-analysis and what was used in Baxter, *et al*. First, the prevoius report's models investigated the classification of lesions (individuals with adenoma or carcinoma) and not adenoma alone. The Baxter, *et al.* models also contained Fecal Immunoglobulin Test data while our meta-analysis models only contained 16S rRNA gene sequencing data. Although being able to classify individuals with adenomas is important, the classification of advanced adenomas is a more clinically meaningful diagnostic tool (i.e. those that are at high risk of progressing to a carcinoma). It is possible that we might have been able to detect differences in the bacterial community if advanced adenomas were separated from adenomas but that data was not available for the majority of studies analyzed. Additionally, the initial changes to the bacterial community could be focal to where the initial adenoma develops and would not be easily assessed with a fecal sample.    

Although stool represents an easy and less invasive way to assess risk, it is not clear how well this sample reflects adenoma- and carcinoma- tissue associated microbial communities. The colon tissue-based studies did not provide a clearer understanding of how the microbiota may be associated with tumors. Generally, the full OTU-based models of unmatched and matched colon tissue samples were concordant with stool samples showing that GI resident microbes were the most prevalent in the top 10 most important variables across study [Figure S3]. *Fusobacterium* was not consistently identified across studies and this could be due to both a small number of studies and a small sample size within these studies. The majority of the colon tissue-based results were also study specific with many of the top 10 taxa being present only in a single study. Additionally, the presence of genera associated with contamination [@Salter_contamination_2014], within the top 10 most important variables for the genera and OTU models is worrying (e.g. *Novosphingobium*, *Acidobacteria Gp2*, *Sphingomonas*, etc.). The low bacterial biomass of tissue samples coupled with potential contamination and small sample sizes could explain why these results seem to be more sporadic than the stool results.

One important caveat to this study is that even though genera associated with certain species such as *Bacteroides fragilis* and *Streptococcus gallolyticus* subsp. *gallolyticus* were not identified, it does not necessarily mean that these specific species are not important in human CRC [@bfrag_Boleij_2014; @strep_Aymeric_2017]. There are reports that *Bacteroides fragilis*, positive for the enterotoxigenic gene, are found at specific locations along the colon but the samples we were able to use in this meta-analysis could not identify these types of differences [@Purcell2017]. Additionally, since we are limited in our aggregation of the data to the genus level, it is not possible to clearly delineate which species are contributing to overall disease progression. Our observations are not inconsistent with the previous literature on either *Bacteroides fragilis* or *Streptococcus gallolyticus* subsp. *gallolyticus*. As an example, the stool-based full community models consistently identified the *Bacteroides* taxa to be important model components across studies. This suggests that even though *Bacteroides* may not increase the OR of individuals having an adenoma or carcinoma and may not vary in relative abundance, like *Fusobacterium*, it is still important in CRC. 

Despite these limitations to the findings, meta-analyses are a useful tool in microbiota research because they can both validate existing research and make new discoveries by pooling many independent investigations together. Yet, it is still difficult to perform these studies because of inaccessible 16S sequencing data, missing or vague metadata (e.g. which samples are carcioma and which are not), varying sequence quality, and multiple small data sets. Better attention to these specific problems could help to increase the reproducibility and replicability of microbiota studies and make it easier to perform these crucial meta-analyses. Moving forward, meta-analyses will be important tools to help aggregate and find commonalities across studies when investigating the microbiota in the context of a specific disease and more are needed [@Sze2016; @Walters2014; @Finucane2014; @shah_leveraging_2017]. 

By aggregating together a large collection of studies analyzing both fecal and colon tissue samples, we are able to provide evidence supporting the importance of the bacterial community in carcinoma tumors. Although further validation of the biomarkers presented here need to be undertaken, the replicability of the AUC of a specific collection of taxa across multiple studies suggests a strong potential for the use of the microbiota as a risk stratification tool for individuals with carcinomas. 



\newpage

## Methods

***Obtaining Data Sets:***
The studies used for this meta-analysis were identified through the review articles written by Keku, *et al.* and Vogtmann, *et al.* [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016] and additional studies not mentioned in the reviews were obtained based on the authors' knowledge of the literature. Studies that used tissue or feces as their sample source for 454 or Illumina 16S rRNA gene sequencing analysis and had data sets with sequences available for analysis were included. Some studies were excluded because they did not have publicly available sequences or did not have metadata in which the authors were able to share. After these filtering steps, the following studies remained: Ahn, *et al.* [@ahn_human_2013], Baxter, *et al.* [@baxter_microbiota-based_2016], Brim, *et al.* [@brim_microbiome_2013], Burns, *et al.* [@burns_virulence_2015], Chen, *et al.*  [@chen_human_2012], Dejea, *et al.* [@dejea_microbiota_2014], Flemer, *et al.* [@flemer_tumour-associated_2017], Geng, *et al.* [@geng_diversified_2013], Hale, *et al.* [@hale_shifts_2017], Kostic, *et al.* [@kostic_genomic_2012], Lu, *et al.* [@lu_mucosal_2016], Sanapareddy, *et al.* [@sanapareddy_increased_2012], Wang, *et al.* [@wang_structural_2012], Weir, *et al.* [@weir_stool_2013], and Zeller, *et al.*  [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included because the 90 individuals analyzed within the study are contained within the larger Baxter study [@baxter_microbiota-based_2016]. After sequence processing, all the case samples for the Kostic study had 100 or less sequences remaining and was excluded, leaving a total of 14 studies that analysis could be completed on.

***Data Set Breakdown:***
In total, there were seven studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), five studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].  

***Sequence Processing:***
For the majority of studies, raw sequences were downloaded from the Sequence Read Archive (SRA) (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata were obtained by searching the respective accession number of the study at the following website: http://www.ncbi.nlm.nih.gov/Traces/study/. Of the studies that did not have sequences and metadata on the SRA, data was obtained from DBGap (n = 1, [@ahn_human_2013]) and directly from the authors (n = 4, [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]). Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009] and quality filtering utilized the default methods for both 454 and Illumina based sequencing. If it was not possible to use the defaults, the stated quality cut-offs, from the study itself, were used instead. Sequences that were made up of an artificial combination of two or more different sequences and commonly known as chimeras were identified and removed using VSEARCH [@rognes_vsearch_2016] before *de novo* OTU clustering at 97% similarity was completed using the OptiClust algorithm [@westcott_opticlust_2017]. 


***Study Analysis Overview:***
OTU richness, evenness, and Shannon diversity were first assessed for differences between controls, adenoma tumors, and carcinoma turmors using both linear mixed-effect models and ORs. For each individual study the Bray-Curtis index was used to assess differences between control-adenoma and control-carcinoma individuals. Next, all common genera were assessed for differences in ORs for individuals having an adenoma or carcinoma and corrected for multiple comparisons using the Benjamini-Hochberg method [@benjamini_controlling_1995]. We then built Random Forest models based on all genera, all OTUs, or significant OR taxa (only using taxa still significant after multiple comparison correction). For both the full genera and significant OR taxa, models were trained on one study then tested on the remaining studies using genera-based relative abundances. The OTU-based models were built using OTU level data and a 10-fold CV over 100 different iterations, based on random 80/20 splitting of the data, was used to generate a range of expected AUCs. This process was repeated for every study in the meta-analysis. Comparisons of the initial trained model AUCs for the full genera and significant OR taxa were made to the mean AUC generated from the 100 different 10-fold CV runs of the respective OTU-based model. For comparisons in which only control versus adenoma individuals were made, the carcinoma individuals were excluded from each respective study. Similarly, for comparisons in which control versus carcinoma individuals were made the adenoma individuals were excluded from each respective study. For all analysis completed fecal and tissue samples were kept separate. Within the tissue groups the data were further divided between samples from the same individual (matched) and those from different individuals (unmatched).

***Obtaining Genera Relative Abundance and Significant OR Taxa Models:***
For the genera analysis of the ORs, OTUs were added together based on the genus or lowest available taxonomic classification level and the total average counts, for 100 different subsamplings was obtained. The significant OR taxa models for the  Random Forest models utilized all taxa that had significant ORs after multiple comparison correction. This meant only models for the carcinoma stool (`r length(crc_OR_stool$measure)` variables) and carcinoma unmatched (`r length(crc_unmatched_OR_tissue$measure)` variables) samples were possible to be created and analyzed. 


***Matched versus Unmatched Tissue Samples:***
In general, tissue samples with control and tumor samples from different individuals were classified as unmatched while samples that belonged to the same individual were classified as matched. Studies with matched data included Burns, Dejea, Geng, and Lu while those with unmatched data were from Burns, Flemer, Chen, and Sanapareddy. For some studies samples became unmatched when a corresponding matched sample did not make it through sequence processing. All samples, from both matched and unmatched tissue samples, were analyzed together for the linear mixed-effect models with samples from the same individual being corrected for. All other analysis, where it is not specified explicitly, matched and unmatched samples were analyzed separately using the statistical approaches mentioned in the Statistical Analysis section. 

***Assessing Important Random Forest Model Variables:***
Using Mean Decrease in Accuracy (MDA) the top 10 most important variables to the Random Forest model were obtained for the full models of the two different approaches used. For the first approach utilizing genus-based models, the number of times that a specific taxa showed up in the top 10 of the training set across each study was counted. For the second approach, that utilized the OTU-based models, the medians for each OTU across 100 different 80/20 splits of the data was generated and the top 10 OTUs then counted for each study. Common taxa were then identified by using the lowest classification for each of the specific OTUs obtained from these counts and the number of times this classification occurred across the top 10 of each study was recorded. Finally, the two studies that had adenoma tumor tissue (Lu and Flemer) were equally divided between matched and unmatched studies and were grouped together for the counting of the top 10 genera and OTUs for both Random Forest approaches.


***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R (v`r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. For OTU richness, evenness, and Shannon diversity analysis, values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package [@rcompanion_citation_2017] and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package [@car_citation_2011]. Testing for OTU richness, evenness, and Shannon diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package [@lme4_citation_2015] to correct for study, repeat sampling of individuals (tissue only), and 16S hyper-variable region used. Odds ratios (OR) were analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages [@epir_citation_2017; @metafor_citation_2010] by assessing how many individuals with and without disease were above and below the overall median value within each specific study. OR significance testing utilized the chi-squared test. Diversity differences measured by the Bray-Curtis index utilized the creation of distance matrix and testing with PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package [@vegan_citation_2017]. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages [@caret_citation_2017; @randomforest_citation_2002]. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages [@ggplot2_citation_2009; @gridextra_citation_2017]. 


***Reproducible Methods:***
The code and analysis can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless otherwise mentioned, the accession number of raw sequences from the studies used in this analysis can be found directly in the respective batch file in the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscripts used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective study batch file in the GitHub repository or within the original publication. For instances when sequences are not publicly available, they may be accessed by contacting the corresponding authors from whence the data came.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a Canadian Institute of Health Research fellowship and a University of Michigan Postdoctoral Translational Scholar Program grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally, we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Total Individuals in each Study Included in the Stool Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: Significant Bacterial Community Metrics for Adenoma or Carcinoma in Stool.** A) Adenoma evenness. B) Carcinoma evenness. C) Carcinoma Shannon diversity. Blue represents controls and red represents either adenoma (panel A) or carcinoma (panel B and C). The black lines represent the median value for each repsective group. 

**Figure 2: Odds Ratio for Adenoma or Carcinoma based on Bacterial Community Metrics in Stool.** A) Community-based odds ratio for adenoma. B) Community-based odds ratio for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 3: The AUC of Indivdiual Significant OR Taxa to classify Carcinoma.** A) Stool samples. B) Unmatched tissue samples. The larger circle represents the median AUC of all studies and the smaller circles represent the individual AUC for a particular study. The dotted line denotes an AUC of 0.5. 

**Figure 4: Stool Random Forest Model Train AUCs.** A) Adenoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa. B) Carcinoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa. The black line represents the median AUC for the respective group. If no values are present in the singificant OR taxa group then there were no significant taxa identified and no model was tested.

**Figure 5: Most Important Members in Significant OR Taxa Carcinoma Models.** A) Common taxa in the top 10 percent for carcinoma Random Forest stool-based models. B) Common taxa in the top 10 percent for carcinoma Random Forest unmatched tissue-based models. Blue represents less important and red represents more important to the Random Forest Model. White means that particular taxa was not in the top 10%.

**Figure 6: Stool Random Forest Genus-Based Model Test AUCs.** A) Test AUCs of adenoma models using all genera across study. B) Test AUCs of carcinoma models using all genera or significant OR taxa only. The black line represents the AUC at 0.5. The red lines represent the median AUC of all test AUCs for a specific study.  





\newpage


**Figure S1: Odds Ratio for Adenoma or Carcinoma based on Bacterial Community Metrics in Tissue.** A) Community-based odds ratio for adenoma. B) Community-based odds ratio for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Most Common Taxa Across Carcinoma Full Community Stool Study Models.** A) Common taxa in the top 10 percent for carcinoma Random Forest all taxa-based models. B) Common taxa in the top 10 percent for carcinoma Random Forest all OTU-based models. Blue represents less important and red represents more important to the Random Forest Model. White means that particular taxa was not in the top 10%.

**Figure S3: Most Common Genera Across Full Community Tissue Study Models.** A) Common genera in the top 10 percent for matched carcinoma Random Forest all genera-based models. B) Common genera in the top 10 percent for unmatched carcinoma Random Forest all genera-based models. C) Common genera in the top 10 percent for matched carcinoma Random Forest all OTU-based models. D) Common genera in the top 10 percent for unmatched carcinoma Random Forest all OTU-based models. Blue represents less important and red represents more important to the Random Forest Model. White means that particular taxa was not in the top 10%.

**Figure S4: Tissue Random Forest Model Train AUCs.** A) Adenoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa in unmatched and matched tissue. B) Carcinoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa in unmatched and matched tissue. The black line represents the median AUC for the respective group. If no values are present in the singificant OR taxa group then there were no significant taxa identified and no model was tested.

**Figure S5: Tissue Random Forest Genus-Based Model Test AUCs.** A) Test AUCs of adenoma models using all genera across study. B) Test AUCs of carcinoma models using all genera for matched tissue studies. C) Test AUCs of carcinoma models using all genera or significant OR taxa only for unmatched tissue studies The black line represents the AUC at 0.5. The red lines represent the median AUC of all test AUCs for a specific study.

\newpage
