---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in relative risk alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_RR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_tissue_composite.csv"))

#Load in relative risk select genera data tables
select_RR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_stool_composite.csv"))
select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_tissue_composite.csv"))

adn_select_RR_stool <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_stool_composite.csv"))
adn_select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_tissue_composite.csv"))

select_inc_4_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_inc_4_stool.csv"))
select_inc_4_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_inc_4_tissue.csv"))

adn_select_inc_4_RR_stool <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_inc_4_stool.csv"))
adn_select_inc_4_RR_tissue <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_inc_4_tissue.csv"))

```



# The Microbiota and Individual Community Members in Colorectal Cancer: Is There a Common Theme?

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that there is a role for the microbiota in colorectal cancer. Important drivers within this axis have ranged from individual microbes to the whole community. A recent meta-analysis investigated whether consistent biomarkers could be identified across studies. This report expands on this previous research and tests the hypothesis that the bacterial community is an important driver of disease from control to adenoma to carcinoma. To test this hypothesis we examined both feces (total individuals = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and tissue (total samples = `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)`) across 14 different studies.

**Results.** Fecal samples had a significant decrease from control to adenoma to carcinoma for both Shannon diversity and evenness (P-value `r ifelse(min(filter(stool_lme, !(alpha_metric %in% "sobs"))[, "BH"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) after correcting for study effect and variable region sequenced. Lower Shannon diversity and evenness in fecal samples resulted in a significant increase in relative risk for carcinoma (P-value `r ifelse(min(filter(alpha_RR_crc_stool, !(measure %in% "sobs"))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) while only evenness for adenoma (P-value `r ifelse(min(filter(alpha_RR_adn_stool, !(measure %in% "sobs"))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) resulted in a slightly increased relative risk. Previously associated colorectal cancer genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*) followed a similar pattern with a significantly increased relative risk by their presence for carcinoma (P-value `r ifelse(min(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) but not adenoma (P-value `r ifelse(min(filter(adn_select_RR_stool, !(measure %in% c("all_four", "total_four", "Porphyromonas")))[, "pvalue"]) > 0.05, invisible(">"), invisible("check tables"))` 0.05) with the exception of *Porphyromonas* (P-value `r ifelse(min(filter(adn_select_RR_stool, measure == "Porphyromonas")[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05). Using the whole community versus only CRC associated genera to build a prediction model resulted in a higher classification AUC for both adenoma and carcinoma for fecal and tissue samples. For the studies that were analyzed, most were adequately powered for large effect size differences which may be more amenable for carcinoma than for adenoma microbiota research.     

**Conclusions.** This data provides support for the importance of the bacteral community to both adenoma and carcinoma genesis. The evidence that has been collated within this study is much stronger for carcinoma and this may be in part due to the low power to detect more subtle changes in the majority of studies that have been performed to date. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world wide health problem [@siegel_cancer_2016] in which the microbiota has been purported to play an active role in disease pathogenesis [@flynn_metabolic_2016]. Numerous studies have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in polyp formation in mouse models. There has also been numerous case/control studies investigating the microbiota in both adenoma and carcinoma. Recently, a meta-analysis was published investigating whether specific biomarkers could be consistently identified using multiple data sets [@shah_leveraging_2017]. Many of the studies along with the current meta-analysis focus on identifying biomarkers or individual microbes but do not critically investigate the community role in the disease. 

Using both fecal (total individuals = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and tissue samples (total samples = `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)`) totalling over `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T) + sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples across 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] we expand both the breadth and scope of the previous meta-analysis; to investigate whether the bacterial community is an important risk factor for both adenoma and carcinoma. To accomplish this we first assessed whether the diversity changes across disease (control to adenoma to carcinoma) and if it results in an increased relative risk for adenoma or carcinoma. Next, we assessed how typical CRC associated genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*) affect the relative risk of adenoma or carcinoma. Third, using Random Forest models we analyzed whether the full community or only the CRC associated genera resulted in better model classification area under the curve (AUC). Finally, we examined at what effect size and sample n the studies that were used were adequately powered for.

Our analysis found a continuous decrease in Shannon diversity from control to adenoma to carcinoma and a significantly increased relative risk for carcinoma with this lower diversity. Using only the CRC associated genera this relative risk was higher than what was observed for Shannon diversity. However, by using the full community instead of only the CRC associated genera the AUC of the classification models increased. Although we analyze a data set with a large number of total individuals each individual study was underpowered for effect size differences of 10% or below between the case and control group.   

\newpage

## Results

***Diversity is Lower in Those with Carcinoma and Increases Relative Risk:***
Using power transformed and Z-score normalized alpha diversity metrics, both evenness and Shannon diversity in feces and not tissue were  lower in those with carcinoma [Figure 1]. Using linear mixed-effect models to control for study and variable region there was a significant decrease from control to adenoma to carcinoma for both evenness (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))`) and Shannon diversity (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`). This effect was not observed in tissue when additionally controlling for whether the sample came from the same individual (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05). For fecal samples a decrease in Shannon diversity and evenness resulted in a significantly increased relative risk for carcinoma (P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))` and P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`, respectively) [Figure 2]. Although these values were significant, the effect size was relatively small for both metrics (Shannon RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` and evenness RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))`) [Figure 2]. Only a decrease in evenness had an increased relative risk for adenoma (P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2A & S1] but this effect size was even smaller than what was observed for carcinoma (RR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))`).

Using the Bray-Curtis distance metric, there was a significant difference across studies in the bacterial community of fecal samples between carcinoma and controls but not between adenoma and controls [Table S1 & S2]. Many studies with unmatched tissue samples had a significant difference for both adenoma and carcinoma versus controls [Table S3 & S4] while studies with matched tissue samples had no differences [Table S3 & S4].

***Genera Previously Associated with Carcinoma Predominately Increases Relative Risk for Carcinoma but not Adenoma:***
The majority of CRC associated genera for both feces and tissue had a significantly increased relative risk for carcinoma but not for adenoma [Figure 3]. In fecal samples the relative risk due to CRC associated genera was greater than either evenness or Shannon diversity [Figure 2 & 3]. Additionally, the relative risk of carcinoma continuously increased as individuals tested positive for more CRC associated genera [Figure 3B & 3D]. The relative risk effect size was greater for stool (RR range = `r paste(round(min(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))` - `r paste(round(max(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))`) then for tissue (RR range = `r paste(round(min(filter(select_RR_tissue, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))` - `r paste(round(max(filter(select_RR_tissue, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))`). This decrease may be explained by the fact that the tissue analysis include matched samples. 

Two measures in stool for adenoma were significant when investigating these CRC associated genera. The first was *Porphyromonas* (P-value = `r format(filter(adn_select_RR_stool, measure == "Porphyromonas")[, "pvalue"], digits = 2, nsmall = 2)`) and the second was being positive for three CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_stool, measure == "three_counts")[, "pvalue"], digits = 2, nsmall = 2)`) [Figure 3A]. For tissue three measures for adenoma were significant; being positive for one CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_tissue, measure == "one_counts")[, "pvalue"], digits = 2, nsmall = 2)`), being positive for two CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_tissue, measure == "two_counts")[, "pvalue"], digits = 2, nsmall = 2)`), and being positive for four CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_tissue, measure == "four_counts")[, "pvalue"], digits = 2, nsmall = 2)`) [Figure 3C].  


***Using the Whole Community Increases Model AUC over CRC Associated Genera:***
For both fecal and tissue samples (matched and unmatched) the AUC decreases when only OTUs from the CRC associated genera are used [Figure 4 & 5]. This decrease is observed in both adenoma and carcinoma groups [Figure 4 & 5]. The genus models generally had similar trends as observed for the OTU based models with the full genera models performing better then the CRC associated genera models [Figure S2-S3]. Both genus models perform similarily in their ability to be able to predict lesion (adenoma or carcinoma) with carcinoma having a higher AUC then adenoma [Figure S4-S5]. Matched tissue samples for those with carcinoma had an AUC that was more similar to the adenoma models [Figure S4A, S5B, & S6] then carcinoma models [Figure S4B & S5A].    


***Majority of Studies are Underpowered for Detecting Small Effect Size Differences:***
When assessing the power of each study at different effect sizes the majority of studies for both adenoma and carcinoma have an 80% power to detect a 30% difference [Figure 6A & B]. No single study that was analyzed had the standard 80% power to detect an effect size difference that was eqaul to or below 10% [Figure 6A & B].  In order to achieve adequate power for small effect sizes it would be necessary to recruit over 1000 individuals for each arm of the study [Figure 6C]. 



\newpage

## Discussion

Our study identifies clear diversity changes both at the community level and within individual genera that are present in individuals with carcinoma versus those without the disease. Although there was a step wise decrease in diversity from control to adenoma to carcinoma; this did not translate into large effect sizes for the relative risk of either of these two conditions. Even though CRC associated genera increases the relative risk of carcinoma it does not consistently increase the relative risk of adenoma. This information suggests that these specific genera are important in carcinoma genesis but may not be the primary members of the microbial community that contribute to the formation of an adenoma. Additionally, our data shows that by using the whole community our models perform better then when they only use the CRC associated genera. CRC associated genera are clearly important to carcinoma but the context or community in which these microbes are a part of can drastically increase the ability of models to make predictions. This data supports the concept that small localized changes within the community may be occuring that are important in the disease progression of colorectal cancer and that they may not directly involve CRC associated genera.    


The driver-passenger model of the microbial role in CRC, as summarized by Flynn [@flynn_metabolic_2016], can be supported with this data for carcinoma but not necessarily for adenoma. The drasitically increased relative risk of disease, when considering the CRC associated genera, is highly supportive of this type of process, especially in the context of increasing relative risk with more CRC associated genera positivity. It is also possible that in a driver-passenger scenario that simply having the driver present or only identifying the passenger is a good enough proxy that the event is occuring. This would account for the observation that there is no constant additive effect on relative risk for increasing positivity. Additionally, the initial establishment of the driver within the system is also dependent on the community that is present and this is supported by the observation that when adding the community context to our models along with the CRC associated genera the model AUC increases. 


Our carcinoma observations fit the driver-passenger model and support this concept within the framework of the transition from adenoma to carcinoma. In contrast, with the present data we can only suggest that the adenoma observations might fit with this model but the changes that occur at this timepoint are small and possibly focal to the adenoma. The stepwise decrease in diversity suggests that the adenoma community is not normal but this change is subtle. Although there may be localized changes that do depend on the driver-passenger model, supported by an increased relative risk for one, two, and four positive CRC associated genera in tissue [Figure 3C], there may be other processes involved that ultimately exacerbates the condition from a subtle localized change to a global community one. The poor performance of the Random Forest models for classifying adenoma based only on the microbiota would suggest that this is the case. It is possible to hypothesize that at early stages of the diease, how the host interacts to these subtle changes could be the catlyst that ultimately leads to this larger global dysfunctional community. 


Although there are still questions that need to be answered for the microbiota and carcinoma, a clearer framework is beginning to develop as to how this occurs. The role of the microbiota in adenoma is still not clear and part of the reason this may be is because many studies are not powered effectively to observe the small changes reported here. It is realistic to suspect that many changes in carcinoma could easily result in effect sizes that are 30% or more between the case and control. Most of the studies analyzed have sufficient power to detect these type of changes. In contrast, our data suggests that the adenoma effect size is relatively small. None of the studies analyzed were properly powered to detect a 10% or lower change between case and controls and this may well be the range in which differences consistently occur in adenoma. Future studies investigating adenoma and the microbiota need to take these factors into consideration if we are to work out the role of the microbiota in adenoma formation.     

## Conclusion

By aggregating together a large collection of studies from both feces and tissue we are able to provide information in support of the importance of the bacterial community in both adenoma and carcinoma. We are also able to provide support for the driver-passenger model in the context of carcinoma. However, within the context of adenoma it is less clear that this relationship exists. These observations highlight the importance of power and sample number considerations when considering investigations into the microbiota and adenoma due to the subtle changes in the community. 

\newpage

## Methods

***Obtaining Data Sets:***
Studies used for this meta-analysis were identified through the review articles written by Keku, et al. and Vogtmann, et al. [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016]. Additional studies not mentioned in the reviews were obtained from the authors knowledge of the literature. All studies were included that used tissue or feces as their sample source for 16S rRNA gene sequencing analysis. Studies using either 454 or Illumina sequencing technology were included. Only data sets that had sequences available for analysis were included. Some studies did not have publically available sequences or did not have meta data in which the authors were able to share. After these filtering steps the following studies remained: Ahn [@ahn_human_2013], Baxter [@baxter_microbiota-based_2016], Brim [@brim_microbiome_2013], Burns [@burns_virulence_2015], Chen [@chen_human_2012], Dejea [@dejea_microbiota_2014], Flemer [@flemer_tumour-associated_2017], Geng [@geng_diversified_2013], Hale [@hale_shifts_2017], Kostic [@kostic_genomic_2012], Lu [@lu_mucosal_2016], Sanapareddy [@sanapareddy_increased_2012], Wang [@wang_structural_2012], Weir [@weir_stool_2013], and Zeller [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included becasue the 90 individuals analyzed within the study are contained within the larger Baxter study. The Kostic study was not used since after sequence processing all the case samples did not have more than 100 sequences remaining. This left a total of 14 studies in which complete analysis could be completed.

***Data Set Breakdown:***
In total there were 7 studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), 5 studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and 2 studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals that were analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].      

***Sequence Processing:***
For the majority of studies raw sequences were downloaded from the SRA (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata was obtained from the following website: http://www.ncbi.nlm.nih.gov/Traces/study/ by searching the respective accession number of the study. Of the studies that did not have sequences and meta data on the SRA one study had the data stored on DBGap [@ahn_human_2013] and for four studies the data was obtained directly from the authors [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]. Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009]. Where possible quality filtering utilized the default methods used in mothur for either 454 or Illumina based sequencing. If it was not possible to use these defaults the author stated quality cut-offs were used instead. Chimeras were identifed and removed using the VSEARCH [@rognes_vsearch_2016] program and *de novo* OTU clustering at 97% similarity using the OptiClust algorithm [@westcott_opticlust_2017] was utilized. 

***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R software package (v`r paste(version$major, ".", version$minor, sep = "")`). For the alpha diversity analysis values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package. Testing for alpha diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package to correct for study and variable region effects in feces and study, variable region, and individual effects in tissue. Relative Risk was analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages. Relative risk significance testing utilized the chi-squred test. Beta-diversity differences utilized a Bray-Curtis distance matrix and PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages. Random Forest testing of the obtained AUC versus a random model AUC utilized T-tests. Power analysis and estimations were made using the pwr (v`r paste(packageVersion("pwr"))[1]`) and statmod (v`r paste(packageVersion("statmod"))[1]`) packages. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages. 

***Study Analysis Overview:***
Alpha diversity was first assessed for differences between controls, adenoma, and carcinoma. We analyzed the data using linear mixed-effect models and relative risk. Beta-diversity was then assessed for each inidividual study. Next, four specific CRC-associated genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas*) were assessed for differences in relative risk. We then built Random Forest models based on all genera or the select CRC-associated genera. The models were trained on one study then tested on the remaining studies for every study. The data was split between feces and tissue samples. Within the tissue groups the data was further divided between matched and unmatched tissue samples. Where applicable for each study predictions for adenoma and carcinoma were tested. This same approach was then applied at the OTU level with the exception that instead of testing on the other studies a 10-fold cross validation was utilized and 100 different models were created based on random 80/20 splitting of the data to generate a range of expected AUCs. For OTU based models the CRC associated genera included all OTUs that had a taxonomic classification to *Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*. The power of each study was assessed for an effect size ranging from 1% to 30%. An estimated sample n for these effect sizes was also generated based on 80% power.

***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless mentioned otherwise the accession number for the raw sequences for the studies used in this analysis can be found directly in the respective batch file, on the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscript used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reporducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective studies batch file in the GitHub repo or within the original publication. When sequences were not publicly available contacting the corresponding author for raw sequences needs to be undertaken.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a CIHR fellowship and a University of Michigan PTSP fellowship grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were apart of each of the individual studies uitlized. We would also like to thank each of the study authors for making their data available for use. Finally we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Studies with Stool Samples Included in the Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "16S Region", "Controls", "Adenoma", "Carcinoma")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-V3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "16S Region", "Controls", "Adenoma", "Carcinoma")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: Alpha Diversity Differences between Control, Adenoma, and Carcinoma Across Sampling Site.** A) Alpha diversity metric differences by group in stool samples. B) Alpha diversity metric differences by group in unmatched tissue samples. C) Alpha diversity metric differences by group in matched tissue samples. The dashed line represents a Z-score of 0 or no difference from the median.

**Figure 2: Relative Risk for Adenoma or Carcinoma based on Alpha Diversity Metrics in Stool.** A) Alpha metric relative risk for adenoma. B) Alpha metric relative risk for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 3: Colorectal Cancer Associated Genera Relative Risk for Adenoma and Carcinoma in Stool and Tissue.** A) Adenoma relative risk in stool. B) Carinoma relative risk in stool. C) Adenoma relative risk in tissue. D) Carcinoma relative risk in tissue. For all panels the relative risk was also compared to whether one, two, three, or four of the CRC associated genera were present.  

**Figure 4: OTU Random Forest Model of Stool Across Studies.** A) Adenoma random forest model between the full community and CRC associated genera OTUs only. B) Carcinoma random forest model between the full community and CRC associated genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between.

**Figure 5: OTU Random Forest Model of Tissue Across Studies.** A) Adenoma random forest model between the full community and CRC associated genera OTUs only. B) Carcinoma random forest model between the full community and CRC associated genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between.

**Figure 6: Power and Effect Size Analysis of Studies Included.** A) Power based on effect size for studies with Adenoma individuals. B) Power based on effect size for studies with Carcinoma individuals. C) The estimated sample number needed for each arm of each study to detect aneffect size of 1-30%. The dotted red lines in A) and B) represent the generally used power of 0.8.  


\newpage


**Figure S1: Relative Risk for Adenoma or Carcinoma based on Alpha Diversity Metrics in Tissue.** A) Alpha metric relative risk for adenoma. B) Alpha metric relative risk for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Random Forest Genus Model AUC for each Stool Study.** A) AUC of Adenoma models using all genera or CRC associated genera only. B) AUC of Carcinoma models using all genera or CRC associated genera only. The black line represents the median within each group.  

**Figure S3: Random Forest Genus Model AUC for each Tissue Study.** A) AUC of Adenoma models using all genera or CRC associated genera only divided between matched and unmatched tissue. B) AUC of Carcinoma models using all genera or CRC associated genera only. The black line represents the median within each group divided between matched and unmatched tissue. 

**Figure S4: Random Forest Prediction Success Using Genera for each Stool Study.** A) AUC for prediction in Adenoma using all genera or CRC associated genera only. B) AUC for prediction in Carcinoma using all genera or CRC associated genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. 

**Figure S5: Random Forest Prediction Success of Carcinoma Using Genera for each Tissue Study.** A) AUC for prediction in unmatched tissue for all genera or CRC associated genera only. B) AUC for prediction in matched tissue using all genera or CRC associated genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on.

**Figure S6: Random Forest Prediction Success of Adenoma Using Genera for each Tissue Study.** 



\newpage
