---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")
flemer_data <- read_tsv("../data/process/flemer/flemer.metadata")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in odds ratio alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_OR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_tissue_composite.csv"))

#Load in odds ratio select genera data tables
crc_OR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(bh < 0.05))

crc_unmatched_OR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_OR_unmatched_tissue_composite.csv") %>% mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(bh < 0.05))




```



# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Tumors

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that there is a crucial role for the microbiota in colorectal cancer (CRC) pathogenesis. Important drivers within this context have ranged from individual microbes to the whole community. Our study expands on a recent meta-analysis investigating microbial biomarkers for tumors by testing the hypothesis that the bacterial community has important associations to both adenoma and carcinoma tumors. To test this hypothesis we examined both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and colon tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) across 14 previously published 16S rRNA gene sequencing studies on colorectal tumors and the microbiota.

**Results.** Fecal samples had a significant decrease for both Shannon diversity and evenness as tumor severity increased, after correcting for study effect and variable region sequenced (P-value `r ifelse(min(filter(stool_lme, !(alpha_metric %in% "sobs"))[, "BH"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05). This reduction in evenness translated into small increases in the odds ratio for individuals to have both adenoma (P-value = `r format(min(filter(alpha_RR_adn_stool, !(measure %in% "sobs"))[, "pvalue"]), digits = 2)`) and carcinoma tumors (P-value = `r format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2)`) while the reduction in Shannon diversity only translated into an increased odds ratio for individuals to have carcinomas (P-value = `r format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2)`). Increases in mouth-associated microbes were commonly in the top 5 most increased odds ratios for individuals to have either adenoma or carcinoma tumors, regardless of sample type. Prediction models built to classify either individuals with adenoma or carcinoma were trained on the whole community or selected genera (top 5 highest and lowest odds ratios) from either fecal or tissue samples. Both the full and select models for either adenoma or carcinoma resulted in similar classification success according to Area Under the Curve (AUC). The most important groups within the full community models consistently belonged to genera such as *Ruminococcus*, *Bacteroides*, and *Roseburia* across studies. Although a number of associations between the microbiota and tumor were identified, the majority of studies that we used in this meta-analysis were only individually adequately powered for large effect sizes. 

**Conclusions.** These data provide support for the importance of the bacterial community to both adenoma and carcinoma tumorgenesis. The evidence collected within this study on the role of the microbiota in those with tumors identifies a number of correlations that may not have been detected because of the low power associated with the majority of studies that have been performed to date. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; tumor; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world-wide health problem in which the microbiota has been purported to play an active role in disease pathogenesis [@siegel_cancer_2016; @flynn_metabolic_2016]. Numerous studies have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in tumorgenesis using mouse models of CRC. There have also been numerous case-control studies investigating the microbiota in the formation of both adenoma and carcinoma. A recent meta-analysis investigated whether specific biomarkers could be consistently identified using multiple data sets [@shah_leveraging_2017]. This meta-analysis focused on identifying microbial signatures of tumors (biomarkers) but did so on a small total number of individuals and only investigated stool. This present meta-analysis addresses some of these major shortcomings. 


Although there has been an intense focus on microbiota-based biomarker discovery for tumors, the number of candidate genera seem to be endless. Some studies point towards mouth-associated genera such as *Fusobacterium*, *Peptostreptococcus*, *Parvimonas*, and *Porphyromonas* as key enriched genera [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Yet, even in these studies, mouth-associated genera are far from the only microbes identified to be associated with tumors. These other genera include, but are not limited to, *Providencia*, *Mogibacterium*, *Enterococcus*, *Escherichia/Shigella*, *Klebsiella*, and *Streptococcus* [@burns_virulence_2015; @chen_human_2012; @wang_structural_2012]. In fact, there is good *in vivo* evidence that *Escherichia/Shigella* and *Streptococcus* can be important in the pathogenesis of CRC [@ecoli_Arthur_2014; @arthur_intestinal_2012; @strep_Aymeric_2017]. Other studies have also identified *Akkermansia muciniphila* and *Bacteroides fragilis* as potential markers of CRC with good mechanistic studies for the latter [@weir_stool_2013; @wang_structural_2012; @bfrag_Boleij_2014]. A recent meta-analysis confirmed the correlations of certain mouth-associated genera and *Akkermansia muciniphila* with carcinoma [@shah_leveraging_2017]. However, the sample size (n = 509) is equal to or less than some of the more recent individual studies investigating the microbiota and colorectal tumors, making it hard to know how extrapolatable these findings are. That particular meta-analysis also added more potential microbial associations to both carcinoma (*Pantoea agglomerans* *Ruminococcus*, *Lactobacillus*) and adenoma (*Prevotella*, *Methanosphaera*, *Succinovibrio*, *Haemophilus parainfluenzae*, *Ruminococcus*, *Lactobacillus*) stages of disease that need to be investigated further, since a number of these genera have been found to be enriched in controls and not disease [@baxter_microbiota-based_2016; @zeller_potential_2014; @burns_virulence_2015]. Additionally, genera like *Roseburia* have been found in some studies to be increased in tumors but in others to either be decreased or have no difference [@geng_diversified_2013; @wang_structural_2012; @flemer_tumour-associated_2017; @dejea_microbiota_2014].  

Most of these studies have focused on individuals with carcinomas but associations with the adenoma stage of disease are not any clearer at identifying candidate genera correlated with these earlier tumors. Groups focusing on broad scale community metrics have found that measures such as richness are decreased in those with adenomas versus controls. Other studies have identified *Lactococcus*, *Pseudomonas*, *Acidovorax*, *Cloacibacterium*, *Helicobacter*, *Lactobacillus*, *Bilophila*, *Desulfovibrio*, and *Mogibacterium* to be increased in those with adenoma tumors [@sanapareddy_increased_2012; @lu_mucosal_2016; @hale_shifts_2017]. Additionally, based on these studies mentioned, there seems to be very few common genera that are associated with both adenoma and carcinoma tumors, with *Lactobacillus* being one of the few commonalities.   

Targeting the identification of tumor microbial biomarkers within stool seems logical since it offers an easy and cost-effective way to stratify risk of disease. The current gold standard for diagnosis, a colonoscopy, can be time-consuming and is not without risk of complications. Although stool represents an easy and less invasive way to assess risk, it is not clear how well this sample reflects adenoma- and carcinoma- associated microbial communities. Some studies have tried to assess this in health and disease but are limited by their sample size [@flemer_tumour-associated_2017; @Flynn_preprint_2017]. Sampling the microbiota directly associated with colon tissue may provide clearer answers but is not without their own limitations. After the colonoscopy bowel prep the bacterial community sampled may reflect the better adhered microbiota versus the resident community. Additionally, these samples contain more host DNA, potentially limiting the types of analysis that can be done. It is well known that low biomass samples can be very difficult to work with and results can be study dependent due to the randomness of contamination [@Salter_contamination_2014].       

In comparison to the previous meta-analysis, this study significantly increases the total stool samples investigated, re-examines important genera across adenoma and carcinoma across study, and examines differences and similarities between stool and tissue microbiota in the context of colorectal tumors. Importantly, this analysis and approach could provide valuable insights into the common genera that are both protective and detrimental in individuals with adenoma or carcinoma and whether broad bacterial community measurements can account for these changes that were not provided by earlier meta-analysis studies [@shah_leveraging_2017].


Using both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and colon tissues (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) totaling over `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T) + sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples across 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] [Table 1 & 2], we expand both the breadth and scope of the previous meta-analysis to investigate whether the bacterial community or specific members are more important risk factors for both adenoma and carcinoma stages of disease. To accomplish this we first assessed whether bacterial diversity changes throughout disease (control to adenoma to carcinoma) and if it results in an increased odds ratio (OR) for individuals to have either an adenoma or carcinoma. We then assessed what genera, if any, increase or decrease the OR of an individual to have an adenoma or carcinoma. Next, using Random Forest models, we analyzed whether the full community or only the combined top 5 increased and top 5 decreased OR genera resulted in better model classification, based on the area under the curve (AUC). Finally, we also examined at what effect and sample size the studies used were powered for and the sample size needed to get to the traditionally accepted 80% power. Our results from these analyses suggests that the bacterial community changes as disease severity worsens, that specific members are important for disease classification, and that many of the individual studies are underpowered for assessing small effect sizes. 

\newpage

## Results

***Lower Bacterial Diversity is Associated with Increased OR of Tumors:***
To assess differences in broad scale community metrics as disease severity worsens Operational Taxonomic Unit (OTU) richness, evenness, and Shannon diversity measurements were power transformed and Z-score normalized. These metrics are commonly used to assess the total number of OTUs, the equality of their abundance, and the overall diversity, respectively. Using linear mixed-effect models to control for study and variable region we assessed whether OTU richness, evenness, or Shannon diversity changed in a step-wise manner with disease severity. In stool, there was a significant decrease in both evenness and Shannon diversity as disease severity moved from control to adenoma to carcinoma (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))` and `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`, respectively). We next tested whether the detectable differences in community significantly increased in OR of having an adenoma or carcinoma. For fecal samples, a decrease versus the overall median in evenness resulted in a significantly increased OR for carcinoma  (OR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`),  P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) and adenoma (OR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) while a decrease versus the overall median in Shannon diversity only increased the OR for carcinoma (OR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 1]. Using the Bray-Curtis distance metric and PERMANOVA, it was also possible to identify significant bacterial community changes, in specific studies, for both carcinoma-associated and adenoma-associated microbiota versus control [Table S1].

Using similar transformations for tissue samples, linear mixed-effect models were used on the transformed combined data to control for study, re-sampling of the same individual, and 16S variable region to test whether OTU richness, evenness, or Shannon diversity changed in a step-wise manner as disease severity increased. For colon tissue, there were no significant changes in OTU richness, evenness, or Shannon diversity as disease severity progressed from control to adenoma to carcinoma (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05). We next analyzed the OR, for matched (unaffected tissue and an adenoma or carcinoma from the same individual) and unmatched (control and adenoma or carcinoma tissue not from the same individual) colon tissue samples. For individuals at either an adenoma or carcinoma stage of disease there was no significant change in OR based on lower than median values for OTU richness, evenness, and Shannon diversity [Figure S1 & Table S2]. Similar to stool samples, significant differences in bacterial community, assessed by PERMANOVA, were identified in unmatched tissue samples, for those at either adenoma or carcinoma stage of CRC [Table S1]. For studies with matched samples no differences in bacterial community were observed when assessed with PERMANOVA [Table S1]. These tissue results suggest that the microbiota within an individual are similar to each other regardless of disease status.

***Mouth-Associated Genera are Associated with an Increased OR of Tumor:***
Next, we asked if being higher than the median relative abundance, for any specific genera, resulted in an altered OR for adenoma or carcinoma, in stool and colon tissue, due to our previous observations of small increases in OR using OTU richness and Shannon diversity. To investigate this we analyzed all common genera across each study, in colon tissue or stool, and assessed whether a relative abundance higher than the median results in an increase or decrease in OR. For both tissue and stool samples only ORs associated with an increase or decrease in carcioma tumors were significant after multiple comparison correction [Table S3 & S4]. Out of the `r length(crc_OR_stool$measure)` taxa that had significant ORs in stool samples `r length(filter(crc_OR_stool, measure %in% c("Fusobacterium", "Parvimonas", "Porphyromonas", "Peptostreptococcus"))[, "measure"])` were mouth-associated microbes. These mouth-associated genera significantly increased the ORs of carcinoma for stool samples and included *Fusobacterium*, *Parvimonas*, *Porphyromonas*, and *Peptostreptococcus* [Table S3]. Conversely, mouth-associated genera were not significantly associated with an increased OR for carcinoma in tissue samples [Table S4]. Only unmatched tissue samples had significant OR taxa and these were *Dorea* (OR = `r round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "rr"], digits = 2)` (`r round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "ci_lb"], digits = 2)` - `r round(filter(crc_unmatched_OR_tissue, measure == "Dorea")[, "ci_ub"], digits = 2)`)) and *Weissella* (OR = `r round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "rr"], digits = 2)` (`r round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "ci_lb"], digits = 2)` - `r round(filter(crc_unmatched_OR_tissue, measure == "Weissella")[, "ci_ub"], digits = 2)`)). Overall, there was little direct overlap in increased or decreased OR genera between both tumor type and sample site. 

***Select Community Models can Recapitulate Whole Community Models:***
Since specific genera increased the OR for carcinoma over diversity metrics we assessed whether the bacterial community was better at classifying disease versus only a select group of genera. We selected these genera based on significance after multiple comparison correction. If no taxa were significant after multiple comparison correction then no model for that specific grouping (i.e. adenoma stool) was analyzed. We first tested three model AUCs. These models were created using Random Forest and where either all genera, all OTUs, or significant OR taxa only. Next, the all genera models and any significant OR taxa models were tested across all studies that were not used to train the model. For stool, all models used had similar AUCs [Figure 2]. Although for adenoma and unmatched carcinoma this trend held, there were large differences in matched tissue based on whether all genera or OTUs were utilized [Figure S2]. When analyzing the tests sets that were comprised of genera data from other studies, both the all genera and singificant OR taxa only models had a similar ability to detect individuals with carcinomas, for both stool and tissue samples [Figure 3-S3]. 

In stool, the most common genera in the top 10 most important variables, in the full community models using all genera-based models, were *Ruminococcus*, *Bacteroides*, and *Roseburia* [Figure 4]. Regardless of sample type, mouth-associated genera were present in models for carcinomas [Figure 4B & D]. Yet, none were present in the majority of studies and none were present in the adenoma models [Figure 4A & B]. For the full community OTU-based models, *Ruminococcaceae* was present in the top 10 consistently for both adenoma and carcinoma models while *Roseburia* was only present in many adenoma models and *Bacteroides* was present in the overwhelming majority of the carcinoma models [Figure 4C & 4D].

Unlike the stool-based Random Forest models, the tissue-based models, for the full genera from the first approach, showed no consistent representation of *Ruminococcaceae*, *Ruminococcus*, *Bacteroides*, and *Roseburia* in the top 10 most important model variables across study [Figure S4]. The vast majority of the top 10 model variables for the full community genera- and OTU-based models using colon tissue tended to be study specific. Further, there was very little overlap in the top 10 important variables between adenoma and carcinoma stage models, regardless of whether colon tissue or stool was used  [Figure S4]. This discordance between stool and colon tissue samples also applies to the mouth-associated genera with one noticeable skew being that *Fusobacterium* and *Fusobacteriaceae* occur more often in the top 10 of matched versus unmatched colon tissue Random Forest models [Figure S4B-C & S4E-F]. This suggests that either the colon tissue microbiota is study or person dependent, that kit and/or other types of contamination associated with low biomass samples may be skewing the results, or that multiple microbes could act as the inflammatory stimulus needed to exacerbate mutations.

***CRC Studies are Underpowered for Detecting Small Effect Sizes:***
Next, we assessed how much confidence should be placed in the reported outcomes from each individual study by calculating the ability to detect a difference (power) and sample size needed for small, medium, and large effect size differences between cases and controls. When assessing the power of each study at different effect sizes the majority of studies achieved 80% power to detect a 30% or greater difference between groups [Figure 5A & B]. No study that we analyzed had the standard 80% power to detect an effect size difference equal to or below 10% [Figure 5A & B].  In order to achieve a power of 80%, for small effect sizes, studies used in our meta-analysis would need to recruit over 1000 individuals for both the case and control arms [Figure 5C]



\newpage

## Discussion

Our study identifies clear but small differences in diversity at the community level and larger differences for individual genera, present in patients with tumors versus controls [Figure 1-3]. Although there was a step-wise decrease in diversity as disease progressed from control to adenoma to carcinoma, this did not translate into large effect size increases in OR for either adenoma or carcinoma tumors. Even though mouth-associated genera increased individaul's OR of having a carcinoma for certain sample types, they did not consistently increase the OR of having an adenoma. By using these taxa that had significant ORs after multiple comparison correction we found that we could classify indviduals with either adenoma or carcinoma as well as models that use either all genera or all OTUs. Finally, many studies were individually under powered to be able to reject the null hypothesis and this could one reason only the comparison between control and carcinoma individuals for stool samples had relible detectable differences.


The data presented herein support the importance of specific taxa for carcinoma, but not necessarily adenoma, tumor formation. The results that we have presented show that the significant OR taxa model and both the full genera and OTU models, for indviduals with carcinoma, had similar AUCs [Figure 2 & 3]. This suggests that an interplay between a select number of potentially protective and exacerbating microbes within the GI community could be crucial for carcinoma formation. Importantly, it suggests that there may be key members of the GI community that should be studied further to potentially help reduce the risk of carcinoma tumor formation. Conversely, using the present data, it is clear that new approaches may be needed to identify members of the community associated with adenoma tumors. Regardless of sample type and whether a full genera- or OTU-based model was used, our Random Forest models consistently performed poorly. Yet, the step-wise decrease in diversity suggests that the adenoma-associated community is not normal but has changed subtly. This change in diversity, at this early stage of disease, could be focal to the adenoma itself. How the host interacts with these subtle changes at early stages of the disease could be what leads to a thoroughly dysfunctional community that is supportive of tumorgenesis. 


For the full genera- and OTU-based models within stool, common GI microbes were most consistently present in the top 10 genera or OTUs across studies [Figure 4]. Changes in *Bacteroides*, *Ruminococcaceae*, *Ruminococcus*, and *Roseburia* were consistently found to be in the top 10 most important variables across the different studies for both individuals with adenoma and carcinoma [Figure 4]. These data suggest that whether the non-resident bacterium is *Fusobacteria* or *Peptostreptococcus* may not be as important as how these bacteria interact with the changing resident community. Based on these observations, it is possible to hypothesize that small changes in community structure lead to new niches in which any one of the mouth-associated or general inflammatory genera can gain a foothold, exacerbating the initial changes in community and facilitating the transition from adenoma to carcinoma stage of disease.     

The colon tissue-based studies did not provide a clearer understanding of how the microbiota may be associated with tumors. Generally, the full OTU-based models of unmatched and matched colon tissue samples were concordant with stool samples showing that GI resident microbes were the most prevalent in the top 10 most important variables across study [Figure S4E & F]. Unlike in stool, *Fusobacterium* was the only mouth-associated bacteria consistently present in the top 10 most important variables of the full carcinoma stage models [Figure S4B-C & E-F]. The majority of the colon tissue-based results seem to be study specific with many of the top 10 taxa being present only in a single study. Additionally, the presence of genera associated with contamination, within the top 10 most important variables for the genera and OTU models is worrying. The low bacterial biomass of tissue samples coupled with potential contamination could explain why these results seem to be more sporadic than the stool results.

One important caveat to this study is that even though genera associated with certain species such as *Bacteroides fragilis* and *Streptococcus gallolyticus* subsp. *gallolyticus* were not identified, it does not necessarily mean that these specific species are not important in human CRC [@bfrag_Boleij_2014; @strep_Aymeric_2017]. Since we are limited in our aggregation of the data to the genus level, it is not possible to clearly delineate which species are contributing to overall disease progression. Our observations are not inconsistent with the previous literature on either *Bacteroides fragilis* or *Streptococcus gallolyticus* subsp. *gallolyticus*. As an example, the stool-based full community models consistently identified the genus *Bacteroides*, as well as OTUs that classified as *Bacteroides*, to be important model components across studies. This suggests that even though *Bacteroides* may not increase the OR of individuals having an adenoma or carcinoma and may not vary in relative abundance, like *Fusobacterium*, it is still important in CRC. Additionally, *Streptococcus gallolyticus* subsp. *gallolyticus* is a mouth-associated microbe, and the results from this study suggest that regardless of sample type, mouth-associated genera are commonly associated with an increased OR for individuals to have a carcinoma tumor.    

The associations between the microbiota and individuals with adenoma tumors are inconclusive, in part, because many studies may not be powered effectively to observe small effect sizes. None of the studies analyzed were properly powered to detect a 10% or lower change between cases and controls. The results within our meta-analysis suggest that a small effect size may well be the scope in which differences consistently occur between controls and those with adenomas. Future studies investigating adenoma tumors and the microbiota need to take power into consideration to reproducibly study whether the microbiota contributes to adenoma formation. In contrast to adenoma stage of disease, our observations suggest that most studies analyzed have sufficient power to detect many changes in the carcinoma-associated microbiota because of large effect size differences between cases and controls [Figure 5].

\newpage

## Conclusion

By aggregating together a large collection of studies analyzing both fecal and colon tissue samples, we are able to provide evidence supporting the importance of the bacterial community in colorectal tumors. The data presented here suggests that mouth-associated microbes can gain a foothold within the colon and are commonly associated with the greatest OR of individuals having a carcinoma. Conversely, no conclusive signal with these mouth-associated microbes could be detected for individuals with an adenoma. Our observations also highlight the importance of power and sample number considerations when investigating the microbiota and adenoma tumors due to possible subtle changes in the community. Overall, the associations between the microbiota and individuals with carcinomas were much stronger than with those with adenomas. 

\newpage

## Methods

***Obtaining Data Sets:***
The studies used for this meta-analysis were identified through the review articles written by Keku, *et al.* and Vogtmann, *et al.* [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016] and additional studies not mentioned in the reviews were obtained based on the authors' knowledge of the literature. Studies that used tissue or feces as their sample source for 454 or Illumina 16S rRNA gene sequencing analysis and had data sets with sequences available for analysis were included. Some studies were excluded because they did not have publicly available sequences or did not have metadata in which the authors were able to share. After these filtering steps, the following studies remained: Ahn, *et al.* [@ahn_human_2013], Baxter, *et al.* [@baxter_microbiota-based_2016], Brim, *et al.* [@brim_microbiome_2013], Burns, *et al.* [@burns_virulence_2015], Chen, *et al.*  [@chen_human_2012], Dejea, *et al.* [@dejea_microbiota_2014], Flemer, *et al.* [@flemer_tumour-associated_2017], Geng, *et al.* [@geng_diversified_2013], Hale, *et al.* [@hale_shifts_2017], Kostic, *et al.* [@kostic_genomic_2012], Lu, *et al.* [@lu_mucosal_2016], Sanapareddy, *et al.* [@sanapareddy_increased_2012], Wang, *et al.* [@wang_structural_2012], Weir, *et al.* [@weir_stool_2013], and Zeller, *et al.*  [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included because the 90 individuals analyzed within the study are contained within the larger Baxter study [@baxter_microbiota-based_2016]. After sequence processing, all the case samples for the Kostic study had 100 or less sequences remaining and was excluded, leaving a total of 14 studies that analysis could be completed on.

***Data Set Breakdown:***
In total, there were seven studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), five studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].  

***Sequence Processing:***
For the majority of studies, raw sequences were downloaded from the Sequence Read Archive (SRA) (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata were obtained by searching the respective accession number of the study at the following website: http://www.ncbi.nlm.nih.gov/Traces/study/. Of the studies that did not have sequences and metadata on the SRA, data was obtained from DBGap (n = 1, [@ahn_human_2013]) and directly from the authors (n = 4, [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]). Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009] and quality filtering utilized the default methods for both 454 and Illumina based sequencing. If it was not possible to use the defaults, the stated quality cut-offs, from the study itself, were used instead. Sequences that were made up of an artificial combination of two or more different sequences and commonly known as chimeras were identified and removed using VSEARCH [@rognes_vsearch_2016] before *de novo* OTU clustering at 97% similarity was completed using the OptiClust algorithm [@westcott_opticlust_2017]. 

***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R (v`r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. For OTU richness, evenness, and Shannon diversity analysis, values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package [@rcompanion_citation_2017] and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package [@car_citation_2011]. Testing for OTU richness, evenness, and Shannon diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package [@lme4_citation_2015] to correct for study, repeat sampling of individuals (tissue only), and 16S hyper-variable region used. Odds ratios (OR) were analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages [@epir_citation_2017; @metafor_citation_2010] by assessing how many individuals with and without disease were above and below the overall median value within each specific study. OR significance testing utilized the chi-squared test. Diversity differences measured by the Bray-Curtis index utilized the creation of distance matrix and testing with PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package [@vegan_citation_2017]. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages [@caret_citation_2017; @randomforest_citation_2002]. Power analysis and estimations were made using the pwr (v`r paste(packageVersion("pwr"))[1]`) and statmod (v`r paste(packageVersion("statmod"))[1]`) packages [@pwr_citation_2017; @statmod_citation_2016]. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages [@ggplot2_citation_2009; @gridextra_citation_2017]. 

***Study Analysis Overview:***
OTU richness, evenness, and Shannon diversity were first assessed for differences between controls, adenoma tumors, and carcinoma turmors using both linear mixed-effect models and ORs. For each individual study the Bray-Curtis index was used to assess differences between control-adenoma and control-carcinoma individuals. Next, all common genera were assessed for differences in ORs for individuals having an adenoma or carcinoma and corrected for multiple comparisons using the Benjamini-Hochberg method [@benjamini_controlling_1995]. We then built Random Forest models based on all genera, all OTUs, or significant OR taxa (if any were present after multiple comparison correction). For both the full genera and significant OR taxa, models were trained on one study then tested on the remaining studies using genera-based relative abundances. The OTU-based models were built using OTU level data and a 10-fold CV over 100 different iterations, based on random 80/20 splitting of the data, was used to generate a range of expected AUCs. This process was repeated for every study in the meta-analysis. Comparisons of the initial trained model AUCs for the full genera and significant OR taxa were made to the mean AUC generated from the 100 different 10-fold CV runs of the respective OTU-based model. Finally, the power of each study was assessed for an effect size ranging from 1% to 30% and an estimated sample size, for these effect sizes, was generated based on 80% power. For comparisons in which only control versus adenoma individuals were made, the carcinoma individuals were excluded from each respective study. Similarly, for comparisons in which control versus carcinoma individuals were made the adenoma individuals were excluded from each respective study. For all analysis completed fecal and tissue samples were kept separate. Within the tissue groups the data were further divided between samples from the same individual (matched) and those from different individuals (unmatched).

***Obtaining Genera Relative Abundance and Significant OR Taxa Models:***
For the genera analysis of the ORs, OTUs were added together based on the genus or lowest available taxonomic classification level and the total average counts, for 100 different subsamplings was obtained. The significant OR taxa models for the  Random Forest models utilized all taxa that had significant ORs after multiple comparison correction. This meant only models for the carcinoma stool (`r length(crc_OR_stool$measure)` variables) and carcinoma unmatched (`r length(crc_unmatched_OR_tissue$measure)` variables) samples were possible to be created and analyzed. 


***Matched versus Unmatched Tissue Samples:***
In general, tissue samples with control and tumor samples from different individuals were classified as unmatched while samples that belonged to the same individual were classified as matched. Studies with matched data included Burns, Dejea, Geng, and Lu while those with unmatched data were from Burns, Flemer, Chen, and Sanapareddy. For some studies samples became unmatched when a corresponding matched sample did not make it through sequence processing. All samples, from both matched and unmatched tissue samples, were analyzed together for the linear mixed-effect models with samples from the same individual being corrected for. All other analysis, where it is not specified explicitly, matched and unmatched samples were analyzed separately using the statistical approaches mentioned in the Statistical Analysis section. 

***Assessing Important Random Forest Model Variables:***
Using Mean Decrease in Accuracy (MDA) the top 10 most important variables to the Random Forest model were obtained for the full models of the two different approaches used. For the first approach utilizing genus-based models, the number of times that a specific taxa showed up in the top 10 of the training set across each study was counted. For the second approach, that utilized the OTU-based models, the medians for each OTU across 100 different 80/20 splits of the data was generated and the top 10 OTUs then counted for each study. Common taxa were then identified by using the lowest classification for each of the specific OTUs obtained from these counts and the number of times this classification occurred across the top 10 of each study was recorded. Finally, the two studies that had adenoma tumor tissue (Lu and Flemer) were equally divided between matched and unmatched studies and were grouped together for the counting of the top 10 genera and OTUs for both Random Forest approaches.

***Reproducible Methods:***
The code and analysis can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless otherwise mentioned, the accession number of raw sequences from the studies used in this analysis can be found directly in the respective batch file in the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscripts used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective study batch file in the GitHub repository or within the original publication. For instances when sequences are not publicly available, they may be accessed by contacting the corresponding authors from whence the data came.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a Canadian Institute of Health Research fellowship and a University of Michigan Postdoctoral Translational Scholar Program grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally, we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Total Individuals in each Study Included in the Stool Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-V3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: Odds Ratio for Adenoma or Carcinoma based on Bacterial Community Metrics in Stool.** A) Community-based odds ratio for adenoma. B) Community-based odds ratio for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 2: Stool Random Forest Model Train AUCs.** A) Adenoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa. B) Carcinoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa. The black line represents the median AUC for the respective group. If no values are present in the singificant OR taxa group then there were no significant taxa identified and no model was tested.

**Figure 3: Stool Random Forest Genus-Based Model Test AUCs.** A) Test AUCs of adenoma models using all genera across study. B) Test AUCs of carcinoma models using all genera or significant OR taxa only. The black line represents the AUC at 0.5. The red lines represent the median AUC of all test AUCs for a specific study.  

**Figure 4: Most Common Taxa Across Full Community Stool Study Models.** A) Common taxa in the top 10 for adenoma Random Forest all genera-based models. B) Common taxa in the top 10 for carcinoma Random Forest all genera-based models. C) Common taxa in the top 10 for adenoma Random Forest all OTU-based models. D) Common genera in the top 10 for carcinoma Random Forest all OTU-based models.

**Figure 5: Power and Effect Size Analysis of Studies Included.** A) Power based on effect size for studies with adenoma individuals. B) Power based on effect size for studies with carcinoma individuals. C) The estimated sample number needed for each arm of each study to detect an effect size of 1-30%. The dotted red lines in A) and B) represent a power of 0.8.  


\newpage


**Figure S1: Odds Ratio for Adenoma or Carcinoma based on Bacterial Community Metrics in Tissue.** A) Community-based odds ratio for adenoma. B) Community-based odds ratio for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Tissue Random Forest Model Train AUCs.** A) Adenoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa in unmatched and matched tissue. B) Carcinoma random forest model AUCs between all genera, all OTU, and select model based on significant OR taxa in unmatched and matched tissue. The black line represents the median AUC for the respective group. If no values are present in the singificant OR taxa group then there were no significant taxa identified and no model was tested.

**Figure S3: Tissue Random Forest Genus-Based Model Test AUCs.** A) Test AUCs of adenoma models using all genera across study. B) Test AUCs of carcinoma models using all genera for matched tissue studies. C) Test AUCs of carcinoma models using all genera or significant OR taxa only for unmatched tissue studies The black line represents the AUC at 0.5. The red lines represent the median AUC of all test AUCs for a specific study.

**Figure S4: Most Common Genera Across Full Community Tissue Study Models.** A) Common genera in the top 10 for adenoma Random Forest for all genera-based models. B) Common genera in the top 10 for unmatched carcinoma Random Forest for all genera-based models. B) Common genera in the top 10 for matched carcinoma Random Forest for all genera-based models. D) Common genera in the top 10 for adenoma Random Forest for all OTU-based models. E) Common genera in the top 10 for unmatched carcinoma Random Forest for all OTU-based models. F) Common genera in the top 10 for matched carcinoma Random Forest for all OTU-based models.

\newpage
