---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")
flemer_data <- read_tsv("../data/process/flemer/flemer.metadata")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in relative risk alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_RR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_tissue_composite.csv"))

#Load in relative risk select genera data tables
select_RR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_stool_composite.csv"))
select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_tissue_composite.csv"))

adn_select_RR_stool <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_stool_composite.csv"))
adn_select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_tissue_composite.csv"))



```



# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Cancer

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that there is a crucial role for the microbiota in colorectal cancer (CRC) pathogenesis. Important drivers within this context have ranged from individual microbes to the whole community. Our study expands on a recent meta-analysis investigating microbial biomarkers for CRC by testing the hypothesis that the bacterial community has important associations to both early (adenoma) and late (carcinoma) stage disease. To test this hypothesis we examined both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) across 14 different studies.

**Results.** Fecal samples had a significant decrease from control to adenoma to carcinoma for both Shannon diversity and evenness after correcting for study effect and variable region sequenced (P-value `r ifelse(min(filter(stool_lme, !(alpha_metric %in% "sobs"))[, "BH"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05). This reduction in evenness resulted in small increases in relative risk for adenoma (P-value = `r format(min(filter(alpha_RR_adn_stool, !(measure %in% "sobs"))[, "pvalue"]), digits = 2)`) and carcinoma (P-value = `r format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2)`) while the reduction in Shannon diversity only resulted in an increased relative risk for carcinoma (P-value = `r format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2)`). Previously associated colorectal cancer genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*) followed a similar pattern, with their presence significantly increasing the relative risk for carcinoma (P-value `r ifelse(min(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) but not adenoma (P-value `r ifelse(min(filter(adn_select_RR_stool, !(measure %in% c("all_four", "total_four", "Porphyromonas")))[, "pvalue"]) > 0.05, invisible(">"), invisible("check tables"))` 0.05) with the exception of *Porphyromonas* (P-value = `r format(min(filter(adn_select_RR_stool, measure == "Porphyromonas")[, "pvalue"]), digits = 2)`). Using the whole community versus only CRC-associated genera to build a prediction model resulted in higher classification success, based on Area Under the Curve (AUC), for both adenoma and carcinoma using fecal and tissue samples. The most important OTUs for these models consistently belonged to genera such as *Ruminococcus*, *Bacteroides*, and *Roseburia* across studies. Overall, there were less associations between the microbiota and adenoma and one reason why this may be is that most studies were only adequately powered for large effect sizes. 

**Conclusions.** This data provides support for the importance of the bacterial community to both adenoma and carcinoma genesis. The evidence collected within this study on the role of the microbiota in CRC pathogenesis shows stronger associations between carcinoma then adenoma. One reason for this may be in part due to the low power to detect more subtle changes in the majority of studies that have been performed to date. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world-wide health problem [@siegel_cancer_2016] in which the microbiota has been purported to play an active role in disease pathogenesis [@flynn_metabolic_2016]. Numerous studies have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in tumorgenesis using mouse models of CRC. There have also been numerous case/control studies investigating the microbiota in the formation of both adenoma and carcinoma. A recent meta-analysis investigated whether specific biomarkers could be consistently identified using multiple data sets [@shah_leveraging_2017]. This meta-analysis focused on identifying biomarkers or individual microbes but did not critically investigate how the community changes in CRC. 


Although there has been an intense focus on microbiota based biomarker discovery for CRC, the candidate genera seem to be numerous. Some studies point towards mouth-associated genera such as *Fusobacterium*, *Peptostreptococcus*, *Parvimonas*, and *Porphyromonas* as key enriched genera [@ahn_human_2013; @baxter_microbiota-based_2016; @chen_human_2012; @wang_structural_2012; @kostic_fusobacterium_2013; @burns_virulence_2015; @zeller_potential_2014; @flemer_tumour-associated_2017]. Yet, even in some of these studies mouth-associated genera are far from the only microbes identified to be associated with CRC. These other genera include, but are not necessarily limited to, *Providencia*, *Mogibacterium*, *Enterococcus*, *Escherichia/Shigella*, *Klebsiella*, and *Streptococcus* [@burns_virulence_2015; @chen_human_2012; @wang_structural_2012]. In fact, *Escherichia/Shigella* and *Streptococcus* also have good *in vivo* evidence that individual species within the genus can be important in the pathogenesis of CRC [@ecoli_Arthur_2014; @arthur_intestinal_2012; @strep_Aymeric_2017]. Other studies have also identified *Akkermansia muciniphila* and *Bacteroides fragilis* as potential additional markers of CRC with *Bacteroides fragilis* having good mechanistic studies into how this may occur [@weir_stool_2013; @wang_structural_2012; @bfrag_Boleij_2014]. Still, other studies have identified genera like *Roseburia* as increased in CRC while others have reported it to be decreased or have found no difference in any genera at all [@geng_diversified_2013; @wang_structural_2012; @flemer_tumour-associated_2017; @dejea_microbiota_2014].  

Most of these studies have focused on carcinoma but the adenoma observations are not any clearer at identifying candidate genera. Some groups have focused on more broad scale community metrics and have found that metrics such as richness are decreased in adenoma versus controls while others have identified *Lactococcus*, *Pseudomonas*, *Acidovorax*, *Cloacibacterium*, *Helicobacter*, *Lactobacillus*, *Bilophila*, *Desulfovibrio*, and *Mogibacterium* to be increased in adenoma [@sanapareddy_increased_2012; @lu_mucosal_2016; @hale_shifts_2017]. A recent meta-analysis confirmed the correlations of certain mouth-associated genera and *Akkermansia muciniphila* with carcinoma [@shah_leveraging_2017]. However, it only analyzed a total of 509 stool samples and with some recent studies, that were not included, matching this sample size alone, it is hard to know how extrapolatable these findings truly are. That particular meta-analysis also added more potential microbial associations to both carcinoma (*Pantoea agglomerans* *Ruminococcus*, *Lactobacillus*) and adenoma (*Prevotella*, *Methanosphaera*, *Succinovibrio*, *Haemophilus parainfluenzae*, *Ruminococcus*, *Lactobacillus*) that need to be investigated further, since a number of these genera have been found to be enriched in controls and not lesion (adenoma or carcinoma) [@baxter_microbiota-based_2016; @zeller_potential_2014; @burns_virulence_2015].   

Targeting the identification of biomarkers within stool seems logical since it offers an easy and cost-effective way to stratify risk and the current gold standard for diagnosis, a colonoscopy, can be time-consuming and is not risk free. Although stool represents an easy and less invasive way to assess risk, it is not clear how reflective this sample is to the community on the adenoma or carcinoma. Some studies have tried to assess this in health and disease but are limited by their sample size [@flemer_tumour-associated_2017; @Flynn_preprint_2017]. Sampling the tissue directly may provide clearer answers but is not without limitations. Due to bowel prep the communities left for sampling may not be reflective of the resident microbiota, but rather a collection of what is able to keep adhered to the mucosa. Additionally, these samples contain more host DNA, potentially limiting the types of analysis that can be done. It is well known that low biomass samples can be very difficult to work with and results can end up being study dependent due to the randomness of contamination [@Salter_contamination_2014]. Due to these many differences that could arise between stool and tissue, one question our meta-analysis aims to answer is whether there are consistent patterns that emerge across studies regardless of whether they used stool or tissue samples.      

In comparison to the previous meta-analysis, this study significantly increases the total stool samples investigated, re-examines important genera across adenoma and carcinoma across study, and examines differences and similarities between stool and tissue microbiota in the context of CRC. Importantly, this analysis and approach could provide valuable insights into the common genera that are both protective and detrimental in CRC and whether broad bacterial community measurements can account for these changes that were not previously provided by earlier meta-analysis studies [@shah_leveraging_2017].


Using both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) totaling over `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T) + sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples across 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] [Table 1 & 2], we expand both the breadth and scope of the previous meta-analysis to investigate whether the bacterial community is an important risk factor or if select members can recapitulate the same information for both adenoma and carcinoma. To accomplish this we first assessed whether bacterial diversity changes throughout disease (control to adenoma to carcinoma) and if it results in an increased relative risk (RR) for adenoma or carcinoma. We then assessed what genera, if any, increase or decrease the RR of adenoma or carcinoma. Next, using Random Forest models, we analyzed whether the full community or only the top 5 increased and top 5 decreased RR genera combined resulted in better model classification, based on the area under the curve (AUC). Our results suggest that the community changes as disease severity worsens and that this community is important for disease classification. Finally, we also examined at what effect and sample size the studies that were used were powered for and the sample size needed to get to an 80% power. Our power analysis suggests that each individual study used was underpowered for detecting effect size differences of 10% or below between the case and control groups.


\newpage

## Results

***Lower Bacterial Diversity is Associated with Increased RR of Carcinomas:***
Power transforming and Z-score normalizing OTU richness, evenness, and Shannon diversity for the entire data set allowed for the combination of data from all studies to assess differences in broad scale community metrics as disease severity worsens. Using linear mixed-effect models to control for study and variable region we assessed whether OTU richness, evenness, or Shannon diversity changed in a step-wise manner with disease severity. In stool, there was a significant decrease in both evenness and Shannon diversity from control to adenoma to carcinoma (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))` and `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`, respectively) [Figure 1A]. We next tested whether these detectable differences in community resulted in significant increases in RR of lesion (adenoma or carcinoma). For fecal samples, a decrease versus the overal median in evenness resulted in a significantly increased RR for carcinoma  (RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`),  P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) and adenoma (RR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) while a decrease versus the overal median in Shannon diversity only increased the RR for carcinoma (RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_ub"], digits = 2))`), P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2]. Using the Bray-Curtis distance metric and PERMANOVA, it was also possible to identify significant bacterial community changes, in specific studies, for both carcinoma versus control and adenoma versus control [Table S1 & S2].

Using similar transformations for tissue samples, linear mixed-effect models were used on the transformed combined data to control for study, re-sampling of the same individual, and variable region to test whether OTU richness, evenness, or Shannon diversity changed in a step-wise manner as disease severity increased. For tissue, there were no significant changes in OTU richness, evenness, and Shannon diversity from control to adenoma to carcinoma (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05) [Figure 1B & C]. When analyzing the RR, for matched (unaffected tissue and an adenoma or carcinoma from the same individual) and unmatched (control and adenoma or carcinoma tissue not from the same individual) tissue samples, for both adenoma and carcinoma there was no significant change in RR based on lower than median values for OTU richness, evenness, and Shannon diversity [Table S3-S5]. Similar to stool samples, significant differences in bacterial community, assessed by PERMANOVA, were also identified in unmatched tissue samples  for both those with adenoma and carcinoma [Table S6 & S7]. For studies with matched samples no differences in bacterial community were observed when assessed with PERMANOVA [Table S6 & S7].

***Mouth-Associated Genera are Associated with an Increased RR of Lesion:***
Based on the small increase in RR using $\alpha$-diversity metrics, we assessed if being higher than the median for any specific genera resulted in an altered RR for adenoma or carcinoma in stool and tissue. To investigate this we analyzed all common genera across each study, in tissue or stool, and assessed whether a relative abundance higher than the median results in an increase or decrease in RR. For both stool and tissue none of the genera in the top 5 most significantly increased RR were the same between adenoma and carcinoma [Figure 3]. Mouth-associated genera were in the top 5 genera associated with an increased RR of adenoma (*Pyramidobacter* [Figure 3A] and *Rothia* [Figure 3C]) and carcinoma (*Fusobacterium*, *Parvimonas*, *Porphyromonas*, and *Peptostreptococcus* [Figure 3B] and *Fusobacterium* [Figure 3D]) for both stool and tissue. Conversely, genera commonly associated with a normal gastrointestinal tract were correlated with a decreased RR for both adenoma and carcinoma for both stool and tissue [Figure 3]. Overall, there was little direct overlap of the top 5 increased or decreased RR genera between stool and tissue. 

For adenoma, there was almost no overlap between genera with a RR P-value of less than 0.05 and when they were similar the RR was in opposite directions (e.g. *Lactococcus*) [Table S8 & S9]. Many of the genera that had a P-value under 0.05 for either an increased or dereased RR are also highly prevalent in contamination, specifically, *Novosphingobium*, *Selemonas*, and *Achromobacter* [Figure 3 & Table S8-S9]. For carcinoma, certain mouth-associated genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*) had a high RR for both tissue and stool [Table S10 & S11]. Although the most significant increased RR genera for tissue was *Camplyobacter* while in stool it was *Peptostreptococcus* [Table S10 & S11]. 


***Whole Community Models Add Important Context:***
Since specific genera increased RR for carcinoma over diversity metrics we assessed whether the bacterial community was better at classifying disease versus only a select group of genera based on RR and P-value significance and tested this using two approaches. The first approach used genus level data and tested whether there were any differences in AUC between all genera and selected genera during training on one study or testing on all other studies. The second approach used OTU level data and tested whether there was a generalized decrease in the 10-fold cross validation (CV) model AUC across studies when comparing all OTUs versus only OTUs that taxonomically classified to selected genera. 


For the genera-based models the training set median AUC for model classification was similar for select genera and full genera models, for both tissue and stool studies [Figure S2-S3]. With respect to the test sets, comprised of genera data from other studies, both all and select genera models had a similar ability to detect adenomas or carcinomas, with the select genera models performing better in some instances [Figure S4-S6]. Conversely, the OTU-based models showed a slight decrease in median AUC between the full and select models, with one exception being carcinoma models for matched tissue [Figure 4 & 5].  


In stool the most common genera in the full genera-based models belonged to genera such as *Ruminococcus*, *Bacteroides*, and *Roseburia* [Figure 6A & B]. Although all four CRC-associated genera were present in carcinoma, none were present in the majority of studies and *Fusobacterium* was the only genus present in adenoma [Figure 6A & B]. For the full OTU-based models, *Ruminococcaceae* was present in the top 10 consistently for both adenoma and carcinoma models while *Roseburia* was only present in many adenoma models and *Bacteroides* was present in the overwhelming majority of the carcinoma models [Figure 6C & 6D].


Unlike stool, the tissue full genera-based Random Forest models showed no consistent representation of *Ruminococcaceae*, *Ruminococcus*, *Bacteroides*, and *Roseburia* in the top 10 across study [Figure S7]. The vast majority of the top 10 genera and OTUs occured in a study specific manner for these tissue based Random Forest models. For both the full genera and OTU Random Forest models of adenoma and carcinoma, there appears to be very little overlap in the top 10 most important variables between stool and tissue [Figure 6 & S7]. This discordance between stool and tissue also applies to the mouth-associated genera with one noticeable skew being that *Fusobacterium* and *Fusobacteriaceae* occur more often in the top 10 of Random Forest models using matched versus unmatched tissue samples [Figure S7B-C & S7E-F].

***CRC Studies are Underpowered for Detecting Small Effect Sizes:***
Based on our reported results we assessed whether the studies analyzed were realistically powered to identify small, medium, and large scale differences between cases and controls. When assessing the power of each study at different effect sizes the majority of studies, for both adenoma and carcinoma, acheived 80% power to detect a 30% or greater difference between groups [Figure 7A & B]. No study that was analyzed had the standard 80% power to detect an effect size difference that was equal to or below 10% [Figure 7A & B].  In order to achieve a power of 80%, for small effect sizes, studies used in our meta-analysis would need to recruit over 1000 individuals for each arm [Figure 7C]. 



\newpage

## Discussion

Our study identifies clear differences in diversity, both at the community level and for individual genera, that are present in patients with and without CRC [Figure 1-3]. Although there was a step-wise decrease in diversity from control to adenoma to carcinoma, this did not translate into large effect sizes for the RR of lesion. Even though mouth-associated genera increased the RR of carcinoma, they did not consistently increase the RR of adenoma. These mouth-associated genera are clearly important to carcinoma classification but our observations suggest that accounting for the community in which these microbes exist can increase the ability of models to make predictions. 


The data presented herein supports the importance of select genera for carcinoma but not necessarily adenoma formation. Our observations show that when only using the select genera to create models for carcinoma the AUC are similar to using all genera or OTUs [Figure 4 & 5]. This suggests that an interplay between a select number of potentially protective and exacerbating microbes, within the GI community, are crucial for carcinoma formation. Importantly, it suggests that there may be key members of the community that might be studied further to potentially reduce the risk of carcinoma. Conversely, using the present data, it is clear that new approaches may need to be used to identify members of the community that are associated with adenoma. Regardless of sample type and whether a full or select model was used our Random Forest models consistently performed poorly. Yet, the step-wise decrease in diversity suggests that the adenoma community is not normal but has changed subtly [Figure 1]. This change in diversity, at this early stage of disease, could be focal to the adenoma itself. One potential hypothesis from these observations is that at early stages of the disease, how the host interacts with these subtle changes is what ultimately leads to a thoroughly dysfunctional community that is supportive of CRC genesis. 


Within stool, common GI microbes were most consistently present in the top 10 genera or OTUs across study [Figure 6]. Changes in *Bacteroides*, *Ruminococcaceae*, *Ruminococcus*, and *Roseburia* were consistently found to be disciminative across the different studies for both adenoma and carcinoma [Figure 6]. This data would suggest that whether the non-resident bacterium is *Fusobacteria* or *Peptostreptococcus* is not as important as how these bacteria interact with the changing resident community. Based on these observations, it is possible to hypothesize that small changes in community structure lead to new niches in which any one of the mouth-associated genera can gain a foothold, exacerbating the initial changes in community and facilitating the transition from adenoma to carcinoma.     


The tissue studies did not provide a clearer understanding of how the microbiota may be associated with lesion. Generally, the full OTU-based models of unmatched and matched [Figure S7E & F] tissue samples were concordant with stool showing that resident microbes were the most prevalent in the top 10 across study. Unlike in stool, *Fusobacterium* was the only mouth-associated bacteria consistently present in the top 10 of the full CRC models [Figure S7B-C & E-F]. The majority of the tissue results seem to be study specific with many top 10 taxa being present only in a single study. Additionally, the presence of genera associated with contamination, within the top 10 most important variables for the genera and OTU models, is worrying because it is commonly a marker of contamination. The low bacterial biomass of tissue samples coupled with potential contamination could ultimately explain why the these results seem to be more sporadic than the stool results.

One important caveat to this study is that even though specific genera associated with species such as *Bacteroides fragilis* and *Streptococcus gallolyticus* subsp. *gallolyticus* were not identified, it does not necessarily mean these specific species are not important in human CRC [@bfrag_Boleij_2014; @strep_Aymeric_2017]. Since we are limited in our aggregation of the data, across study, to the genus level, it is not possible to clearly delineate what specific species are contributing to overall disease progression. Our observations are not inconsistent with the previous literature on either *Bacteroides fragilis* or *Streptococcus gallolyticus* subsp. *gallolyticus*. As an example, for stool, the full RF models consistently identified the genus *Bacteroides* as well as OTUs that classified as *Bacteroides* to be important model component across studies. Suggesting that even though *Bacteroides* may not increase the RR of lesion and may not vary in relative abundance, like *Fusobacterium*, it is still important in CRC. Additionaly, *Streptococcus gallolyticus* subsp. *gallolyticus* is a mouth-associated micobe and our study clearly shows that regardless of whether the sample is tissue or stool, mouth-associated genera are commonly associated with an increased RR of both adenoma and carcinoma.    

The associations between the microbiota and adenoma are inconclusive, in part, because many studies may not be powered effectively to observe small effect sizes. None of the studies analyzed were properly powered to detect a 10% or lower change between cases and controls. A small effect size may well be the scope in which differences consistently occur in adenoma based on the results observed within our meta-analysis. Future studies investigating adenoma and the microbiota need to take power into consideration if we are to reproducibly study whether the microbiota contributes to poly formation. In contrast to adenoma, our observations suggest that most studies analyzed have sufficient power to detect many changes in carcinoma because of large effect size differences between cases and controls [Figure 7].

\newpage

## Conclusion

By aggregating together a large collection of studies from both feces and tissue, we are able to provide evidence in support of the importance of the bacterial community in CRC. Further, the data presented here suggests that mouth-associated microbes can gain a foothold within the colon and are are commonly associated with the greatest RR of carcinoma. Conversely, no conclusive signal with these mouth-associated micobes could be detected for adenoma. Our observations also highlight the importance of power and sample number considerations when undertaking investigations into the microbiota and adenoma due to the subtle changes in the community. Overall, the microbiota associations with carcinoma are much stronger than the associations with adenoma.

\newpage

## Methods

***Obtaining Data Sets:***
The studies used for this meta-analysis were identified through the review articles written by Keku, *et al.* and Vogtmann, *et al.* [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016] and additional studies not mentioned in the reviews were obtained based on the authors' knowledge of the literature. Studies that used tissue or feces as their sample source for 454 or Illumina 16S rRNA gene sequencing analysis were included and only data sets that had sequences available for analysis were included. Some studies were excluded because they did not have publicly available sequences or did not have metadata in which the authors were able to share and were excluded. After these filtering steps, the following studies remained: Ahn, *et al.* [@ahn_human_2013], Baxter, *et al.* [@baxter_microbiota-based_2016], Brim, *et al.* [@brim_microbiome_2013], Burns, *et al.* [@burns_virulence_2015], Chen, *et al.*  [@chen_human_2012], Dejea, *et al.* [@dejea_microbiota_2014], Flemer, *et al.* [@flemer_tumour-associated_2017], Geng, *et al.* [@geng_diversified_2013], Hale, *et al.* [@hale_shifts_2017], Kostic, *et al.* [@kostic_genomic_2012], Lu, *et al.* [@lu_mucosal_2016], Sanapareddy, *et al.* [@sanapareddy_increased_2012], Wang, *et al.* [@wang_structural_2012], Weir, *et al.* [@weir_stool_2013], and Zeller, *et al.*  [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included because the 90 individuals analyzed within the study are contained within the larger Baxter study [@baxter_microbiota-based_2016]. After sequence processing all the case samples for the Kostic study had 100 or less sequences remaining and was not used which left a total of 14 studies that analysis could be completed on.

***Data Set Breakdown:***
In total, there were seven studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), five studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals that were analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].  

***Sequence Processing:***
For the majority of studies, raw sequences were downloaded from the Sequence Read Archive (SRA) (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata was obtained by searching the respective accession number of the study at the following website: http://www.ncbi.nlm.nih.gov/Traces/study/. Of the studies that did not have sequences and metadata on the SRA, data was obtained from DBGap (n = 1, [@ahn_human_2013]) and directly from the authors (n = 4, [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]). Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009] and quality filtering utilized the default methods, used in mothur, for either 454 or Illumina based sequencing. If it was not possible to use the defaults, the stated quality cut-offs, from the study itself, were used instead. Chimeras were identified and removed using VSEARCH [@rognes_vsearch_2016] before *de novo* OTU clustering at 97% similarity was completed using the OptiClust algorithm [@westcott_opticlust_2017]. 

***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R (v`r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. For the $\alpha$-diversity analysis, values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package [@rcompanion_citation_2017] and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package [@car_citation_2011]. Testing for $\alpha$-diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package [@lme4_citation_2015] to correct for study, repeat sampling of individuals (tissue only), and 16S hypervariable region used. Relative risk was analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages [@epir_citation_2017; @metafor_citation_2010] by assessing how many individuals with and without disease were above and below the overall median value within each specific study. Relative risk significance testing utilized the chi-squared test. $\beta$-diversity differences utilized a Bray-Curtis distance matrix and PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package [@vegan_citation_2017]. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages [@caret_citation_2017; @randomforest_citation_2002]. Power analysis and estimations were made using the pwr (v`r paste(packageVersion("pwr"))[1]`) and statmod (v`r paste(packageVersion("statmod"))[1]`) packages [@pwr_citation_2017; @statmod_citation_2016]. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages [@ggplot2_citation_2009; @gridextra_citation_2017]. 

***Study Analysis Overview:***
$\alpha$-diversity was first assessed for differences between controls, adenoma, and carcinoma using both linear mixed-effect models and RR. $\beta$-diversity was then assessed for each individual study for differences between control-adenoma and control-carcinoma. Next, all common genera were assessed for differences in RR for adenoma and carcinoma and ranked based on P-value. We then built Random Forest models based on all or a selected model, based on the top 5 increased and top 5 decreased RR based on P-value, and these models were trained on one study then tested on the remaining studies, for every study. A similar approach was then applied at the OTU level with the exception that a 10-fold CV over 100 different models, based on random 80/20 splitting of the data, was used to generate a range of expected AUCs. For these OTU-based models, the selected model included all OTUs that had a taxonomic classification to a variable in the top 5 increased and top 5 decreased RR based on P-value. Finally, the power of each study was assessed for an effect size ranging from 1% to 30% and an estimated sample size, for these effect sizes, was generated based on 80% power. For comparisons in which normal versus adenoma were made the carcinoma samples were excluded from each respective study. Similarly, for comparisons in which normal versus carcinoma were made the adenoma samples were excluded from each respective study. The data was split between feces and tissue samples. Within the tissue groups the data was further divided between samples from the same individual (matched) and those from different individuals (unmatched). Where applicable for each study, predictions for adenoma and carcinoma were then tested for feces, matched tissue, and unmatched tissue.

***Obtaining Genera Relative Abundance and Selected Models:***
For the genera analysis of the RR, OTUs were added together based on the genus or lowest available taxonomic classification level and the total average counts, for 100 different subsamplings, were collected. The OTU based Random Forest Models using selected OTUS utilized a similar approach except that the OTUs were not aggregated together by taxonomic identity but kept as separate OTUs. OTU Random Forest models using the full community included all OTUs while those for the selected model included only those OTUs that had a taxonomic classification to a variable in the top 5 increased of top 5 decreased RR based on P-value.   

***Matched versus Unmatched Tissue Samples:***
In general, tissue samples that had control and lesion samples, that did not belong to the same individual, were classified as unmatched while samples, that belonged to the same individual, were classified as matched. Studies with matched data included Burns, Dejea, Geng, and Lu while those with unmatched data were from Burns, Flemer, Chen, and Sanapareddy. For some studies samples became unmatched due to one of the corresponding matched samples not making it through sequence processing. All samples, from both tissue sample types, were analyzed together for the linear mixed-effect models with samples from the same individual corrected for. For all other analysis, not mentioned herein, matched and unmatched samples were analyzed separately using the statistical approaches mentioned in the Statistical Analysis section. 

***Assessing Important Random Forest Model Variables:***
Using Mean Decrease in Accuracy (MDA) the top 10 most important variables were obtained in two different ways depending on whether the model used genera or OTU data. For the genus based models, the number of times that a genus showed up in the top 10 of the training set across each study was counted while, for the OTU based models, the medians for each OTU across 100 different 80/20 splits of the data was generated and the top 10 OTUs then counted for each study. Common taxa, for the OTU based models, were identified by using the lowest classification within the RDP database for each of the specific OTUs obtained from the previous counts and the number of times this classification occurred in the top 10, in each study, was recorded. The two studies that had adenoma tissue were equally divided between matched and unmatched groups and were grouped together for the counting of the top 10 genera and OTUs.

***Reproducible Methods:***
The code and analysis can be found here at  https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless otherwise mentioned, the accession number for the raw sequences for the studies used in this analysis can be found directly in the respective batch file in the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscripts used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective study batch file in the GitHub repository or within the original publication. For instances when sequences are not publicly available, they may be accessed by contacting the corresponding authors from whence the data came.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a Canadian Institute of Health Research fellowship and a University of Michigan Postdoctoral Translational Scholar Program grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally, we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Total Individuals in each Study Included in the Stool Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-V3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: $\alpha$-Diversity Differences between Control, Adenoma, and Carcinoma Across Sampling Site.** A) $\alpha$-diversity metric differences by group in stool samples. B) $\alpha$-diversity metric differences by group in unmatched tissue samples. C) $\alpha$-diversity metric differences by group in matched tissue samples. The dashed line represents a Z-score of 0 or no difference from the median.

**Figure 2: Relative Risk for Adenoma or Carcinoma based on $\alpha$-Diversity Metrics in Stool.** A) $\alpha$-metric relative risk for adenoma. B) $\alpha$-metric relative risk for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 3: Top 5 Genera that Decrease and Increase Relative Risk for Lesion.** A) Adenoma relative risk in stool. B) Carcinoma relative risk in stool. C) Adenoma relative risk in tissue. D) Carcinoma relative risk in tissue. For all panels the relative risk was also compared to whether one, two, three, or four of the CRC-associated genera were present.  

**Figure 4: Stool OTU Random Forest Model Across Studies.** A) Adenoma random forest model between the full community and select genera OTUs only. B) Carcinoma random forest model between the full community and select genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between. The solid red line represents the median AUC of all the studies for either the full community or select genera OTUS only model.

**Figure 5: Tissue OTU Random Forest Model Across Studies.** A) Adenoma random forest model between the full community and select genera OTUs only. B) Carcinoma random forest model between the full community and select genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between. The solid red line represents the median AUC of all the studies for either the full community or select genera OTUS only model.

**Figure 6: Most Common Genera Across Full Community Stool Study Models.** A) Common genera in the top 10 for adenoma Random Forest genus models. B) Common genera in the top 10 for carcinoma Random Forest genus models. C) Common genera in the top 10 for adenoma Random Forest OTU models. D) Common genera in the top 10 for carcinoma Random Forest OTU models.

**Figure 7: Power and Effect Size Analysis of Studies Included.** A) Power based on effect size for studies with adenoma individuals. B) Power based on effect size for studies with carcinoma individuals. C) The estimated sample number needed for each arm of each study to detect an effect size of 1-30%. The dotted red lines in A) and B) represent a power of 0.8.  


\newpage


**Figure S1: Relative Risk for Adenoma or Carcinoma based on $\alpha$-Diversity Metrics in Tissue.** A) $\alpha$-metric relative risk for adenoma. B) $\alpha$-metric relative risk for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Stool Random Forest Genus Model AUC for each Study.** A) AUC of adenoma models using all genera or select genera only. B) AUC of carcinoma models using all genera or select genera only. The black line represents the median within each group.  

**Figure S3: Tissue Random Forest Genus Model AUC for each Study.** A) AUC of adenoma models using all genera or only select genera divided between matched and unmatched tissue. B) AUC of carcinoma models using all genera or select genera only. The black line represents the median within each group divided between matched and unmatched tissue. 

**Figure S4: Stool Random Forest Prediction Success Using Genera Across Studies.** A) AUC for prediction in adenoma using all genera or select genera only. B) AUC for prediction in carcinoma using all genera or select genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. The red lines represent the median AUC using that specific study as the training set.

**Figure S5: Tissue Random Forest Prediction Success of Carcinoma Using Genera Across Studies.** A) AUC for prediction in unmatched tissue for all genera or select genera only. B) AUC for prediction in matched tissue using all genera or select genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. The red lines represent the median AUC using that specific study as the training set.

**Figure S6: Tissue Random Forest Prediction Success of Adenoma Using Genera Across Studies.** The red lines represent the median AUC using that specific study as the training set.

**Figure S7: Most Common Genera Across Full Community Tissue Study Models.** A) Common genera in the top 10 for adenoma Random Forest genus models. B) Common genera in the top 10 for unmatched carcinoma Random Forest genus models. B) Common genera in the top 10 for matched carcinoma Random Forest genus models. D) Common genera in the top 10 for adenoma Random Forest OTU models. E) Common genera in the top 10 for unmatched carcinoma Random Forest OTU models. F) Common genera in the top 10 for matched carcinoma Random Forest OTU models.

\newpage
