---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read study data files
stool_n <- read_csv("../data/process/tables/stool_study_n_analyzed.csv")
tissue_n <- read_csv("../data/process/tables/tissue_study_n_analyzed.csv")
flemer_data <- read_tsv("../data/process/flemer/flemer.metadata")

#Load in linear mixed-effect alpha diversity data tables 
stool_lme <- as.data.frame(read_csv("../data/process/tables/alpha_mixeffect_results.csv"))
unmatch_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_unmatched_mixeffect_results.csv"))
match_tissue_lme <- as.data.frame(read_csv("../data/process/tables/alpha_matched_mixeffect_results.csv"))

#Load in relative risk alpha diversity data tables
alpha_RR_crc_stool <- as.data.frame(read_csv("../data/process/tables/alpha_RR_composite.csv"))
alpha_RR_adn_stool <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_composite.csv"))

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_unmatched_tissue_composite.csv"))
alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_matched_tissue_composite.csv"))

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_tissue_composite.csv"))

#Load in relative risk select genera data tables
select_RR_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_stool_composite.csv"))
select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_RR_tissue_composite.csv"))

adn_select_RR_stool <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_stool_composite.csv"))
adn_select_RR_tissue <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_RR_tissue_composite.csv"))

select_inc_4_stool <- as.data.frame(read_csv("../data/process/tables/select_genus_inc_4_stool.csv"))
select_inc_4_tissue <- as.data.frame(read_csv("../data/process/tables/select_genus_inc_4_tissue.csv"))

adn_select_inc_4_RR_stool <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_inc_4_stool.csv"))
adn_select_inc_4_RR_tissue <- as.data.frame(read_csv("../data/process/tables/adn_select_genus_inc_4_tissue.csv"))

```



# Investigating the Microbiota and Colorectal Cancer: The Importance of Community

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** An increasing body of literature suggests that there is a crucial role for the microbiota in colorectal cancer (CRC) pathogenesis. Important drivers within this context have ranged from individual microbes to the whole community. Our study expands on a recent meta-analysis investigating microbial biomarkers for CRC by testing the hypothesis that the bacterial community is an important driver of both early (adenoma) and late (carcinoma) stage of disease. To test this hypothesis we examined both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) across 14 different studies.

**Results.** Fecal samples had a significant decrease from control to adenoma to carcinoma for both Shannon diversity and evenness (P-value `r ifelse(min(filter(stool_lme, !(alpha_metric %in% "sobs"))[, "BH"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) after correcting for study effect and variable region sequenced. Only evenness for adenoma (P-value `r ifelse(min(filter(alpha_RR_adn_stool, !(measure %in% "sobs"))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) resulted in a slightly increased relative risk while lower Shannon diversity and evenness in fecal samples resulted in a significant increase in relative risk for carcinoma (P-value `r ifelse(min(filter(alpha_RR_crc_stool, !(measure %in% "sobs"))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05). Previously associated colorectal cancer genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*) followed a similar pattern with a significantly increased relative risk by their presence for carcinoma (P-value `r ifelse(min(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05) but not adenoma (P-value `r ifelse(min(filter(adn_select_RR_stool, !(measure %in% c("all_four", "total_four", "Porphyromonas")))[, "pvalue"]) > 0.05, invisible(">"), invisible("check tables"))` 0.05) with the exception of *Porphyromonas* (P-value `r ifelse(min(filter(adn_select_RR_stool, measure == "Porphyromonas")[, "pvalue"]) < 0.05, invisible("<"), invisible("check tables"))` 0.05). Using the whole community versus only CRC associated genera to build a prediction model resulted in higher classification success based on Area Under the Curve (AUC) for both adenoma and carcinoma using fecal and tissue samples. The most important OTUs for these models consistently belonged to genera such as *Ruminococcus*, *Bacteroides*, and *Roseburia* across studies. For the included studies, most were adequately powered for large effect size differences. This may be more amenable for carcinoma than for adenoma microbiota research due to the smaller community level changes observed in our results.     

**Conclusions.** This data provides support for the importance of the bacteral community to both adenoma and carcinoma genesis. The evidence collected within this study on the role of the microbiota in CRC pathogenesis is much stronger for carcinoma then adenoma. A strong reason for this may be in part due to the low power to detect more subtle changes in the majority of studies that have been performed to date. 


### Keywords

microbiota; colorectal cancer; polyps; adenoma; meta-analysis.

\newpage


## Background

Colorectal cancer (CRC) is a growing world-wide health problem [@siegel_cancer_2016] in which the microbiota has been purported to play an active role in disease pathogenesis [@flynn_metabolic_2016]. Numerous studies have shown the importance of both individual microbes [@goodwin_polyamine_2011; @abed_fap2_2016; @arthur_intestinal_2012; @kostic_fusobacterium_2013; @wu_human_2009] and the overall community [@zackular_manipulation_2016; @zackular_gut_2013; @baxter_structure_2014] in polyp formation using mouse models of CRC. There have also been numerous case/control studies investigating the microbiota in the formation of both adenoma and carcinoma. Recently, a meta-analysis was published investigating whether specific biomarkers could be consistently identified using multiple data sets [@shah_leveraging_2017]. Many of the studies, along with the previous meta-analysis, focus on identifying biomarkers or individual microbes but do not critically investigate the role the community has in CRC. 

Targeting the identification of biomarkers within stool seems logical since the current gold standard for diagnosis, a colonoscopy, can be time-consuming and is not risk free. Stool offers an easy and cost-effective way in which to stratify CRC risk within the population. Although stool represents an easy and less invasive way to assess risk, it is not clear how reflective this sample is to what is actually happening on the adenoma or carcinoma. Some studies have begun to try and assess this in health and disease but are limited by their sample size [@flemer_tumour-associated_2017; @Flynn_preprint_2017]. Since the stool possibly represents an agregate community of the overall gastrointestinal (GI) tract, sampling the adenoma or carinoma directly could more accurately identify important microbes which could be drastically different then what is identified in stool. Sampling the tissue directly may provide clearer answers but it is not without problems. Due to bowel prep the communities left for sampling may not be reflective of the resident microbiota, but rather a collection of what is able to keep adhered to the mucosa. Additionally, these samples would contain much more host DNA then microbial DNA, potentially limiting the types of analysis that can be done. It has also been well published that low biomass samples can be very difficult to work with and results can end up being study dependent due to the randomness of contamination [@Salter_contamination_2014]. Due to these many differences, one question our meta-analysis aims to answer is whether it is possible to find consistent patterns that emerge across studies regardless of whether they used stool or tissue samples.      

This intense focus on identifying biomarkers has identified mostly mouth-associated microbes as potential CRC-associated microbes in the GI tract [@kostic_genomic_2012; @baxter_microbiota-based_2016; @zeller_potential_2014]. The main bacteria of interest arising from this set of microbes has been those within the *Fusobacterium* genus. Yet, the question remains as to whether or not this is indeed the most important genera to be focusing on, since many microbiota-based studies typically have identified a collection of oral microbes rather than single species from a single genera [@baxter_microbiota-based_2016; @zeller_potential_2014]. Based on this discrepency, the second question we want to answer with this meta-analysis is if there is one dominant CRC-associated genera that can be identified across studies.    

The heavy concentration on the identification of biomarkers has mostly dominated the study of CRC and has had an unattended consequence of reducing the focus on changes that occur within the underlying resident community.  This has been borne out by the majority of previous studies, within stool, tissue, and the only meta-analysis investigating this area, that have focused predominately on biomarker identification. Since it has not been investigated in detail the final question that this meta-analysis aims to answer is whether there are consistent detectable community differences as disease severity increases. 

Using both feces (n = `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)`) and tissue (`r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` samples from `r length(unique(flemer_data$Individual)) + sum(filter(tissue_n, study == "lu")[, c("control", "adenoma")])/2 + sum(filter(tissue_n, study == "burns")[, c("control", "cancer")])/2 + sum(filter(tissue_n, study == "chen")[, c("control", "cancer")])/2 + round(sum(filter(tissue_n, study == "dejea")[, c("control", "cancer")])/2) + 1 + sum(filter(tissue_n, study == "geng")[, c("control", "cancer")])/2 +  sum(filter(tissue_n, study == "sana")[, c("control", "cancer")])` individuals) totalling over `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T) + sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` total samples across 14 studies [@hale_shifts_2017; @flemer_tumour-associated_2017; @chen_human_2012; @wang_structural_2012; @lu_mucosal_2016; @brim_microbiome_2013; @weir_stool_2013; @dejea_microbiota_2014; @sanapareddy_increased_2012; @ahn_human_2013; @burns_virulence_2015; @zeller_potential_2014; @baxter_microbiota-based_2016; @geng_diversified_2013] [Table 1 & 2], we expand both the breadth and scope of the previous meta-analysis to investigate whether the bacterial community is an important risk factor for both adenoma and carcinoma. To accomplish this we first assessed whether the diversity changes throughout disease (control to adenoma to carcinoma) and if it results in an increased relative risk (RR) for adenoma or carcinoma. We then assessed how common CRC-associated genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*) affect the relative risk of adenoma or carcinoma. Next, using Random Forest models, we analyzed whether the full community or only the CRC-associated genera resulted in better model classification based on the area under the curve (AUC). We observed that the community changes as disease severity worsens and that this community is important for disease classification. However, since the changes in community were subtle for adenoma we also examined what effect and sample size the studies that were used were adequately powered for. Although we analyzed data sets which sampled large numbers of individuals, our results indicate the individual studies were underpowered for detecting effect size differences of 10% or below between the case and control groups.


\newpage

## Results

***Lower Community Diversity is Associated with Increased RR of Carcinomas:***
Using the combined data set we first assessed whether there were any broad scale community differences that could be detected as disease severity worsened. Using power transformed and Z-score normalized $\alpha$-diversity metrics, both evenness and Shannon diversity in feces, but not tissue, were  lower in those with carcinoma [Figure 1]. Using linear mixed-effect models to control for study and variable region, there was a significant decrease from control to adenoma to carcinoma for both evenness (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannoneven")[, "BH"], digits = 2, nsmall = 2))`) and Shannon diversity (P-value = `r paste(format(filter(stool_lme, alpha_metric == "shannon")[, "BH"], digits = 2, nsmall = 2))`). However, in tissue, this effect was not observed when resampling of the same individual was also controlled for (P-value `r ifelse(min(match_tissue_lme$BH) < 0.05 & min(unmatch_tissue_lme$BH) < 0.05, invisible("check tables"), invisible(">"))` 0.05). We next tested whether these detectable differences in the community resulted in a significant increase in RR. Within fecal samples, a decrease in Shannon diversity and evenness resulted in a significantly increased RR for carcinoma (P-value = `r paste(format(filter(alpha_RR_crc_stool, measure == "shannon")[, "pvalue"], digits = 2, nsmall = 2))` and `r paste(format(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`, respectively) [Figure 2]. Although these values were significant, the effect size was relatively small for both metrics (Shannon RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannon")[, "ci_ub"], digits = 2))`) and evenness RR = `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_crc_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`)) [Figure 2]. Only a decrease in evenness had an increased RR for adenoma (P-value = `r paste(format(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "pvalue"], digits = 2, nsmall = 2))`) [Figure 2A & S1] but this effect size was even smaller than what was observed for carcinoma (RR = `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "rr"], digits = 2))` (`r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_lb"], digits = 2))` - `r paste(round(filter(alpha_RR_adn_stool, measure == "shannoneven")[, "ci_ub"], digits = 2))`)). Interestingly, for both adenoma and carcinoma there was no increase in RR within tissue samples for any alpha diversity metric investigated [Table S1-S3].

Using the Bray-Curtis distance metric, there was a significant difference across studies in the bacterial community of fecal samples between carcinoma and controls, but not adenoma and controls [Table S4 & S5]. For studies with unmatched tissue samples a similar trend was observed [Table S3 & S4] while studies with tissue samples from the same individual (matched) had no differences [Table S6 & S7].

***Carcinoma-Associated Genera Minimally Impacts RR of Adenoma:***
Based on the small increase in RR using $\alpha$-diversity metrics, we wanted to know if the presence of specific genera resulted in a higher RR for both stool and tissue. To investigate this we analyzed the classically associated CRC genera, *Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas* for and increase in RR. The majority of CRC-associated genera for both feces and tissue had a significantly increased RR for carcinoma but not for adenoma [Figure 3]. In fecal samples the RR due to CRC associated genera was greater than either the RR assoicated with evenness or Shannon diversity [Figure 2 & 3]. Additionally, the RR of carcinoma continuously increased as individuals tested positive for more CRC associated genera [Figure 3B & 3D]. The RR effect size was greater for stool (RR range = `r paste(round(min(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))` - `r paste(round(max(filter(select_RR_stool, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))`) than for tissue (RR range = `r paste(round(min(filter(select_RR_tissue, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))` - `r paste(round(max(filter(select_RR_tissue, !(measure %in% c("all_four", "total_four")))[, "rr"]), digits = 2))`). This decrease may be explained by the fact that the tissue analysis included matched samples from the same individual. 

There were two significant measures for increased RR of adenoma when investigating CRC-associated genera in stool: 1) Having a higher then median value of *Porphyromonas* (P-value = `r format(filter(adn_select_RR_stool, measure == "Porphyromonas")[, "pvalue"], digits = 2, nsmall = 2)`) and 2) whether samples were positive for three CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_stool, measure == "three_counts")[, "pvalue"], digits = 2, nsmall = 2)`) [Figure 3A]. With tissue, there were three significant measures for an increased RR of adenoma: 1) being positive for one CRC-associated genera (P-value = `r format(filter(adn_select_inc_4_RR_tissue, measure == "one_counts")[, "pvalue"], digits = 2, nsmall = 2)`), 2) being positive for two CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_tissue, measure == "two_counts")[, "pvalue"], digits = 2, nsmall = 2)`), and 3) being positive for four CRC associated genera (P-value = `r format(filter(adn_select_inc_4_RR_tissue, measure == "four_counts")[, "pvalue"], digits = 2, nsmall = 2)`) [Figure 3C].  


***Using the Whole Community Instead of Only CRC-Associated Genera Increases Model AUC:***
We then tested whether the overall bacterial community was at all important to classifying disease or if the CRC-associated genera were sufficient alone. To test this we used two approaches. The first used genus level data and tested whether there were any differences in AUC when training on one study and testing on all the others when using either all genera present or only the CRC-associated genera. The second approach used OTU level data and tested whether there was a generalized decrease in the 10-fold cross validation (CV) model across studies using either all OTUs or only OTUs that taxonomically classified to CRC-associated genera. 

The genus based models showed an AUC decrease in model classification on the training set for both stool and tissue studies [Figure S2-S3]. With respect to the test sets, comprised of genera data from other studies, both the all genera model and CRC-associated models had a similar ability to detect adenomas or carcinomas [Figure S4-S6]. The AUC for classification of adenomas was lower than carcionmas for both tissue and stool [Figure S4-S6].    

The OTU based models for both fecal and tissue (matched and unmatched) samples, showed an AUC decrease when only OTUs from the CRC-associated genera are used versus full community of OTUs [Figure 4 & 5]. This decrease is observed in both adenoma and carcinoma groups [Figure 4 & 5] with the largest difference in median AUC for stool being in carcinoma classification [Figure 4B] and for tissue being in adenoma [Figure 5A].  

In stool the most common genera in the genus based models belonged mostly to resident genera [Figure 6A & B]. This included genera such as *Ruminococcus*, *Bacteroides*, and *Roseburia*. With respect to the CRC-associated genera *Fusobacterium* was the only genus present in adenoma while all four were present in carcinoma [Figure 6A & B]. However, none of these CRC-associated genera were present in the majority of studies. For the adenoma OTU models, OTUs that classified as  *Ruminococcaceae* or *Roseburia* were present in the top 10 OTUs for the vast majority of studies [Figure 6C]. Although for the carcinoma OTU models *Ruminococcaceae* was in the top 10 for many studies, *Bacteroides* was present in the overwhelming majority of the carcinoma OTU stool models [Figure 6D].

Conversely to the stool models, both genera and OTU based models in tissue had the vast majority of their top 10 occur in a study specific manner [Figure S7]. *Fusobacterium* and *Fusobacteriaceae* show up more often in the top 10 for matched tissue samples but are not present in the top 10 or much lower for unmatched tissue [Figure S7B-C & S7E-F]. There appears to be very little overlap in the top 10 most important variables between stool and tissue for both adenoma and carcinoma [Figure 6 & S7].

***A Majority of Studies are Underpowered for Detecting Small Effect Size Differences:***
Based on the previous observations we then assessed whether the studies that we included are realistically powered to identify small, medium, and large scale differences between case and control. When assessing the power of each study at different effect sizes the majority of studies for both adenoma and carcinoma have an 80% power to detect a 30% difference [Figure 7A & B]. No single study that was analyzed had the standard 80% power to detect an effect size difference that was equal to or below 10% [Figure 7A & B].  In order to achieve adequate power for small effect sizes, studies would need to recruit over 1000 individuals for each arm [Figure 7C]. 



\newpage

## Discussion

Our study identifies clear differences in diversity both at the community level and within individual genera that are present in individuals with CRC versus those without the disease [Figure 1-3]. Although there was a step-wise decrease in diversity from control to adenoma to carcinoma, this did not translate into large effect sizes for the relative risk of either of these two conditions. Even though CRC-associated genera increase the RR of carcinoma, they do not consistently increase the relative risk of adenoma. This information suggests that these specific genera are important in carcinoma genesis but may not be the primary members of the microbial community contributing to the formation of an adenoma. Additionally, our data show that by using the whole community, our models perform better than when only the CRC-associated genera are included. CRC-associated genera are clearly important to carcinoma pathogenesis but accounting for the community in which these microbes exist can drastically increase the ability of models to make predictions. These observations suggest that small localized changes within the community on tissue may be occuring that are important in early disease progression of CRC and that this process may not directly involve CRC-associated genera.    


The data presented herein supports the driver-passenger model of the microbial role in CRC, as summarized by Flynn [@flynn_metabolic_2016], when applied to carcinoma but not necessarily adenoma. The central idea of the model is that a single bacterium initiates an environment in which other non-resident microbes may then be able to colonize, creating a vicious cycle that is conducive for CRC. Both the drastically increased RR of CRC-associated genera versus $\alpha$-diversity metrics for carcinoma and increasing RR with more CRC-associated generea positivity are highly supportive of this model. However, the initial establishment of the driver within the system appears to be dependent on the current community. This is supported by our finding that when adding the community context to our models in addition to the CRC-associated genera, the model AUC increases [Figure 4 & 5]. Conversely, using the present data, it is less likely that adenoma development fits this model. The changes that occur at this timepoint are small and possibly focal to the adenoma itself. The stepwise decrease in diversity suggests that the adenoma community is not normal but has changed subtly [Figure 1]. Although there appears to be localized changes that do depend on the driver-passenger model, as supported by an increased RR for one, two, and four positive CRC-associated genera in tissue [Figure 3C], there may be other processes at play that ultimately exacerbate the condition from a subtle localized change, to a change in the global community. The poor performance of the Random Forest models for classifying adenoma based only on the microbiota would suggest that this is the case. One potential hypothesis from these observations is that at early stages of the diease, how the host interacts with these subtle changes is what ultimately leads to a thoroughly dysfunctional community that is supportive of CRC genesis. 


Within stool, common resident microbes were most consistently present in the top 10 genera or OTUs across study [Figure 6]. Changes in *Bacteroides*, *Ruminococcus*, and *Roseburia* were consistently found to be disciminative across the different studies for both adenoma and carcinoma models [Figure 6]. This data would suggest that whether the non-resident bacterium is *Fusobacteria* or *Peptostreptococcus* is not as important as how these bacteria interact with the changing resident community. These observations would also suggest that initial changes in the resident community, specifically to *Ruminococcaceae*, *Ruminococcus*, and *Roseburia*, carry on from adenoma to carcinoma. Based on these observations, it is possible to hypothesize that the early changes in the community may give rise to initial polyp formation via interactions with the host and not necessarily via interactions with CRC-assciated genera. These changes then create new niches in which any one of the CRC-associated genera could gain a foothold, exacerbating the initial changes in community and facilitating the transition from adenoma to carcinoma via a driver-passenger type mechanism.    


The tissue studies did not provide a clearer understanding of how the microbiota may drive the progression of disease severity. For the OTU models both the unmatched and matched [Figure S7E & F] tissue samples had some concordance with the stool data, with resident bacteria being the most prevalent in the top 10 important variables across studies. Unlike in stool, *Fusobacterium* was the only CRC-associated bacteria consistently present in the top 10 of the CRC models [Figure S7B-C & E-F]. The majority of the results seem to be study specific with many top 10 taxa being present only in a single study. One other potentially worrying sign is the presence of *Propionibacterium* within the top 10 for the genera and OTU models and could be a marker of contamination. The low biomass of these samples coupled with potential contamination might be a possible reason why the tissue results seem to be more sporadic then the stool results.


Although there are still questions that need to be answered regarding the microbiota and carcinoma, a clearer framework for their relationship is beginning to develop as to how this occurs. From our observations many changes in carcinoma could easily result in effect sizes that are 30% or more between the case and control and most studies analyzed have sufficient power to detect these types of changes [Figure 7]. Conversely, the role of the microbiota in adenoma is less clear and part of the reason this may be is because many studies are not powered effectively to observe the small changes reported here. None of the studies analyzed were properly powered to detect a 10% or lower change between case and controls. This small effect size range may well be the scope in which differences consistently occur in adenoma due to the subtle changes in community that occur between control and adenoma. Future studies investigating adenoma and the microbiota need to take these factors into consideration if we are to work out how the microbiota contributes to adenoma formation.     

\newpage

## Conclusion

By aggregating together a large collection of studies from both feces and tissue, we are able to provide evidence in support of the importance of the bacterial community in both adenoma and carcinoma. Overall, our results support a framework by which early localized community changes give rise to polyp formation. With the host as a potential catalyst, new niches arise by which non-resident CRC-associated microbes can then gain a foothold and create an environment that allows more of these microbes to colonize. This exacerbates the existing community changes and creates a vicious cycle conducive of carcinoma formation. Our observations also highlight the importance of power and sample number considerations when undertaking investigations into the microbiota and adenoma due to the subtle changes in the community. Although there are power limitations associated with adenoma, this report highlights the strong influence the microbiota has on CRC development.

\newpage

## Methods

***Obtaining Data Sets:***
Studies used for this meta-analysis were identified through the review articles written by Keku, *et al.* and Vogtmann, *et al.* [@keku_gastrointestinal_2015; @vogtmann_epidemiologic_2016]. Additional studies not mentioned in the reviews were obtained based on the authors' knowledge of the literature. Studies that used tissue or feces as their sample source for 16S rRNA gene sequencing analysis were included. Studies using either 454 or Illumina sequencing technology were included. Only data sets that had sequences available for analysis were included. Some studies did not have publically available sequences or did not have metadata in which the authors were able to share. After these filtering steps, the following studies remained: Ahn, *et al.* [@ahn_human_2013], Baxter, *et al.* [@baxter_microbiota-based_2016], Brim, *et al.* [@brim_microbiome_2013], Burns, *et al.* [@burns_virulence_2015], Chen, *et al.*  [@chen_human_2012], Dejea, *et al.* [@dejea_microbiota_2014], Flemer, *et al.* [@flemer_tumour-associated_2017], Geng, *et al.* [@geng_diversified_2013], Hale, *et al.* [@hale_shifts_2017], Kostic, *et al.* [@kostic_genomic_2012], Lu, *et al.* [@lu_mucosal_2016], Sanapareddy, *et al.* [@sanapareddy_increased_2012], Wang, *et al.* [@wang_structural_2012], Weir, *et al.* [@weir_stool_2013], and Zeller, *et al.*  [@zeller_potential_2014]. The Zackular [@zackular_human_2014] study was not included becasue the 90 individuals analyzed within the study are contained within the larger Baxter study [@baxter_microbiota-based_2016]. Additionally, after sequence processing all the case samples for the Kostic study only had 100 sequences remaining and was not used. This left a total of 14 studies for which analysis could be completed.

***Data Set Breakdown:***
In total, there were seven studies with only fecal samples (Ahn, Baxter, Brim, Hale, Wang, Weir, and Zeller), five studies with only tissue samples (Burns, Dejea, Geng, Lu, Sanapareddy), and two studies with both fecal and tissue samples (Chen and Flemer). The total number of individuals that were analyzed after sequence processing for feces was `r sum(stool_n$control) + sum(stool_n$adenoma, na.rm = T) + sum(stool_n$cancer, na.rm = T)` [Table 1]. The total number of matched and unmatched tissue samples that were analyzed after sequence processing was `r sum(tissue_n$control) + sum(tissue_n$adenoma, na.rm = T) + sum(tissue_n$cancer, na.rm = T)` [Table 2].  

***Sequence Processing:***
For the majority of studies, raw sequences were downloaded from the Sequence Read Archive (SRA) (ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/) and metadata was obtained from the by searching the respective accession number of the study following website: http://www.ncbi.nlm.nih.gov/Traces/study/. Of the studies that did not have sequences and metadata on the SRA, data was obtained from DBGap for one study [@ahn_human_2013] and for four studies was obtained directly from the authors [@flemer_tumour-associated_2017; @hale_shifts_2017; @sanapareddy_increased_2012; @weir_stool_2013]. Each study was processed using the mothur (v1.39.3) software program [@schloss_introducing_2009]. Where possible, quality filtering utilized the default methods used in mothur for either 454 or Illumina based sequencing. If it was not possible to use these defaults, the stated quality cut-offs were used instead. Chimeras were identifed and removed using VSEARCH [@rognes_vsearch_2016] before *de novo* OTU clustering at 97% similarity using the OptiClust algorithm [@westcott_opticlust_2017] was utilized. 

***Statistical Analysis:***
All statistical analysis after sequence processing utilized the R (v`r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. For the $\alpha$-diversity analysis, values were power transformed using the rcompanion (v`r paste(packageVersion("rcompanion")[1])`) package [@rcompanion_citation_2017] and then Z-score normalized using the car (v`r paste(packageVersion("car")[1])`) package [@car_citation_2011]. Testing for $\alpha$-diversity differences utilized linear mixed-effect models created using the lme4 (v`r paste(packageVersion("lme4"))[1]`) package [@lme4_citation_2015] to correct for study and variable region effects in feces and study, variable region, and individual effects in tissue. Relative risk was analyzed using both the epiR (v`r paste(packageVersion("epiR"))[1]`) and metafor (v`r paste(packageVersion("metafor"))[1]`) packages [@epir_citation_2017; @metafor_citation_2010] by assessing how many with and without disease were above and below the overall median value within the specific study. Relative risk significance testing utilized the chi-squred test. $\beta$-diversity differences utilized a Bray-Curtis distance matrix and PERMANOVA executed with the vegan (v`r paste(packageVersion("vegan"))[1]`) package [@vegan_citation_2017]. Random Forest models were built using both the caret (v`r paste(packageVersion("caret"))[1]`) and randomForest (v`r paste(packageVersion("randomForest"))[1]`) packages [@caret_citation_2017; @randomforest_citation_2002]. Differences between the obtained AUC versus a random model AUC was assessed using T-tests. Power analysis and estimations were made using the pwr (v`r paste(packageVersion("pwr"))[1]`) and statmod (v`r paste(packageVersion("statmod"))[1]`) packages [@pwr_citation_2017; @statmod_citation_2016]. All figures were created using both ggplot2 (v`r paste(packageVersion("ggplot2"))[1]`) and gridExtra (v`r paste(packageVersion("gridExtra"))[1]`) packages [@ggplot2_citation_2009; @gridextra_citation_2017]. 

***Study Analysis Overview:***
$\alpha$-diversity was first assessed for differences between controls, adenoma, and carcinoma. We analyzed the data using linear mixed-effect models and relative risk. $\beta$-diversity was then assessed for each inidividual study. Next, four specific CRC-associated genera (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas*) were assessed for differences in relative risk. We then built Random Forest models based on all genera or the select CRC-associated genera. The models were trained on one study then tested on the remaining studies for every study. The data was split between feces and tissue samples. Within the tissue groups the data was further divided between samples from the same individual (matched) and those from different individuals (unmatched) tissue samples. Where applicable for each study, predictions for adenoma and carcinoma were tested. This same approach was then applied at the OTU level with the exception that instead of testing on the other studies, a 10-fold cross validation was utilized and 100 different models were created based on random 80/20 splitting of the data to generate a range of expected AUCs. For OTU based models, the CRC associated genera included all OTUs that had a taxonomic classification to *Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, or *Porphyromonas*. The power of each study was assessed for an effect size ranging from 1% to 30%. An estimated sample n for these effect sizes was also generated based on 80% power. For comparisons in which normal versus adenoma were made the carcinoma samples were excluded from each respective study. Similarily, for comparisons in which normal versus carcinoma were made the adenoma samples were excluded from each respective study.

***Obtaining CRC-Associated Genera:***
For the CRC-associated genera analysis of the RR, the total average counts were collected for each respective OTU that had a genus level taxonomic classification to *Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas* for 100 different subsamplings. The OTU based Random Forest Models that used CRC-associated genera used a similar approach except that the OTUs were not aggregated together by genus by kept as separate OTUs. So, OTU Random Forest models using the full community included all OTUs while those using CRC-associated genera included only those OTUs that had a genus level taxonomic classification to *Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas*.   

***Matched versus Unmatched Tissue Samples:***
In general, tissue samples that had control and lesion samples that did not belong to the same indivdiual were classified as unmatched while samples that belonged to the same individual were classified as matched. Studies with matched data included Burns, Dejea, Geng, and Lu. Studies that had unmatched data were Burns, Flemer, Chen, and Sanapareddy. For some studies samples became unmatched due to one of the corresponding matched samples not making it through sequence processing. For the linear mixed-effect models samples from the same individual were taken into account. For all other analysis matched and unmatched samples were analyzed separately using the statistical approaches mentioned in the Statistical Analysis section. 

***Assessing Important Random Forest Model Variables:***
The genus based models collected the top 10 most important variables from each training set and assessed how many times that genera showed up in the top 10 across each study. The OTU based models recorded the medians for each OTU across 100 different 80/20 splits of the data for each study. The lowest classification for each OTU was obtained using the RDP database and the number of times the specific classification occured in the top 10 across studies was recorded. For the adenoma tissue genus and OTU models there was only one matched and unmatched study and these results were grouped together for the counting of the top 10.

***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Unless mentioned otherwise, the accession number for the raw sequences for the studies used in this analysis can be found directly in the respective batch file in the GitHub repository or in the original manuscript.


\newpage


## Declarations

### Ethics approval and consent to participate

Ethics approval and informed consent for each of the studies used is mentioned in the respective manuscripts used in this meta-analysis.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed for each study can be found at https://github.com/SchlossLab/Sze_CRCMetaAnalysis_Microbiome_2017. Raw sequences can be downloaded from the SRA in most cases and can be found in the respective study batch file in the GitHub repository or within the original publication. For instances when sequences are not publicly available, they may be accessed by contacting the corresponding authors from whence the data came.  

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

MAS is supported by a Candian Institute of Health Research fellowship and a University of Michigan Postdoctoral Translational Scholar Program grant.

### Authors' contributions

All authors helped to design and conceptualize the study. MAS identified and analyzed the data. MAS and PDS interpreted the data. MAS wrote the first draft of the manuscript and both he and PDS reviewed and revised updated versions. All authors approved the final manuscript.

### Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: Total Individuals in each Study Included in the Stool Analysis**

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}

stool_n <- stool_n %>% arrange(study)

studies <- c("Ahn", "Baxter", "Brim", "Flemer", "Hale", "Wang", "Weir", "Zeller")


data_obtained <- c("DBGap", "SRA", "SRA", "Author", "Author", "SRA", "Author", "SRA")

variable_region <- c("V3-4", "V4", "V1-3", "V3-4", "V3-5", "V3", "V4", "V4")

con_n <- gsub("NA", 0, str_replace_na(stool_n$control))
adn_n <- gsub("NA", 0, str_replace_na(stool_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(stool_n$cancer))

table1 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table1) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table1, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage

**Table 2: Studies with Tissue Samples Included in the Analysis**

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}

tissue_n <- tissue_n %>% arrange(study)

studies <- c("Burns", "Chen", "Dejea", "Flemer", "Geng", "Lu", "Sanapareddy")

data_obtained <- c("SRA", "SRA", "SRA", "Author", "SRA", "SRA", "Author")

variable_region <- c("V5-6", "V1-V3", "V3-5", "V3-4", "V1-2", "V3-4", "V1-2")

con_n <- tissue_n$control
adn_n <- gsub("NA", 0, str_replace_na(tissue_n$adenoma))
crc_n <- gsub("NA", 0, str_replace_na(tissue_n$cancer))

table2 <- cbind(studies, data_obtained, variable_region, con_n, adn_n, crc_n)

colnames(table2) <- c("Study", "Data Stored", "16S Region", "Control (n)", "Adenoma (n)", "Carcinoma (n)")


kable(table2, format = "markdown", align = 'c', padding = 2)
```

\normalsize
\newpage



**Figure 1: $\alpha$-Diversity Differences between Control, Adenoma, and Carcinoma Across Sampling Site.** A) $\alpha$-diversity metric differences by group in stool samples. B) $\alpha$-diversity metric differences by group in unmatched tissue samples. C) $\alpha$-diversity metric differences by group in matched tissue samples. The dashed line represents a Z-score of 0 or no difference from the median.

**Figure 2: Relative Risk for Adenoma or Carcinoma based on $\alpha$-Diversity Metrics in Stool.** A) $\alpha$-metric relative risk for adenoma. B) $\alpha$-metric relative risk for carcinoma. Colors represent the different variable regions used within the respective study. 

**Figure 3: CRC-Associated Genera Relative Risk for Adenoma and Carcinoma in Stool and Tissue.** A) Adenoma relative risk in stool. B) Carcinoma relative risk in stool. C) Adenoma relative risk in tissue. D) Carcinoma relative risk in tissue. For all panels the relative risk was also compared to whether one, two, three, or four of the CRC-associated genera were present.  

**Figure 4: OTU Random Forest Model of Stool Across Studies.** A) Adenoma random forest model between the full community and CRC-associated genera OTUs only. B) Carcinoma random forest model between the full community and CRC-associated genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between. The solid red line represents the median AUC of all the studies for either the full community or CRC-associated genera OTUS only model.

**Figure 5: OTU Random Forest Model of Tissue Across Studies.** A) Adenoma random forest model between the full community and CRC-associated genera OTUs only. B) Carcinoma random forest model between the full community and CRC-associated genera OTUs only. The dotted line represents an AUC of 0.5 and the lines represent the range in which the AUC for the 100 different 80/20 runs fell between. The solid red line represents the median AUC of all the studies for either the full community or CRC-associated genera OTUS only model.

**Figure 6: Most Common Genera Across Full Community Stool Study Models.** A) Common genera in the top 10 for adenoma Random Forest genus models. B) Common genera in the top 10 for carcinoma Random Forest genus models. C) Common genera in the top 10 for adenoma Random Forest OTU models. D) Common genera in the top 10 for carcinoma Random Forest OTU models.

**Figure 7: Power and Effect Size Analysis of Studies Included.** A) Power based on effect size for studies with adenoma individuals. B) Power based on effect size for studies with carcinoma individuals. C) The estimated sample number needed for each arm of each study to detect an effect size of 1-30%. The dotted red lines in A) and B) represent a power of 0.8.  


\newpage


**Figure S1: Relative Risk for Adenoma or Carcinoma based on $\alpha$-Diversity Metrics in Tissue.** A) $\alpha$-metric relative risk for adenoma. B) $\alpha$-metric relative risk for carcinoma. Colors represent the different variable regions used within the respective study.

**Figure S2: Random Forest Genus Model AUC for each Stool Study.** A) AUC of adenoma models using all genera or CRC-associated genera only. B) AUC of carcinoma models using all genera or CRC-associated genera only. The black line represents the median within each group.  

**Figure S3: Random Forest Genus Model AUC for each Tissue Study.** A) AUC of adenoma models using all genera or only CRC-associated genera divided between matched and unmatched tissue. B) AUC of carcinoma models using all genera or CRC-associated genera only. The black line represents the median within each group divided between matched and unmatched tissue. 

**Figure S4: Random Forest Prediction Success Using Genera for each Stool Study.** A) AUC for prediction in adenoma using all genera or CRC associated genera only. B) AUC for prediction in carcinoma using all genera or CRC-associated genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. The red lines represent the median AUC using that specific study as the training set.

**Figure S5: Random Forest Prediction Success of Carcinoma Using Genera for each Tissue Study.** A) AUC for prediction in unmatched tissue for all genera or CRC-associated genera only. B) AUC for prediction in matched tissue using all genera or CRC-associated genera only. The dotted line represents an AUC of 0.5. The x-axis is the data set in which the model was initially trained on. The red lines represent the median AUC using that specific study as the training set.

**Figure S6: Random Forest Prediction Success of Adenoma Using Genera for each Tissue Study.** The red lines represent the median AUC using that specific study as the training set.

**Figure S7: Most Common Genera Across Full Community Tissue Study Models.** A) Common genera in the top 10 for adenoma Random Forest genus models. B) Common genera in the top 10 for unmatched carcinoma Random Forest genus models. B) Common genera in the top 10 for matched carcinoma Random Forest genus models. D) Common genera in the top 10 for adenoma Random Forest OTU models. E) Common genera in the top 10 for unmatched carcinoma Random Forest OTU models. F) Common genera in the top 10 for matched carcinoma Random Forest OTU models.

\newpage
