---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
geometry: margin=1.0in
bibliography: null
---

# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Tumors

\vspace{10mm}

\begin{center}
Marc A Sze and Patrick D Schloss

\vspace{10mm}

Supplemental
\end{center}


\newpage

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```


**Table S1: Bray-Curtis Distance Matrix Analyzed by PERMANOVA for Stool Adenoma**

```{r supplemental_table1, echo=FALSE, message=FALSE, warning=FALSE}

temp_t1 <- read_csv("../data/process/tables/beta_perm_adn_stool_summary.csv")

table_s1 <- temp_t1 %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = stringr::str_replace(study, 
                                      c("brim", "zeller", "baxter", "hale"), 
                                      c("Brim", "Zeller", "Baxter", "Hale")))

colnames(table_s1) <- c("F statistic", "R2", "P-value", "Study", "Adenoma (n)", "Control (n)")

kable(table_s1, format = "markdown", align = 'c', padding = 2)
```


\newpage

**Table S2: Bray-Curtis Distance Matrix Analyzed by PERMANOVA for Stool Carcinoma**

```{r supplemental_table2, echo=FALSE, message=FALSE, warning=FALSE}

temp_t2 <- read_csv("../data/process/tables/beta_perm_stool_summary.csv")

table_s2 <- temp_t2 %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = stringr::str_replace(study, 
                                      c("wang", "weir", "ahn", "zeller", "baxter", "hale", "flemer"), 
                                      c("Wang", "Weir", "Ahn", "Zeller", "Baxter", "Hale", "Flemer")))


colnames(table_s2) <- c("F statistic", "R2", "P-value", "Study", "Carcinoma (n)", "Control (n)")

kable(table_s2, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S3: Alpha Diversity Metrics RR in Tissue for Adenoma**

```{r supplemental_table3, echo=FALSE, message=FALSE, warning=FALSE}

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_RR_tissue_composite.csv")) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 2), 
         measure = c("OTU Richness", "Shannon Diversity", "Evenness"))

colnames(alpha_RR_adn_tissue) <- c("Relative Risk", "Lower Bound", "Upper Bound", "P-value", "Measure")

kable(alpha_RR_adn_tissue, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S4: Alpha Diversity Metrics RR in Tissue for Unmatched Carcinoma Non-Carcinoma Samples**

```{r supplemental_table4, echo=FALSE, message=FALSE, warning=FALSE}

alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_unmatched_tissue_composite.csv")) %>%   mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 2), 
         measure = c("OTU Richness", "Shannon Diversity", "Evenness"))

colnames(alpha_RR_crc_unmatch_tissue) <- c("Relative Risk", "Lower Bound", "Upper Bound", "P-value", "Measure")

kable(alpha_RR_crc_unmatch_tissue, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S5: Alpha Diversity Metrics RR in Tissue for Matched Carcinoma Non-Carcinoma Samples**

```{r supplemental_table5, echo=FALSE, message=FALSE, warning=FALSE}

alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_RR_matched_tissue_composite.csv")) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 2), 
         measure = c("OTU Richness", "Shannon Diversity", "Evenness"))

colnames(alpha_RR_crc_match_tissue) <- c("Relative Risk", "Lower Bound", "Upper Bound", "P-value", "Measure")

kable(alpha_RR_crc_match_tissue, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S6: Bray-Curtis Distance Matrix Analyzed by PERMANOVA for Tissue Adenoma**

```{r supplemental_table6, echo=FALSE, message=FALSE, warning=FALSE}

temp_t3a <- read_csv("../data/process/tables/beta_perm_adn_unmatched_tissue_summary.csv")
temp_t3b <- read_csv("../data/process/tables/beta_perm_adn_matched_tissue_summary.csv")

table_s6 <- temp_t3a %>% 
  bind_rows(temp_t3b) %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = c("Lu", "Flemer", "Lu (Matched)"))



colnames(table_s6) <- c("F statistic", "R2", "P-value", "Study", "Adenoma (n)", "Control (n)")

kable(table_s6, format = "markdown", align = 'c', padding = 2)
```

\newpage

**Table S7: Bray-Curtis Distance Matrix Analyzed by PERMANOVA for Tissue Carcinoma**

```{r supplemental_table7, echo=FALSE, message=FALSE, warning=FALSE}

temp_t4a <- read_csv("../data/process/tables/beta_perm_unmatched_tissue_summary.csv")
temp_t4b <- read_csv("../data/process/tables/beta_perm_matched_tissue_summary.csv")

table_s7 <- temp_t4a %>% 
  bind_rows(temp_t4b) %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = c("Sanapareddy", "Burns", "Flemer", "Chen", 
                   "Dejea (Matched)", "Geng (Matched)", "Burns (Matched)"))



colnames(table_s7) <- c("F statistic", "R2", "P-value", "Study", "Carcinoma (n)", "Control (n)")

kable(table_s7, format = "markdown", align = 'c', padding = 2)



```


\newpage

**Table S8: RR of Adenoma in Stool for each Individual Genera with Initial P-Value under 0.05**

```{r supplemental_table8, echo=FALSE, message=FALSE, warning=FALSE}

temp_8 <- read_csv("../data/process/tables/adn_select_genus_RR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 4), 
         bh = round(bh, digits = 3), 
         measure = str_replace_all(measure, "_", " ")) %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure)
  
colnames(temp_8) <- c("RR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID")

kable(temp_8, format = "markdown", align = 'c', padding = 2)

```


\newpage

**Table S9: RR of Adenoma in Tissue for each Individual Genera with Initial P-Value under 0.05**

```{r supplemental_table9, echo=FALSE, message=FALSE, warning=FALSE}

temp_9 <- read_csv("../data/process/tables/adn_select_genus_RR_tissue_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 4), 
         bh = round(bh, digits = 3), 
         measure = str_replace_all(measure, "_", " ")) %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure)
  
colnames(temp_9) <- c("RR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID")

kable(temp_9, format = "markdown", align = 'c', padding = 2)

```


\newpage

**Table S10: RR of Carcinoma in Stool for each Individual Genera with Initial P-Value under 0.05**

```{r supplemental_table10, echo=FALSE, message=FALSE, warning=FALSE}

temp_10 <- read_csv("../data/process/tables/select_genus_RR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " ")) %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure)
  
colnames(temp_10) <- c("RR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID")

kable(temp_10, format = "markdown", align = 'c', padding = 2)

```


\newpage


**Table S11: RR of Carcinoma in Tissue for each Individual Genera with Initial P-Value under 0.05**

```{r supplemental_table11, echo=FALSE, message=FALSE, warning=FALSE}

temp_11 <- read_csv("../data/process/tables/select_genus_RR_tissue_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " ")) %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure)
  
colnames(temp_11) <- c("RR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID")

kable(temp_11, format = "markdown", align = 'c', padding = 2)

```





