---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
geometry: margin=1.0in
bibliography: null
---

# Making Sense of the Noise: Leveraging Existing 16S rRNA Gene Surveys to Identify Key Community Members in Colorectal Tumors

\vspace{10mm}

\begin{center}
Marc A Sze and Patrick D Schloss

\vspace{10mm}

Supplemental
\end{center}


\newpage

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```


**Table S1: Bray-Curtis Distance Matrix Analyzed by PERMANOVA for Stool**

```{r supplemental_table1, echo=FALSE, message=FALSE, warning=FALSE}

temp_t1 <- read_csv("../data/process/tables/beta_perm_adn_stool_summary.csv")

table_s1 <- temp_t1 %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = stringr::str_replace(study, 
                                      c("brim", "zeller", "baxter", "hale"), 
                                      c("Brim", "Zeller", "Baxter", "Hale")), 
         comparison = "Control vs. Adenoma") %>% 
  select(fstat, r2, pvalue, study, comparison)

colnames(table_s1) <- c("F statistic", "R2", "P-value", "Study", "Comparison")

temp_t2 <- read_csv("../data/process/tables/beta_perm_stool_summary.csv")

table_s2 <- temp_t2 %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = stringr::str_replace(study, 
                                      c("wang", "weir", "ahn", "zeller", "baxter", "hale", "flemer"), 
                                      c("Wang", "Weir", "Ahn", "Zeller", "Baxter", "Hale", "Flemer")), 
         comparison = "Control vs. Carcinoma") %>% 
  select(fstat, r2, pvalue, study, comparison)


colnames(table_s2) <- c("F statistic", "R2", "P-value", "Study", "Comparison")

combined_table_s1 <- table_s1 %>% bind_rows(table_s2)


kable(combined_table_s1, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S2: Bray-Curtis Distance Matrix Analyzed by PERMANOVA for Tissue**

```{r supplemental_table6, echo=FALSE, message=FALSE, warning=FALSE}

temp_t3a <- read_csv("../data/process/tables/beta_perm_adn_unmatched_tissue_summary.csv")
temp_t3b <- read_csv("../data/process/tables/beta_perm_adn_matched_tissue_summary.csv")

table_s3 <- temp_t3a %>% 
  bind_rows(temp_t3b) %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = c("Lu", "Flemer", "Lu (Matched)"), 
         comparison = "Control vs. Adenoma") %>% 
  select(fstat, r2, pvalue, study, comparison)

colnames(table_s3) <- c("F statistic", "R2", "P-value", "Study", "Comparison")


temp_t4a <- read_csv("../data/process/tables/beta_perm_unmatched_tissue_summary.csv")
temp_t4b <- read_csv("../data/process/tables/beta_perm_matched_tissue_summary.csv")

table_s4 <- temp_t4a %>% 
  bind_rows(temp_t4b) %>% 
  mutate(fstat = round(fstat, digits = 2), 
         r2 = round(r2, digits = 3), 
         pvalue = round(pvalue, digits = 4), 
         study = c("Sanapareddy", "Burns", "Flemer", "Chen", 
                   "Dejea (Matched)", "Geng (Matched)", "Burns (Matched)"), 
         comparison = "Control vs. Carcinoma") %>% 
  select(fstat, r2, pvalue, study, comparison)



colnames(table_s4) <- c("F statistic", "R2", "P-value", "Study", "Comparison")

combined_table_s2 <- table_s3 %>% bind_rows(table_s4)


kable(combined_table_s2, format = "markdown", align = 'c', padding = 2)
```

\newpage



**Table S3: Alpha Diversity Metrics OR in Tissue**

```{r supplemental_table3, echo=FALSE, message=FALSE, warning=FALSE}

alpha_RR_adn_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_adn_OR_tissue_composite.csv")) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 2), 
         measure = c("OTU Richness", "Shannon Diversity", "Evenness"), 
         comparison = "Control vs. Adenoma", 
         tissue_type = "Combined")

colnames(alpha_RR_adn_tissue) <- c("Odds Ratio", "Lower Bound", "Upper Bound", "P-value", "Measure", 
                                   "Comparison", "Tissue Group")



alpha_RR_crc_unmatch_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_unmatched_tissue_composite.csv")) %>%   mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 2), 
         measure = c("OTU Richness", "Shannon Diversity", "Evenness"), 
         comparison = "Control vs. Carcinoma", 
         tissue_type = "Unmatched")

colnames(alpha_RR_crc_unmatch_tissue) <- c("Odds Ratio", "Lower Bound", "Upper Bound", "P-value", "Measure", 
                                           "Comparison", "Tissue Group")


alpha_RR_crc_match_tissue <- as.data.frame(read_csv("../data/process/tables/alpha_OR_matched_tissue_composite.csv")) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = round(pvalue, digits = 2), 
         measure = c("OTU Richness", "Shannon Diversity", "Evenness"), 
         comparison = "Control vs. Carcinoma", 
         tissue_type = "Matched")

colnames(alpha_RR_crc_match_tissue) <-  c("Odds Ratio", "Lower Bound", "Upper Bound", "P-value", "Measure", 
                                          "Comparison", "Tissue Group")

combined_table_S3 <- alpha_RR_adn_tissue %>% bind_rows(alpha_RR_crc_unmatch_tissue, alpha_RR_crc_match_tissue)



kable(combined_table_S3, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S4: OR of Tumor in Stool for each Individual Genera with Initial P-Value under 0.05**

```{r supplemental_table8, echo=FALSE, message=FALSE, warning=FALSE}

temp_8 <- read_csv("../data/process/tables/adn_select_genus_OR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " "), 
         comparison = "Control vs. Adenoma") %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure, comparison)
  
colnames(temp_8) <- c("OR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID", "Comparison")

temp_10 <- read_csv("../data/process/tables/select_genus_OR_stool_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " "), 
         comparison = "Control vs. Carcinoma") %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure, comparison)

colnames(temp_10) <- c("OR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID", "Comparison")

combined_table_S4 <- temp_8 %>% bind_rows(temp_10)

kable(combined_table_S4, format = "markdown", align = 'c', padding = 2)

```


\newpage


**Table S9: OR of Adenoma in Tissue for each Individual Genera with Initial P-Value under 0.05**

```{r supplemental_table9, echo=FALSE, message=FALSE, warning=FALSE}

temp_9 <- read_csv("../data/process/tables/adn_select_genus_OR_tissue_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " "), 
         comparison = "Control vs. Adenoma", 
         tissue_type = "Combined") %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure, comparison, tissue_type)
  
colnames(temp_9) <- c("OR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID", 
                      "Comparison", "Tissue Group")

temp_11 <- read_csv("../data/process/tables/select_genus_OR_unmatched_tissue_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " "), 
         comparison = "Control vs. Carcinoma", 
         tissue_type = "Unmatched") %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure, comparison, tissue_type)
  
colnames(temp_11) <- c("OR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID", 
                      "Comparison", "Tissue Group")

temp_12 <- read_csv("../data/process/tables/select_genus_OR_matched_tissue_composite.csv") %>% 
  mutate(bh = p.adjust(pvalue, method = "BH")) %>% filter(pvalue < 0.05) %>% 
  arrange(bh, pvalue, rr) %>% 
  mutate(rr = round(rr, digits = 2), 
         ci_lb = round(ci_lb, digits = 2), 
         ci_ub = round(ci_ub, digits = 2), 
         pvalue = format(pvalue, scientific = T, digits = 3), 
         bh = format(bh, scientific = T, digits = 3), 
         measure = str_replace_all(measure, "_", " "), 
         comparison = "Control vs. Carcinoma", 
         tissue_type = "Matched") %>% 
  select(rr, ci_lb, ci_ub, pvalue, bh, measure, comparison, tissue_type)
  
colnames(temp_12) <- c("OR", "Lower Bound", "Upper Bound", "P-value", "BH", "Lowest Genera ID", 
                      "Comparison", "Tissue Group")

combined_table_S5 <- temp_9 %>% bind_rows(temp_11, temp_12)

kable(combined_table_S5, format = "markdown", align = 'c', padding = 2)

```


\newpage



